timestampFn = {
  false,
}
mrcT = {
  alias2modT = {},
  hiddenT = {},
  version2modT = {},
}

mrcMpathT = {}
spiderT = {
  ["/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf"] = {
    eagle = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["eagle/ctr-0.9.0--py34_0"] = {
          Category = "['Sequence feature detection', 'Sequence word comparison']",
          Description = "EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.",
          Name = "eagle",
          URL = "https://quay.io/repository/biocontainers/eagle",
          Version = "ctr-0.9.0--py34_0",
          canonical = "ctr-0.9.0--py34_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eagle/ctr-0.9.0--py34_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eagle:0.9.0--py34_0, which exposes the
following programs:

 - 2to3-3.4
 - eagle
 - easy_install-3.4
 - flask
 - gif2h5
 - h52gif
 - h5c++
 - h5cc
 - h5copy
 - h5debug
 - h5diff
 - h5dump
 - h5import
 - h5jam
 - h5ls
 - h5mkgrp
 - h5perf_serial
 - h5redeploy
 - h5repack
 - h5repart
 - h5stat
 - h5unjam
 - idle3.4
 - pydoc3.4
 - python3.4
 - python3.4-config
 - python3.4m
 - python3.4m-config
 - pyvenv-3.4

This container was pulled from:

	https://quay.io/repository/biocontainers/eagle

If you encounter errors in eagle or need help running the
tools it contains, please contact the developer at

	http://bioinformatics.ua.pt/software/eagle/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.*y.000000034.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.*y.000000034.*_.*zfinal",
          whatis = {
            "Name: eagle", "Version: ctr-0.9.0--py34_0", "Category: ['Sequence feature detection', 'Sequence word comparison']", "Keywords: ['Genomics', 'Sequencing', 'Virology']", "Description: EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.", "URL: https://quay.io/repository/biocontainers/eagle",
          },
        },
        ["eagle/ctr-0.9.0--py35_0"] = {
          Category = "['Sequence feature detection', 'Sequence word comparison']",
          Description = "EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.",
          Name = "eagle",
          URL = "https://quay.io/repository/biocontainers/eagle",
          Version = "ctr-0.9.0--py35_0",
          canonical = "ctr-0.9.0--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eagle/ctr-0.9.0--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eagle:0.9.0--py35_0, which exposes the
following programs:

 - eagle
 - easy_install-3.5
 - flask
 - gif2h5
 - h52gif
 - h5c++
 - h5cc
 - h5copy
 - h5debug
 - h5diff
 - h5dump
 - h5fc
 - h5import
 - h5jam
 - h5ls
 - h5mkgrp
 - h5perf_serial
 - h5redeploy
 - h5repack
 - h5repart
 - h5stat
 - h5unjam
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/eagle

If you encounter errors in eagle or need help running the
tools it contains, please contact the developer at

	http://bioinformatics.ua.pt/software/eagle/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: eagle", "Version: ctr-0.9.0--py35_0", "Category: ['Sequence feature detection', 'Sequence word comparison']", "Keywords: ['Genomics', 'Sequencing', 'Virology']", "Description: EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.", "URL: https://quay.io/repository/biocontainers/eagle",
          },
        },
        ["eagle/ctr-0.9.0--py36_0"] = {
          Category = "['Sequence feature detection', 'Sequence word comparison']",
          Description = "EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.",
          Name = "eagle",
          URL = "https://quay.io/repository/biocontainers/eagle",
          Version = "ctr-0.9.0--py36_0",
          canonical = "ctr-0.9.0--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eagle/ctr-0.9.0--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eagle:0.9.0--py36_0, which exposes the
following programs:

 - eagle
 - easy_install-3.6
 - flask
 - gif2h5
 - h52gif
 - h5c++
 - h5cc
 - h5copy
 - h5debug
 - h5diff
 - h5dump
 - h5import
 - h5jam
 - h5ls
 - h5mkgrp
 - h5perf_serial
 - h5redeploy
 - h5repack
 - h5repart
 - h5stat
 - h5unjam

This container was pulled from:

	https://quay.io/repository/biocontainers/eagle

If you encounter errors in eagle or need help running the
tools it contains, please contact the developer at

	http://bioinformatics.ua.pt/software/eagle/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: eagle", "Version: ctr-0.9.0--py36_0", "Category: ['Sequence feature detection', 'Sequence word comparison']", "Keywords: ['Genomics', 'Sequencing', 'Virology']", "Description: EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.", "URL: https://quay.io/repository/biocontainers/eagle",
          },
        },
        ["eagle/ctr-0.9.3.3--py35_0"] = {
          Category = "['Sequence feature detection', 'Sequence word comparison']",
          Description = "EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.",
          Name = "eagle",
          URL = "https://quay.io/repository/biocontainers/eagle",
          Version = "ctr-0.9.3.3--py35_0",
          canonical = "ctr-0.9.3.3--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eagle/ctr-0.9.3.3--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eagle:0.9.3.3--py35_0, which exposes the
following programs:

 - eagle
 - flask
 - gif2h5
 - h52gif
 - h5c++
 - h5cc
 - h5copy
 - h5debug
 - h5diff
 - h5dump
 - h5fc
 - h5import
 - h5jam
 - h5ls
 - h5mkgrp
 - h5perf_serial
 - h5redeploy
 - h5repack
 - h5repart
 - h5stat
 - h5unjam
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - vcf_filter.py
 - vcf_melt
 - vcf_sample_filter.py

This container was pulled from:

	https://quay.io/repository/biocontainers/eagle

If you encounter errors in eagle or need help running the
tools it contains, please contact the developer at

	http://bioinformatics.ua.pt/software/eagle/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000003.000000003.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000003.000000003.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: eagle", "Version: ctr-0.9.3.3--py35_0", "Category: ['Sequence feature detection', 'Sequence word comparison']", "Keywords: ['Genomics', 'Sequencing', 'Virology']", "Description: EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.", "URL: https://quay.io/repository/biocontainers/eagle",
          },
        },
        ["eagle/ctr-0.9.3.3--py35h24bf2e0_2"] = {
          Category = "['Sequence feature detection', 'Sequence word comparison']",
          Description = "EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.",
          Name = "eagle",
          URL = "https://quay.io/repository/biocontainers/eagle",
          Version = "ctr-0.9.3.3--py35h24bf2e0_2",
          canonical = "ctr-0.9.3.3--py35h24bf2e0_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eagle/ctr-0.9.3.3--py35h24bf2e0_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eagle:0.9.3.3--py35h24bf2e0_2, which exposes the
following programs:

 - eagle
 - flask
 - gif2h5
 - h52gif
 - h5c++
 - h5cc
 - h5clear
 - h5copy
 - h5debug
 - h5diff
 - h5dump
 - h5fc
 - h5format_convert
 - h5import
 - h5jam
 - h5ls
 - h5mkgrp
 - h5perf_serial
 - h5redeploy
 - h5repack
 - h5repart
 - h5stat
 - h5unjam
 - h5watch
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - vcf_filter.py
 - vcf_melt
 - vcf_sample_filter.py

This container was pulled from:

	https://quay.io/repository/biocontainers/eagle

If you encounter errors in eagle or need help running the
tools it contains, please contact the developer at

	http://bioinformatics.ua.pt/software/eagle/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 28,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000003.000000003.*y.000000035.*h.000000024.*bf.000000002.*e.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000003.000000003.*y.000000035.*h.000000024.*bf.000000002.*e.*_.000000002.*zfinal",
          whatis = {
            "Name: eagle", "Version: ctr-0.9.3.3--py35h24bf2e0_2", "Category: ['Sequence feature detection', 'Sequence word comparison']", "Keywords: ['Genomics', 'Sequencing', 'Virology']", "Description: EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.", "URL: https://quay.io/repository/biocontainers/eagle",
          },
        },
        ["eagle/ctr-0.9.3.3--py36_0"] = {
          Category = "['Sequence feature detection', 'Sequence word comparison']",
          Description = "EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.",
          Name = "eagle",
          URL = "https://quay.io/repository/biocontainers/eagle",
          Version = "ctr-0.9.3.3--py36_0",
          canonical = "ctr-0.9.3.3--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eagle/ctr-0.9.3.3--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eagle:0.9.3.3--py36_0, which exposes the
following programs:

 - eagle
 - flask
 - gif2h5
 - h52gif
 - h5c++
 - h5cc
 - h5copy
 - h5debug
 - h5diff
 - h5dump
 - h5fc
 - h5import
 - h5jam
 - h5ls
 - h5mkgrp
 - h5perf_serial
 - h5redeploy
 - h5repack
 - h5repart
 - h5stat
 - h5unjam
 - vcf_filter.py
 - vcf_melt
 - vcf_sample_filter.py

This container was pulled from:

	https://quay.io/repository/biocontainers/eagle

If you encounter errors in eagle or need help running the
tools it contains, please contact the developer at

	http://bioinformatics.ua.pt/software/eagle/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000003.000000003.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000003.000000003.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: eagle", "Version: ctr-0.9.3.3--py36_0", "Category: ['Sequence feature detection', 'Sequence word comparison']", "Keywords: ['Genomics', 'Sequencing', 'Virology']", "Description: EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.", "URL: https://quay.io/repository/biocontainers/eagle",
          },
        },
        ["eagle/ctr-0.9.3.3--py36h24bf2e0_2"] = {
          Category = "['Sequence feature detection', 'Sequence word comparison']",
          Description = "EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.",
          Name = "eagle",
          URL = "https://quay.io/repository/biocontainers/eagle",
          Version = "ctr-0.9.3.3--py36h24bf2e0_2",
          canonical = "ctr-0.9.3.3--py36h24bf2e0_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eagle/ctr-0.9.3.3--py36h24bf2e0_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eagle:0.9.3.3--py36h24bf2e0_2, which exposes the
following programs:

 - eagle
 - flask
 - gif2h5
 - h52gif
 - h5c++
 - h5cc
 - h5clear
 - h5copy
 - h5debug
 - h5diff
 - h5dump
 - h5fc
 - h5format_convert
 - h5import
 - h5jam
 - h5ls
 - h5mkgrp
 - h5perf_serial
 - h5redeploy
 - h5repack
 - h5repart
 - h5stat
 - h5unjam
 - h5watch
 - vcf_filter.py
 - vcf_melt
 - vcf_sample_filter.py

This container was pulled from:

	https://quay.io/repository/biocontainers/eagle

If you encounter errors in eagle or need help running the
tools it contains, please contact the developer at

	http://bioinformatics.ua.pt/software/eagle/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 28,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000003.000000003.*y.000000036.*h.000000024.*bf.000000002.*e.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000003.000000003.*y.000000036.*h.000000024.*bf.000000002.*e.*_.000000002.*zfinal",
          whatis = {
            "Name: eagle", "Version: ctr-0.9.3.3--py36h24bf2e0_2", "Category: ['Sequence feature detection', 'Sequence word comparison']", "Keywords: ['Genomics', 'Sequencing', 'Virology']", "Description: EAGLE is an alignment-free method and associated program to compute relative absent words (RAW) in genomic sequences using a reference sequence.", "URL: https://quay.io/repository/biocontainers/eagle",
          },
        },
      },
    },
    easydev = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["easydev/ctr-0.9.14--py35_1"] = {
          Category = "['Bioinformatics']",
          Description = "The easydev package",
          Name = "easydev",
          URL = "https://quay.io/repository/biocontainers/easydev",
          Version = "ctr-0.9.14--py35_1",
          canonical = "ctr-0.9.14--py35_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/easydev/ctr-0.9.14--py35_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/easydev:0.9.14--py35_1, which exposes the
following programs:

 - browse
 - easy_install-3.5
 - easydev_buildPackage
 - ibrowse
 - idle3.5
 - multigit
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/easydev

If you encounter errors in easydev or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/easydev

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000014.*y.000000035.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000014.*y.000000035.*_.000000001.*zfinal",
          whatis = {
            "Name: easydev", "Version: ctr-0.9.14--py35_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The easydev package", "URL: https://quay.io/repository/biocontainers/easydev",
          },
        },
        ["easydev/ctr-0.9.31--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The easydev package",
          Name = "easydev",
          URL = "https://quay.io/repository/biocontainers/easydev",
          Version = "ctr-0.9.31--py27_1",
          canonical = "ctr-0.9.31--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/easydev/ctr-0.9.31--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/easydev:0.9.31--py27_1, which exposes the
following programs:

 - browse
 - easydev_buildPackage
 - ibrowse
 - multigit

This container was pulled from:

	https://quay.io/repository/biocontainers/easydev

If you encounter errors in easydev or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/easydev

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000031.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000031.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: easydev", "Version: ctr-0.9.31--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The easydev package", "URL: https://quay.io/repository/biocontainers/easydev",
          },
        },
        ["easydev/ctr-0.9.31--py34_1"] = {
          Category = "['Bioinformatics']",
          Description = "The easydev package",
          Name = "easydev",
          URL = "https://quay.io/repository/biocontainers/easydev",
          Version = "ctr-0.9.31--py34_1",
          canonical = "ctr-0.9.31--py34_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/easydev/ctr-0.9.31--py34_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/easydev:0.9.31--py34_1, which exposes the
following programs:

 - 2to3-3.4
 - browse
 - easy_install-3.4
 - easydev_buildPackage
 - ibrowse
 - idle3.4
 - multigit
 - pydoc3.4
 - python3.4
 - python3.4-config
 - python3.4m
 - python3.4m-config
 - pyvenv-3.4

This container was pulled from:

	https://quay.io/repository/biocontainers/easydev

If you encounter errors in easydev or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/easydev

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000031.*y.000000034.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000031.*y.000000034.*_.000000001.*zfinal",
          whatis = {
            "Name: easydev", "Version: ctr-0.9.31--py34_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The easydev package", "URL: https://quay.io/repository/biocontainers/easydev",
          },
        },
        ["easydev/ctr-0.9.31--py35_1"] = {
          Category = "['Bioinformatics']",
          Description = "The easydev package",
          Name = "easydev",
          URL = "https://quay.io/repository/biocontainers/easydev",
          Version = "ctr-0.9.31--py35_1",
          canonical = "ctr-0.9.31--py35_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/easydev/ctr-0.9.31--py35_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/easydev:0.9.31--py35_1, which exposes the
following programs:

 - browse
 - easy_install-3.5
 - easydev_buildPackage
 - ibrowse
 - idle3.5
 - multigit
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/easydev

If you encounter errors in easydev or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/easydev

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000031.*y.000000035.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000031.*y.000000035.*_.000000001.*zfinal",
          whatis = {
            "Name: easydev", "Version: ctr-0.9.31--py35_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The easydev package", "URL: https://quay.io/repository/biocontainers/easydev",
          },
        },
      },
    },
    ebfilter = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ebfilter/ctr-0.2.1--0"] = {
          Category = "['Bioinformatics']",
          Description = "The ebfilter package",
          Name = "ebfilter",
          URL = "https://quay.io/repository/biocontainers/ebfilter",
          Version = "ctr-0.2.1--0",
          canonical = "ctr-0.2.1--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ebfilter/ctr-0.2.1--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ebfilter:0.2.1--0, which exposes the
following programs:

 - EBFilter
 - bcftools
 - color-chrs.pl
 - guess-ploidy.py
 - ksu
 - plot-roh.py
 - plot-vcfstats
 - run-roh.pl
 - samtools
 - vcf_filter.py
 - vcf_melt
 - vcf_sample_filter.py
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/ebfilter

If you encounter errors in ebfilter or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ebfilter

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000001.*zfinal",
          whatis = {
            "Name: ebfilter", "Version: ctr-0.2.1--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ebfilter package", "URL: https://quay.io/repository/biocontainers/ebfilter",
          },
        },
        ["ebfilter/ctr-0.2.1--py27h24bf2e0_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ebfilter package",
          Name = "ebfilter",
          URL = "https://quay.io/repository/biocontainers/ebfilter",
          Version = "ctr-0.2.1--py27h24bf2e0_1",
          canonical = "ctr-0.2.1--py27h24bf2e0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ebfilter/ctr-0.2.1--py27h24bf2e0_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ebfilter:0.2.1--py27h24bf2e0_1, which exposes the
following programs:

 - EBFilter
 - bcftools
 - color-chrs.pl
 - guess-ploidy.py
 - plot-roh.py
 - plot-vcfstats
 - run-roh.pl
 - samtools
 - vcf_filter.py
 - vcf_melt
 - vcf_sample_filter.py
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/ebfilter

If you encounter errors in ebfilter or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ebfilter

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 26,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          whatis = {
            "Name: ebfilter", "Version: ctr-0.2.1--py27h24bf2e0_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ebfilter package", "URL: https://quay.io/repository/biocontainers/ebfilter",
          },
        },
      },
    },
    ebisearch = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ebisearch/ctr-0.0.2--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ebisearch package",
          Name = "ebisearch",
          URL = "https://quay.io/repository/biocontainers/ebisearch",
          Version = "ctr-0.0.2--py27_0",
          canonical = "ctr-0.0.2--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ebisearch/ctr-0.0.2--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ebisearch:0.0.2--py27_0, which exposes the
following programs:

 - ebi_metagenomics
 - ebisearch
 - flake8
 - pycodestyle
 - pyflakes

This container was pulled from:

	https://quay.io/repository/biocontainers/ebisearch

If you encounter errors in ebisearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ebisearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000002.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000002.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: ebisearch", "Version: ctr-0.0.2--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ebisearch package", "URL: https://quay.io/repository/biocontainers/ebisearch",
          },
        },
        ["ebisearch/ctr-0.0.3--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ebisearch package",
          Name = "ebisearch",
          URL = "https://quay.io/repository/biocontainers/ebisearch",
          Version = "ctr-0.0.3--py27_0",
          canonical = "ctr-0.0.3--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ebisearch/ctr-0.0.3--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ebisearch:0.0.3--py27_0, which exposes the
following programs:

 - ebi_metagenomics
 - ebisearch
 - flake8
 - pycodestyle
 - pyflakes

This container was pulled from:

	https://quay.io/repository/biocontainers/ebisearch

If you encounter errors in ebisearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ebisearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: ebisearch", "Version: ctr-0.0.3--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ebisearch package", "URL: https://quay.io/repository/biocontainers/ebisearch",
          },
        },
        ["ebisearch/ctr-0.0.3--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ebisearch package",
          Name = "ebisearch",
          URL = "https://quay.io/repository/biocontainers/ebisearch",
          Version = "ctr-0.0.3--py27_1",
          canonical = "ctr-0.0.3--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ebisearch/ctr-0.0.3--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ebisearch:0.0.3--py27_1, which exposes the
following programs:

 - ebi_metagenomics
 - ebisearch
 - flake8
 - pycodestyle
 - pyflakes

This container was pulled from:

	https://quay.io/repository/biocontainers/ebisearch

If you encounter errors in ebisearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ebisearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: ebisearch", "Version: ctr-0.0.3--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ebisearch package", "URL: https://quay.io/repository/biocontainers/ebisearch",
          },
        },
      },
    },
    ecmwfapi = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ecmwfapi/ctr-1.4.1--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecmwfapi package",
          Name = "ecmwfapi",
          URL = "https://quay.io/repository/biocontainers/ecmwfapi",
          Version = "ctr-1.4.1--py27_0",
          canonical = "ctr-1.4.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecmwfapi/ctr-1.4.1--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecmwfapi:1.4.1--py27_0, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/ecmwfapi

If you encounter errors in ecmwfapi or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecmwfapi

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: ecmwfapi", "Version: ctr-1.4.1--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecmwfapi package", "URL: https://quay.io/repository/biocontainers/ecmwfapi",
          },
        },
        ["ecmwfapi/ctr-1.4.1--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ecmwfapi package",
          Name = "ecmwfapi",
          URL = "https://quay.io/repository/biocontainers/ecmwfapi",
          Version = "ctr-1.4.1--py27_1",
          canonical = "ctr-1.4.1--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecmwfapi/ctr-1.4.1--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecmwfapi:1.4.1--py27_1, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/ecmwfapi

If you encounter errors in ecmwfapi or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecmwfapi

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: ecmwfapi", "Version: ctr-1.4.1--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecmwfapi package", "URL: https://quay.io/repository/biocontainers/ecmwfapi",
          },
        },
        ["ecmwfapi/ctr-1.4.1--py34_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecmwfapi package",
          Name = "ecmwfapi",
          URL = "https://quay.io/repository/biocontainers/ecmwfapi",
          Version = "ctr-1.4.1--py34_0",
          canonical = "ctr-1.4.1--py34_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecmwfapi/ctr-1.4.1--py34_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecmwfapi:1.4.1--py34_0, which exposes the
following programs:

 - 2to3-3.4
 - easy_install-3.4
 - idle3.4
 - pydoc3.4
 - python3.4
 - python3.4-config
 - python3.4m
 - python3.4m-config
 - pyvenv-3.4

This container was pulled from:

	https://quay.io/repository/biocontainers/ecmwfapi

If you encounter errors in ecmwfapi or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecmwfapi

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000034.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000034.*_.*zfinal",
          whatis = {
            "Name: ecmwfapi", "Version: ctr-1.4.1--py34_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecmwfapi package", "URL: https://quay.io/repository/biocontainers/ecmwfapi",
          },
        },
        ["ecmwfapi/ctr-1.4.1--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecmwfapi package",
          Name = "ecmwfapi",
          URL = "https://quay.io/repository/biocontainers/ecmwfapi",
          Version = "ctr-1.4.1--py35_0",
          canonical = "ctr-1.4.1--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecmwfapi/ctr-1.4.1--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecmwfapi:1.4.1--py35_0, which exposes the
following programs:

 - easy_install-3.5
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/ecmwfapi

If you encounter errors in ecmwfapi or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecmwfapi

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: ecmwfapi", "Version: ctr-1.4.1--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecmwfapi package", "URL: https://quay.io/repository/biocontainers/ecmwfapi",
          },
        },
        ["ecmwfapi/ctr-1.4.1--py35_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ecmwfapi package",
          Name = "ecmwfapi",
          URL = "https://quay.io/repository/biocontainers/ecmwfapi",
          Version = "ctr-1.4.1--py35_1",
          canonical = "ctr-1.4.1--py35_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecmwfapi/ctr-1.4.1--py35_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecmwfapi:1.4.1--py35_1, which exposes the
following programs:

 - easy_install-3.5
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/ecmwfapi

If you encounter errors in ecmwfapi or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecmwfapi

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000035.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000035.*_.000000001.*zfinal",
          whatis = {
            "Name: ecmwfapi", "Version: ctr-1.4.1--py35_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecmwfapi package", "URL: https://quay.io/repository/biocontainers/ecmwfapi",
          },
        },
        ["ecmwfapi/ctr-1.4.1--py36_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ecmwfapi package",
          Name = "ecmwfapi",
          URL = "https://quay.io/repository/biocontainers/ecmwfapi",
          Version = "ctr-1.4.1--py36_1",
          canonical = "ctr-1.4.1--py36_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecmwfapi/ctr-1.4.1--py36_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecmwfapi:1.4.1--py36_1, which exposes the
following programs:

 - easy_install-3.6

This container was pulled from:

	https://quay.io/repository/biocontainers/ecmwfapi

If you encounter errors in ecmwfapi or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecmwfapi

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000036.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.000000036.*_.000000001.*zfinal",
          whatis = {
            "Name: ecmwfapi", "Version: ctr-1.4.1--py36_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecmwfapi package", "URL: https://quay.io/repository/biocontainers/ecmwfapi",
          },
        },
        ["ecmwfapi/ctr-1.4.1--py_2"] = {
          Category = "['Bioinformatics']",
          Description = "The ecmwfapi package",
          Name = "ecmwfapi",
          URL = "https://quay.io/repository/biocontainers/ecmwfapi",
          Version = "ctr-1.4.1--py_2",
          canonical = "ctr-1.4.1--py_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecmwfapi/ctr-1.4.1--py_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecmwfapi:1.4.1--py_2, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/ecmwfapi

If you encounter errors in ecmwfapi or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecmwfapi

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000004.000000001.*y.*_.000000002.*zfinal",
          whatis = {
            "Name: ecmwfapi", "Version: ctr-1.4.1--py_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecmwfapi package", "URL: https://quay.io/repository/biocontainers/ecmwfapi",
          },
        },
      },
    },
    ecomet = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ecomet/ctr-phenomenal-v1.1_cv0.3.39"] = {
          Category = "['Bioinformatics']",
          Description = "The ecomet package",
          Name = "ecomet",
          URL = "https://hub.docker.com/r/biocontainers/ecomet",
          Version = "ctr-phenomenal-v1.1_cv0.3.39",
          canonical = "ctr-phenomenal-v1.1_cv0.3.39",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecomet/ctr-phenomenal-v1.1_cv0.3.39.lua",
          help = [[This is a module file for the container biocontainers/ecomet:phenomenal-v1.1_cv0.3.39, which exposes the
following programs:

 - R
 - Rscript
 - X
 - Xorg
 - accessdb
 - appres
 - apropos
 - atobm
 - bdftopcf
 - bdftruncate
 - bitmap
 - blankfilter.r
 - bmtoa
 - browse
 - bsd-from
 - bsd-write
 - calendar
 - cameraToFeatureXML.r
 - catman
 - consensusXMLToXcms.r
 - cvt
 - dh_xsf_substvars
 - dilutionfilter.r
 - editres
 - eqn
 - f77
 - f95
 - featureXMLToCAMERA.r
 - featureXMLToXcms.r
 - filenameextractor.r
 - fillPeaks.r
 - findAdducts.r
 - findIsotopes.r
 - findPeaks.r
 - fonttosfnt
 - from
 - genccode
 - gencmn
 - gennorm2
 - gensprep
 - geqn
 - gfortran
 - gfortran-5
 - gpic
 - groff
 - grog
 - grops
 - grotty
 - group.r
 - groupCorr.r
 - groupFWHM.r
 - gtbl
 - gtf
 - h5c++
 - h5cc
 - h5fc
 - hd
 - iceauth
 - ico
 - icupkg
 - isdv4-serial-debugger
 - isdv4-serial-inputattach
 - koi8rxterm
 - lexgrog
 - libpng12-config
 - listres
 - lorder
 - luit
 - lxterm
 - man
 - mandb
 - manpath
 - mkfontdir
 - mkfontscale
 - mtbls520_01_mtbls_download.sh
 - mtbls520_02_extract.sh
 - mtbls520_03_qc_perform.r
 - mtbls520_03_qc_preparations.sh
 - mtbls520_04_preparations.r
 - mtbls520_04_preparations.sh
 - mtbls520_05a_import_maf.r
 - mtbls520_05b_peak_picking.r
 - mtbls520_06_import_traits.r
 - mtbls520_07_species_diversity.r
 - mtbls520_08a_species_shannon.r
 - mtbls520_08b_species_unique.r
 - mtbls520_08c_species_variability.r
 - mtbls520_08d_species_concentration.r
 - mtbls520_08e_species_features.r
 - mtbls520_09_species_venn.r
 - mtbls520_10_species_varpart.r
 - mtbls520_11_species_nmds.r
 - mtbls520_12_species_marchantia.r
 - mtbls520_14_ecology_varpart.r
 - mtbls520_15_ecology_pca.r
 - mtbls520_16_ecology_rda.r
 - mtbls520_17_ecology_splsda.r
 - mtbls520_18_phylogeny.r
 - mtbls520_19a_seasons_shannon.r
 - mtbls520_19b_seasons_unique.r
 - mtbls520_19c_seasons_variability.r
 - mtbls520_19d_seasons_concentration.r
 - mtbls520_19e_seasons_features.r
 - mtbls520_22_seasons_pca.r
 - mtbls520_23_seasons_rda.r
 - mtbls520_24_seasons_nmds.r
 - nc-config
 - ncal
 - neqn
 - nroff
 - oclock
 - paperconf
 - paperconfig
 - pic
 - preconv
 - prepareOutput.r
 - printerbanner
 - rendercheck
 - retCor.r
 - rstart
 - rstartd
 - runTest1.R
 - runTest1.sh
 - save_chromatogram.r
 - sessreg
 - setphenotype.r
 - setxkbmap
 - show_chromatogram.r
 - showrgb
 - smproxy
 - soelim
 - startx
 - synclient
 - syndaemon
 - tbl
 - test_output.r
 - transset
 - troff
 - uconv
 - ucs2any
 - update-fonts-alias
 - update-fonts-dir
 - update-fonts-scale
 - uxterm
 - viewres
 - vmmouse_detect
 - whatis
 - x-terminal-emulator
 - x11perf
 - x11perfcomp
 - x86_64-linux-gnu-gfortran
 - x86_64-linux-gnu-gfortran-5
 - xbiff
 - xcalc
 - xclipboard
 - xclock
 - xcmsCollect.r
 - xcmsdb
 - xcmssplitter.r
 - xconsole
 - xcursorgen
 - xcutsel
 - xdg-desktop-icon
 - xdg-desktop-menu
 - xdg-email
 - xdg-icon-resource
 - xdg-mime
 - xdg-open
 - xdg-screensaver
 - xdg-settings
 - xditview
 - xdpyinfo
 - xdriinfo
 - xedit
 - xev
 - xeyes
 - xfd
 - xfontsel
 - xgamma
 - xgc
 - xhost
 - xinit
 - xinput
 - xkbbell
 - xkbcomp
 - xkbevd
 - xkbprint
 - xkbvleds
 - xkbwatch
 - xkeystone
 - xkill
 - xload
 - xlogo
 - xlsatoms
 - xlsclients
 - xlsfonts
 - xmag
 - xman
 - xmessage
 - xmodmap
 - xmore
 - xprop
 - xrandr
 - xrdb
 - xrefresh
 - xsAnnotate.r
 - xset
 - xsetmode
 - xsetpointer
 - xsetroot
 - xsetwacom
 - xsm
 - xstdcmap
 - xterm
 - xvidtune
 - xvinfo
 - xwd
 - xwininfo
 - xwud

This container was pulled from:

	https://hub.docker.com/r/biocontainers/ecomet

If you encounter errors in ecomet or need help running the
tools it contains, please contact the developer at

	https://hub.docker.com/r/biocontainers/ecomet

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 29,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*henomenal.*v.000000001.000000001.*_.*cv.000000000.000000003.000000039.*zfinal",
          wV = "*ctr.*henomenal.*v.000000001.000000001.*_.*cv.000000000.000000003.000000039.*zfinal",
          whatis = {
            "Name: ecomet", "Version: ctr-phenomenal-v1.1_cv0.3.39", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecomet package", "URL: https://hub.docker.com/r/biocontainers/ecomet",
          },
        },
      },
    },
    ecopcr = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ecopcr/ctr-0.5.0--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopcr package",
          Name = "ecopcr",
          URL = "https://quay.io/repository/biocontainers/ecopcr",
          Version = "ctr-0.5.0--py27_0",
          canonical = "ctr-0.5.0--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopcr/ctr-0.5.0--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopcr:0.5.0--py27_0, which exposes the
following programs:

 - ecoPCR
 - ecoPCRFilter.py
 - ecoPCRFormat.py
 - ecoSort.py
 - ecofind
 - ecogrep

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopcr

If you encounter errors in ecopcr or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopcr

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000005.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000005.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: ecopcr", "Version: ctr-0.5.0--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopcr package", "URL: https://quay.io/repository/biocontainers/ecopcr",
          },
        },
        ["ecopcr/ctr-0.5.0--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopcr package",
          Name = "ecopcr",
          URL = "https://quay.io/repository/biocontainers/ecopcr",
          Version = "ctr-0.5.0--py27_1",
          canonical = "ctr-0.5.0--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopcr/ctr-0.5.0--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopcr:0.5.0--py27_1, which exposes the
following programs:

 - createfontdatachunk.py
 - ecoPCR
 - ecoPCRFilter.py
 - ecoPCRFormat.py
 - ecoSort.py
 - ecofind
 - ecogrep
 - enhancer.py
 - explode.py
 - gifmaker.py
 - painter.py
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - thresholder.py
 - viewer.py

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopcr

If you encounter errors in ecopcr or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopcr

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000005.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000005.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: ecopcr", "Version: ctr-0.5.0--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopcr package", "URL: https://quay.io/repository/biocontainers/ecopcr",
          },
        },
        ["ecopcr/ctr-v0.5.0dfsg-1-deb_cv1"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopcr package",
          Name = "ecopcr",
          URL = "https://hub.docker.com/r/biocontainers/ecopcr",
          Version = "ctr-v0.5.0dfsg-1-deb_cv1",
          canonical = "ctr-v0.5.0dfsg-1-deb_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopcr/ctr-v0.5.0dfsg-1-deb_cv1.lua",
          help = [[This is a module file for the container biocontainers/ecopcr:v0.5.0dfsg-1-deb_cv1, which exposes the
following programs:

 - createfontdatachunk
 - dh_pypy
 - dh_python3
 - ecoPCR
 - ecoPCRFilter
 - ecoPCRFormat
 - ecoSort
 - ecofind
 - ecogrep
 - ecoisundertaxon
 - enhancer
 - explode
 - gifmaker
 - painter
 - pdb3
 - pdb3.5
 - pilconvert
 - pildriver
 - pilfile
 - pilfont
 - pilprint
 - player
 - py3clean
 - py3compile
 - py3versions
 - pybuild
 - pygettext3
 - pygettext3.5
 - python3m
 - thresholder
 - viewer

This container was pulled from:

	https://hub.docker.com/r/biocontainers/ecopcr

If you encounter errors in ecopcr or need help running the
tools it contains, please contact the developer at

	https://hub.docker.com/r/biocontainers/ecopcr

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 25,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000000.000000005.*dfsg.*zfinal-.000000001.*deb.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000000.000000005.*dfsg.*zfinal-.000000001.*deb.*_.*cv.000000001.*zfinal",
          whatis = {
            "Name: ecopcr", "Version: ctr-v0.5.0dfsg-1-deb_cv1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopcr package", "URL: https://hub.docker.com/r/biocontainers/ecopcr",
          },
        },
      },
    },
    ecoprimers = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ecoprimers/ctr-1.0--0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecoprimers package",
          Name = "ecoprimers",
          URL = "https://quay.io/repository/biocontainers/ecoprimers",
          Version = "ctr-1.0--0",
          canonical = "ctr-1.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecoprimers/ctr-1.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecoprimers:1.0--0, which exposes the
following programs:

 - ecoPrimers

This container was pulled from:

	https://quay.io/repository/biocontainers/ecoprimers

If you encounter errors in ecoprimers or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecoprimers

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: ecoprimers", "Version: ctr-1.0--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecoprimers package", "URL: https://quay.io/repository/biocontainers/ecoprimers",
          },
        },
        ["ecoprimers/ctr-1.0--1"] = {
          Category = "['Bioinformatics']",
          Description = "The ecoprimers package",
          Name = "ecoprimers",
          URL = "https://quay.io/repository/biocontainers/ecoprimers",
          Version = "ctr-1.0--1",
          canonical = "ctr-1.0--1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecoprimers/ctr-1.0--1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecoprimers:1.0--1, which exposes the
following programs:

 - ecoPrimers

This container was pulled from:

	https://quay.io/repository/biocontainers/ecoprimers

If you encounter errors in ecoprimers or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecoprimers

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: ecoprimers", "Version: ctr-1.0--1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecoprimers package", "URL: https://quay.io/repository/biocontainers/ecoprimers",
          },
        },
        ["ecoprimers/ctr-1.0--ha92aebf_2"] = {
          Category = "['Bioinformatics']",
          Description = "The ecoprimers package",
          Name = "ecoprimers",
          URL = "https://quay.io/repository/biocontainers/ecoprimers",
          Version = "ctr-1.0--ha92aebf_2",
          canonical = "ctr-1.0--ha92aebf_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecoprimers/ctr-1.0--ha92aebf_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecoprimers:1.0--ha92aebf_2, which exposes the
following programs:

 - ecoPrimers

This container was pulled from:

	https://quay.io/repository/biocontainers/ecoprimers

If you encounter errors in ecoprimers or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecoprimers

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*ha.000000092.*aebf.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*ha.000000092.*aebf.*_.000000002.*zfinal",
          whatis = {
            "Name: ecoprimers", "Version: ctr-1.0--ha92aebf_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecoprimers package", "URL: https://quay.io/repository/biocontainers/ecoprimers",
          },
        },
      },
    },
    ecopy = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ecopy/ctr-0.1.2--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopy package",
          Name = "ecopy",
          URL = "https://quay.io/repository/biocontainers/ecopy",
          Version = "ctr-0.1.2--py27_0",
          canonical = "ctr-0.1.2--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopy/ctr-0.1.2--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopy:0.1.2--py27_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - cygdb
 - cython
 - cythonize
 - dbus-launch
 - designer
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - insserv
 - install_packages
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopy

If you encounter errors in ecopy or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopy

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000002.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000002.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: ecopy", "Version: ctr-0.1.2--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopy package", "URL: https://quay.io/repository/biocontainers/ecopy",
          },
        },
        ["ecopy/ctr-0.1.2--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopy package",
          Name = "ecopy",
          URL = "https://quay.io/repository/biocontainers/ecopy",
          Version = "ctr-0.1.2--py35_0",
          canonical = "ctr-0.1.2--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopy/ctr-0.1.2--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopy:0.1.2--py35_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - cygdb
 - cython
 - cythonize
 - dbus-launch
 - designer
 - easy_install-3.5
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - idle3.5
 - insserv
 - install_packages
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - python3.5-config
 - python3.5m-config
 - pyuic5
 - pyvenv-3.5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopy

If you encounter errors in ecopy or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopy

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000002.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000002.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: ecopy", "Version: ctr-0.1.2--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopy package", "URL: https://quay.io/repository/biocontainers/ecopy",
          },
        },
        ["ecopy/ctr-0.1.2--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopy package",
          Name = "ecopy",
          URL = "https://quay.io/repository/biocontainers/ecopy",
          Version = "ctr-0.1.2--py36_0",
          canonical = "ctr-0.1.2--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopy/ctr-0.1.2--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopy:0.1.2--py36_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - cygdb
 - cython
 - cythonize
 - dbus-launch
 - designer
 - easy_install-3.6
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - insserv
 - install_packages
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopy

If you encounter errors in ecopy or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopy

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000002.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000002.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: ecopy", "Version: ctr-0.1.2--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopy package", "URL: https://quay.io/repository/biocontainers/ecopy",
          },
        },
        ["ecopy/ctr-0.1.2.2--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopy package",
          Name = "ecopy",
          URL = "https://quay.io/repository/biocontainers/ecopy",
          Version = "ctr-0.1.2.2--py27_0",
          canonical = "ctr-0.1.2.2--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopy/ctr-0.1.2.2--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopy:0.1.2.2--py27_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - cygdb
 - cython
 - cythonize
 - dbus-launch
 - designer
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopy

If you encounter errors in ecopy or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopy

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: ecopy", "Version: ctr-0.1.2.2--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopy package", "URL: https://quay.io/repository/biocontainers/ecopy",
          },
        },
        ["ecopy/ctr-0.1.2.2--py27h470a237_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopy package",
          Name = "ecopy",
          URL = "https://quay.io/repository/biocontainers/ecopy",
          Version = "ctr-0.1.2.2--py27h470a237_1",
          canonical = "ctr-0.1.2.2--py27h470a237_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopy/ctr-0.1.2.2--py27h470a237_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopy:0.1.2.2--py27h470a237_1, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - cygdb
 - cython
 - cythonize
 - dbus-launch
 - designer
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopy

If you encounter errors in ecopy or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopy

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 28,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000027.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000027.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          whatis = {
            "Name: ecopy", "Version: ctr-0.1.2.2--py27h470a237_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopy package", "URL: https://quay.io/repository/biocontainers/ecopy",
          },
        },
        ["ecopy/ctr-0.1.2.2--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopy package",
          Name = "ecopy",
          URL = "https://quay.io/repository/biocontainers/ecopy",
          Version = "ctr-0.1.2.2--py35_0",
          canonical = "ctr-0.1.2.2--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopy/ctr-0.1.2.2--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopy:0.1.2.2--py35_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - cygdb
 - cython
 - cythonize
 - dbus-launch
 - designer
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - idle3.5
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - python3.5-config
 - python3.5m-config
 - pyuic5
 - pyvenv-3.5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopy

If you encounter errors in ecopy or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopy

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: ecopy", "Version: ctr-0.1.2.2--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopy package", "URL: https://quay.io/repository/biocontainers/ecopy",
          },
        },
        ["ecopy/ctr-0.1.2.2--py35h470a237_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopy package",
          Name = "ecopy",
          URL = "https://quay.io/repository/biocontainers/ecopy",
          Version = "ctr-0.1.2.2--py35h470a237_1",
          canonical = "ctr-0.1.2.2--py35h470a237_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopy/ctr-0.1.2.2--py35h470a237_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopy:0.1.2.2--py35h470a237_1, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - cygdb
 - cython
 - cythonize
 - dbus-launch
 - designer
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - idle3.5
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - python3.5-config
 - python3.5m-config
 - pyuic5
 - pyvenv-3.5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopy

If you encounter errors in ecopy or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopy

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 28,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000035.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000035.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          whatis = {
            "Name: ecopy", "Version: ctr-0.1.2.2--py35h470a237_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopy package", "URL: https://quay.io/repository/biocontainers/ecopy",
          },
        },
        ["ecopy/ctr-0.1.2.2--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopy package",
          Name = "ecopy",
          URL = "https://quay.io/repository/biocontainers/ecopy",
          Version = "ctr-0.1.2.2--py36_0",
          canonical = "ctr-0.1.2.2--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopy/ctr-0.1.2.2--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopy:0.1.2.2--py36_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - cygdb
 - cython
 - cythonize
 - dbus-launch
 - designer
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopy

If you encounter errors in ecopy or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopy

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: ecopy", "Version: ctr-0.1.2.2--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopy package", "URL: https://quay.io/repository/biocontainers/ecopy",
          },
        },
        ["ecopy/ctr-0.1.2.2--py36h470a237_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ecopy package",
          Name = "ecopy",
          URL = "https://quay.io/repository/biocontainers/ecopy",
          Version = "ctr-0.1.2.2--py36h470a237_1",
          canonical = "ctr-0.1.2.2--py36h470a237_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ecopy/ctr-0.1.2.2--py36h470a237_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ecopy:0.1.2.2--py36h470a237_1, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - cygdb
 - cython
 - cythonize
 - dbus-launch
 - designer
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/ecopy

If you encounter errors in ecopy or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ecopy

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 28,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000036.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000002.000000002.*y.000000036.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          whatis = {
            "Name: ecopy", "Version: ctr-0.1.2.2--py36h470a237_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ecopy package", "URL: https://quay.io/repository/biocontainers/ecopy",
          },
        },
      },
    },
    ectyper = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ectyper/ctr-0.1--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ectyper package",
          Name = "ectyper",
          URL = "https://quay.io/repository/biocontainers/ectyper",
          Version = "ctr-0.1--py35_0",
          canonical = "ctr-0.1--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ectyper/ctr-0.1--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ectyper:0.1--py35_0, which exposes the
following programs:

 - aalookup_unit_test
 - aascan_unit_test
 - align_format_unit_test
 - bcftools
 - bdbloader_unit_test
 - bl2seq_unit_test
 - blast_format_unit_test
 - blast_formatter
 - blast_services_unit_test
 - blast_unit_test
 - blastdb_aliastool
 - blastdb_format_unit_test
 - blastdbcheck
 - blastdbcmd
 - blastdbcp
 - blastdiag_unit_test
 - blastengine_unit_test
 - blastextend_unit_test
 - blastfilter_unit_test
 - blasthits_unit_test
 - blastinput_unit_test
 - blastn
 - blastoptions_unit_test
 - blastp
 - blastsetup_unit_test
 - blastx
 - bowtie2
 - bowtie2-align-l
 - bowtie2-align-s
 - bowtie2-build
 - bowtie2-build-l
 - bowtie2-build-s
 - bowtie2-inspect
 - bowtie2-inspect-l
 - bowtie2-inspect-s
 - color-chrs.pl
 - convert2blastmask
 - createfontdatachunk.py
 - datatool
 - delta_unit_test
 - deltablast
 - dustmasker
 - ectyper
 - enhancer.py
 - explode.py
 - gapinfo_unit_test
 - gencode_singleton_unit_test
 - gene_info_reader
 - gene_info_unit_test
 - gifmaker.py
 - guess-ploidy.py
 - hspfilter_besthit_unit_test
 - hspfilter_culling_unit_test
 - hspstream_unit_test
 - idle3.5
 - ksu
 - legacy_blast.pl
 - linkhsp_unit_test
 - magicblast_unit_test
 - makeblastdb
 - makembindex
 - makeprofiledb
 - mash
 - msa2pssm_unit_test
 - ntlookup_unit_test
 - ntscan_unit_test
 - optionshandle_unit_test
 - painter.py
 - perl5.22.0
 - phiblast_unit_test
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - plot-roh.py
 - plot-vcfstats
 - prelimsearch_unit_test
 - project_tree_builder
 - psibl2seq_unit_test
 - psiblast
 - psiblast_iteration_unit_test
 - psiblast_unit_test
 - pssmcreate_unit_test
 - pssmenginefreqratios_unit_test
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - querydata_unit_test
 - queryinfo_unit_test
 - redoalignment_unit_test
 - remote_blast_unit_test
 - rps_unit_test
 - rpsblast
 - rpstblastn
 - run-roh.pl
 - run_with_lock
 - sample
 - samtools
 - scoreblk_unit_test
 - search_strategy_unit_test
 - seedtop
 - segmasker
 - seqdb_demo
 - seqdb_perf
 - seqdb_unit_test
 - seqinfosrc_unit_test
 - seqsrc_unit_test
 - seqtk
 - setupfactory_unit_test
 - split_query_unit_test
 - stat_unit_test
 - subj_ranges_unit_test
 - tblastn
 - tblastx
 - test_pcre
 - thresholder.py
 - traceback_unit_test
 - tracebacksearch_unit_test
 - uniform_search_unit_test
 - update_blastdb.pl
 - vcfutils.pl
 - version_reference_unit_test
 - viewer.py
 - windowmasker
 - windowmasker_2.2.22_adapter.py
 - writedb_unit_test

This container was pulled from:

	https://quay.io/repository/biocontainers/ectyper

If you encounter errors in ectyper or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ectyper

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: ectyper", "Version: ctr-0.1--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ectyper package", "URL: https://quay.io/repository/biocontainers/ectyper",
          },
        },
        ["ectyper/ctr-0.1--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ectyper package",
          Name = "ectyper",
          URL = "https://quay.io/repository/biocontainers/ectyper",
          Version = "ctr-0.1--py36_0",
          canonical = "ctr-0.1--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ectyper/ctr-0.1--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ectyper:0.1--py36_0, which exposes the
following programs:

 - aalookup_unit_test
 - aascan_unit_test
 - align_format_unit_test
 - bcftools
 - bdbloader_unit_test
 - bl2seq_unit_test
 - blast_format_unit_test
 - blast_formatter
 - blast_services_unit_test
 - blast_unit_test
 - blastdb_aliastool
 - blastdb_format_unit_test
 - blastdbcheck
 - blastdbcmd
 - blastdbcp
 - blastdiag_unit_test
 - blastengine_unit_test
 - blastextend_unit_test
 - blastfilter_unit_test
 - blasthits_unit_test
 - blastinput_unit_test
 - blastn
 - blastoptions_unit_test
 - blastp
 - blastsetup_unit_test
 - blastx
 - bowtie2
 - bowtie2-align-l
 - bowtie2-align-s
 - bowtie2-build
 - bowtie2-build-l
 - bowtie2-build-s
 - bowtie2-inspect
 - bowtie2-inspect-l
 - bowtie2-inspect-s
 - color-chrs.pl
 - convert2blastmask
 - createfontdatachunk.py
 - datatool
 - delta_unit_test
 - deltablast
 - dustmasker
 - ectyper
 - enhancer.py
 - explode.py
 - gapinfo_unit_test
 - gencode_singleton_unit_test
 - gene_info_reader
 - gene_info_unit_test
 - gifmaker.py
 - guess-ploidy.py
 - hspfilter_besthit_unit_test
 - hspfilter_culling_unit_test
 - hspstream_unit_test
 - ksu
 - legacy_blast.pl
 - linkhsp_unit_test
 - magicblast_unit_test
 - makeblastdb
 - makembindex
 - makeprofiledb
 - mash
 - msa2pssm_unit_test
 - ntlookup_unit_test
 - ntscan_unit_test
 - optionshandle_unit_test
 - painter.py
 - perl5.22.0
 - phiblast_unit_test
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - plot-roh.py
 - plot-vcfstats
 - prelimsearch_unit_test
 - project_tree_builder
 - psibl2seq_unit_test
 - psiblast
 - psiblast_iteration_unit_test
 - psiblast_unit_test
 - pssmcreate_unit_test
 - pssmenginefreqratios_unit_test
 - querydata_unit_test
 - queryinfo_unit_test
 - redoalignment_unit_test
 - remote_blast_unit_test
 - rps_unit_test
 - rpsblast
 - rpstblastn
 - run-roh.pl
 - run_with_lock
 - samtools
 - scoreblk_unit_test
 - search_strategy_unit_test
 - seedtop
 - segmasker
 - seqdb_demo
 - seqdb_perf
 - seqdb_unit_test
 - seqinfosrc_unit_test
 - seqsrc_unit_test
 - seqtk
 - setupfactory_unit_test
 - split_query_unit_test
 - stat_unit_test
 - subj_ranges_unit_test
 - tblastn
 - tblastx
 - test_pcre
 - thresholder.py
 - traceback_unit_test
 - tracebacksearch_unit_test
 - uniform_search_unit_test
 - update_blastdb.pl
 - vcfutils.pl
 - version_reference_unit_test
 - viewer.py
 - windowmasker
 - windowmasker_2.2.22_adapter.py
 - writedb_unit_test

This container was pulled from:

	https://quay.io/repository/biocontainers/ectyper

If you encounter errors in ectyper or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ectyper

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: ectyper", "Version: ctr-0.1--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ectyper package", "URL: https://quay.io/repository/biocontainers/ectyper",
          },
        },
        ["ectyper/ctr-0.5.4--py35_2"] = {
          Category = "['Bioinformatics']",
          Description = "The ectyper package",
          Name = "ectyper",
          URL = "https://quay.io/repository/biocontainers/ectyper",
          Version = "ctr-0.5.4--py35_2",
          canonical = "ctr-0.5.4--py35_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ectyper/ctr-0.5.4--py35_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ectyper:0.5.4--py35_2, which exposes the
following programs:

 - bcftools
 - blast_formatter
 - blastdb_aliastool
 - blastdbcheck
 - blastdbcmd
 - blastdbcp
 - blastn
 - blastp
 - blastx
 - bowtie2
 - bowtie2-align-l
 - bowtie2-align-s
 - bowtie2-build
 - bowtie2-build-l
 - bowtie2-build-s
 - bowtie2-inspect
 - bowtie2-inspect-l
 - bowtie2-inspect-s
 - capnp
 - capnpc
 - capnpc-c++
 - capnpc-capnp
 - certtool
 - color-chrs.pl
 - convert2blastmask
 - cpanm
 - datatool
 - deltablast
 - dustmasker
 - ectyper
 - gene_info_reader
 - gnutls-cli
 - gnutls-cli-debug
 - gnutls-serv
 - guess-ploidy.py
 - idle3.5
 - legacy_blast.pl
 - makeblastdb
 - makembindex
 - makeprofiledb
 - mash
 - nettle-hash
 - nettle-lfib-stream
 - nettle-pbkdf2
 - ocsptool
 - perl5.22.0
 - pkcs1-conv
 - plot-roh.py
 - plot-vcfstats
 - project_tree_builder
 - psiblast
 - psktool
 - py.test
 - pytest
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - rpsblast
 - rpstblastn
 - run-roh.pl
 - run_with_lock
 - sample
 - samtools
 - seedtop
 - segmasker
 - seqdb_demo
 - seqdb_perf
 - seqtk
 - sexp-conv
 - srptool
 - tblastn
 - tblastx
 - update_blastdb.pl
 - update_blastdb.pl.bak
 - vcfutils.pl
 - windowmasker
 - windowmasker_2.2.22_adapter.py

This container was pulled from:

	https://quay.io/repository/biocontainers/ectyper

If you encounter errors in ectyper or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ectyper

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000005.000000004.*y.000000035.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000005.000000004.*y.000000035.*_.000000002.*zfinal",
          whatis = {
            "Name: ectyper", "Version: ctr-0.5.4--py35_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ectyper package", "URL: https://quay.io/repository/biocontainers/ectyper",
          },
        },
        ["ectyper/ctr-0.8.1--py_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ectyper package",
          Name = "ectyper",
          URL = "https://quay.io/repository/biocontainers/ectyper",
          Version = "ctr-0.8.1--py_0",
          canonical = "ctr-0.8.1--py_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ectyper/ctr-0.8.1--py_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ectyper:0.8.1--py_0, which exposes the
following programs:

 - bcftools
 - blast_formatter
 - blastdb_aliastool
 - blastdbcheck
 - blastdbcmd
 - blastdbcp
 - blastn
 - blastp
 - blastx
 - bowtie2
 - bowtie2-align-l
 - bowtie2-align-s
 - bowtie2-build
 - bowtie2-build-l
 - bowtie2-build-s
 - bowtie2-inspect
 - bowtie2-inspect-l
 - bowtie2-inspect-s
 - capnp
 - capnpc
 - capnpc-c++
 - capnpc-capnp
 - certtool
 - color-chrs.pl
 - conv-template
 - convert2blastmask
 - datatool
 - deltablast
 - dustmasker
 - ectyper
 - from-template
 - gene_info_reader
 - gnutls-cli
 - gnutls-cli-debug
 - gnutls-serv
 - guess-ploidy.py
 - legacy_blast.pl
 - makeblastdb
 - makembindex
 - makeprofiledb
 - mash
 - ncurses6-config
 - nettle-hash
 - nettle-lfib-stream
 - nettle-pbkdf2
 - ocsptool
 - perl5.26.2
 - pkcs1-conv
 - plot-roh.py
 - plot-vcfstats
 - project_tree_builder
 - psiblast
 - psktool
 - py.test
 - pytest
 - rpsblast
 - rpstblastn
 - run-roh.pl
 - run_with_lock
 - samtools
 - seedtop
 - segmasker
 - seqdb_demo
 - seqdb_perf
 - seqtk
 - sexp-conv
 - srptool
 - tblastn
 - tblastx
 - update_blastdb.pl
 - update_blastdb.pl.bak
 - vcfutils.pl
 - windowmasker
 - windowmasker_2.2.22_adapter.py

This container was pulled from:

	https://quay.io/repository/biocontainers/ectyper

If you encounter errors in ectyper or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ectyper

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000008.000000001.*y.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000008.000000001.*y.*_.*zfinal",
          whatis = {
            "Name: ectyper", "Version: ctr-0.8.1--py_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ectyper package", "URL: https://quay.io/repository/biocontainers/ectyper",
          },
        },
      },
    },
    edd = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["edd/ctr-1.1.18--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The edd package",
          Name = "edd",
          URL = "https://quay.io/repository/biocontainers/edd",
          Version = "ctr-1.1.18--py27_0",
          canonical = "ctr-1.1.18--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edd/ctr-1.1.18--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edd:1.1.18--py27_0, which exposes the
following programs:

 - annotate.py
 - annotateBed
 - bamToBed
 - bamToFastq
 - bcftools
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - closestBed
 - clusterBed
 - color-chrs.pl
 - complementBed
 - coverageBed
 - edd
 - expandCols
 - fastaFromBed
 - flankBed
 - genomeCoverageBed
 - getOverlap
 - groupBy
 - guess-ploidy.py
 - intersectBed
 - intersection_matrix.py
 - intron_exon_reads.py
 - ksu
 - linksBed
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - multiBamCov
 - multiIntersectBed
 - nucBed
 - pairToBed
 - pairToPair
 - pbt_plotting_example.py
 - peak_pie.py
 - plot-roh.py
 - plot-vcfstats
 - pybedtools
 - randomBed
 - run-roh.pl
 - samtools
 - shiftBed
 - shuffleBed
 - slopBed
 - sortBed
 - subtractBed
 - tagBam
 - unionBedGraphs
 - vcfutils.pl
 - venn_gchart.py
 - venn_mpl.py
 - windowBed
 - windowMaker

This container was pulled from:

	https://quay.io/repository/biocontainers/edd

If you encounter errors in edd or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edd

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.000000018.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.000000018.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: edd", "Version: ctr-1.1.18--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edd package", "URL: https://quay.io/repository/biocontainers/edd",
          },
        },
        ["edd/ctr-1.1.18--py27h24bf2e0_1"] = {
          Category = "['Bioinformatics']",
          Description = "The edd package",
          Name = "edd",
          URL = "https://quay.io/repository/biocontainers/edd",
          Version = "ctr-1.1.18--py27h24bf2e0_1",
          canonical = "ctr-1.1.18--py27h24bf2e0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edd/ctr-1.1.18--py27h24bf2e0_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edd:1.1.18--py27h24bf2e0_1, which exposes the
following programs:

 - annotate.py
 - annotateBed
 - bamToBed
 - bamToFastq
 - bcftools
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - closestBed
 - clusterBed
 - color-chrs.pl
 - complementBed
 - coverageBed
 - edd
 - expandCols
 - fastaFromBed
 - flankBed
 - genomeCoverageBed
 - getOverlap
 - groupBy
 - guess-ploidy.py
 - intersectBed
 - intersection_matrix.py
 - intron_exon_reads.py
 - linksBed
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - multiBamCov
 - multiIntersectBed
 - nucBed
 - pairToBed
 - pairToPair
 - pbt_plotting_example.py
 - peak_pie.py
 - plot-roh.py
 - plot-vcfstats
 - pybedtools
 - randomBed
 - run-roh.pl
 - samtools
 - shiftBed
 - shuffleBed
 - slopBed
 - sortBed
 - subtractBed
 - tagBam
 - unionBedGraphs
 - vcfutils.pl
 - venn_gchart.py
 - venn_mpl.py
 - windowBed
 - windowMaker

This container was pulled from:

	https://quay.io/repository/biocontainers/edd

If you encounter errors in edd or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edd

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.000000018.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.000000018.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          whatis = {
            "Name: edd", "Version: ctr-1.1.18--py27h24bf2e0_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edd package", "URL: https://quay.io/repository/biocontainers/edd",
          },
        },
      },
    },
    eden = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["eden/ctr-1.1--hd28b015_0"] = {
          Category = "['Bioinformatics']",
          Description = "The eden package",
          Name = "eden",
          URL = "https://quay.io/repository/biocontainers/eden",
          Version = "ctr-1.1--hd28b015_0",
          canonical = "ctr-1.1--hd28b015_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eden/ctr-1.1--hd28b015_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eden:1.1--hd28b015_0, which exposes the
following programs:

 - EDeN

This container was pulled from:

	https://quay.io/repository/biocontainers/eden

If you encounter errors in eden or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eden

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.*hd.000000028.*b.000000015.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.*hd.000000028.*b.000000015.*_.*zfinal",
          whatis = {
            "Name: eden", "Version: ctr-1.1--hd28b015_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eden package", "URL: https://quay.io/repository/biocontainers/eden",
          },
        },
        ["eden/ctr-2.0--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The eden package",
          Name = "eden",
          URL = "https://quay.io/repository/biocontainers/eden",
          Version = "ctr-2.0--py27_0",
          canonical = "ctr-2.0--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eden/ctr-2.0--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eden:2.0--py27_0, which exposes the
following programs:

 - RNAshapes
 - annotate.py
 - assistant
 - babel
 - bcftools
 - bmp2tiff
 - color-chrs.pl
 - createfontdatachunk.py
 - designer
 - enhancer.py
 - explode.py
 - get_objgraph.py
 - gif2tiff
 - gifmaker.py
 - intersection_matrix.py
 - intron_exon_reads.py
 - lconvert
 - linguist
 - location_predictor
 - lrelease
 - lupdate
 - moc
 - model
 - motif
 - motif_display.py
 - muscle
 - obabel
 - obchiral
 - obconformer
 - obdistgen
 - obenergy
 - obfit
 - obgen
 - obgrep
 - obminimize
 - obprobe
 - obprop
 - obrms
 - obrotamer
 - obrotate
 - obspectrophore
 - obsym
 - obtautomer
 - obthermo
 - painter.py
 - pbt_plotting_example.py
 - peak_pie.py
 - perl5.22.0
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - pixeltool
 - player.py
 - plot-vcfstats
 - pybedtools
 - pylupdate4
 - pyrcc4
 - pyuic4
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc3
 - qhelpconverter
 - qhelpgenerator
 - qmake
 - qmlplugindump
 - qmlviewer
 - qt3to4
 - qtconfig
 - qttracereplay
 - ras2tiff
 - rcc
 - rgb2ycbcr
 - roundtrip
 - sample
 - samtools
 - seqlogo
 - sip
 - thresholder.py
 - thumbnail
 - uic
 - unpickle.py
 - vcfutils.pl
 - venn_gchart.py
 - venn_mpl.py
 - viewer.py
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/eden

If you encounter errors in eden or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eden

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000002.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: eden", "Version: ctr-2.0--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eden package", "URL: https://quay.io/repository/biocontainers/eden",
          },
        },
        ["eden/ctr-2.0--py27h24bf2e0_1"] = {
          Category = "['Bioinformatics']",
          Description = "The eden package",
          Name = "eden",
          URL = "https://quay.io/repository/biocontainers/eden",
          Version = "ctr-2.0--py27h24bf2e0_1",
          canonical = "ctr-2.0--py27h24bf2e0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eden/ctr-2.0--py27h24bf2e0_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eden:2.0--py27h24bf2e0_1, which exposes the
following programs:

 - .dbus-post-link.sh
 - RNAshapes
 - annotate.py
 - annotateBed
 - assistant
 - babel
 - bamToBed
 - bamToFastq
 - bcftools
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - closestBed
 - clusterBed
 - color-chrs.pl
 - complementBed
 - coverageBed
 - createfontdatachunk.py
 - dbus-launch
 - designer
 - dvipdf
 - enhancer.py
 - eps2eps
 - expandCols
 - explode.py
 - fastaFromBed
 - fixqt4headers.pl
 - flankBed
 - font2c
 - genomeCoverageBed
 - getOverlap
 - get_objgraph
 - gifmaker.py
 - groupBy
 - gs
 - gsbj
 - gsdj
 - gsdj500
 - gslj
 - gslp
 - gsnd
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - guess-ploidy.py
 - intersectBed
 - intersection_matrix.py
 - intron_exon_reads.py
 - lconvert
 - linguist
 - linksBed
 - location_predictor
 - lprsetup.sh
 - lrelease
 - lupdate
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - moc
 - model
 - motif
 - motif_display.py
 - multiBamCov
 - multiIntersectBed
 - muscle
 - nucBed
 - obabel
 - obchiral
 - obconformer
 - obdistgen
 - obenergy
 - obfit
 - obgen
 - obgrep
 - obminimize
 - obprobe
 - obprop
 - obrms
 - obrotamer
 - obrotate
 - obspectrophore
 - obsym
 - obtautomer
 - obthermo
 - painter.py
 - pairToBed
 - pairToPair
 - pbt_plotting_example.py
 - pdf2dsc
 - pdf2ps
 - peak_pie.py
 - pf2afm
 - pfbtopfa
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - pixeltool
 - player.py
 - plot-roh.py
 - plot-vcfstats
 - pphs
 - printafm
 - ps2ascii
 - ps2epsi
 - ps2pdf
 - ps2pdf12
 - ps2pdf13
 - ps2pdf14
 - ps2pdfwr
 - ps2ps
 - ps2ps2
 - pybedtools
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - randomBed
 - rcc
 - roundtrip
 - run-roh.pl
 - sample
 - samtools
 - shiftBed
 - shuffleBed
 - sip
 - slopBed
 - sortBed
 - subtractBed
 - syncqt.pl
 - tagBam
 - thresholder.py
 - transformseq
 - uic
 - undill
 - unionBedGraphs
 - unix-lpr.sh
 - vcfutils.pl
 - venn_gchart.py
 - venn_mpl.py
 - viewer.py
 - weblogo
 - wftopfa
 - windowBed
 - windowMaker
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/eden

If you encounter errors in eden or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eden

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 24,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000002.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          whatis = {
            "Name: eden", "Version: ctr-2.0--py27h24bf2e0_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eden package", "URL: https://quay.io/repository/biocontainers/eden",
          },
        },
        ["eden/ctr-2.0--py27hfd84ecd_2"] = {
          Category = "['Bioinformatics']",
          Description = "The eden package",
          Name = "eden",
          URL = "https://quay.io/repository/biocontainers/eden",
          Version = "ctr-2.0--py27hfd84ecd_2",
          canonical = "ctr-2.0--py27hfd84ecd_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eden/ctr-2.0--py27hfd84ecd_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eden:2.0--py27hfd84ecd_2, which exposes the
following programs:

 - .dbus-post-link.sh
 - RNAshapes
 - acyclic
 - annotate.py
 - annotateBed
 - assistant
 - babel
 - bamToBed
 - bamToFastq
 - bcftools
 - bcomps
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - ccomps
 - circo
 - closestBed
 - cluster
 - clusterBed
 - color-chrs.pl
 - complementBed
 - coverageBed
 - dbus-launch
 - designer
 - dijkstra
 - dot
 - dot2gxl
 - dot_builtins
 - dvipdf
 - edgepaint
 - eps2eps
 - expandCols
 - fastaFromBed
 - fc-conflist
 - fdp
 - fixqt4headers.pl
 - flankBed
 - font2c
 - gc
 - genomeCoverageBed
 - getOverlap
 - get_objgraph
 - gml2gv
 - graphml2gv
 - groupBy
 - gs
 - gsbj
 - gsdj
 - gsdj500
 - gslj
 - gslp
 - gsnd
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - guess-ploidy.py
 - gv2gml
 - gv2gxl
 - gvcolor
 - gvgen
 - gvmap
 - gvmap.sh
 - gvpack
 - gvpr
 - gxl2dot
 - gxl2gv
 - hb-subset
 - intersectBed
 - intersection_matrix.py
 - intron_exon_reads.py
 - lconvert
 - libtool
 - libtoolize
 - linguist
 - linksBed
 - location_predictor
 - lprsetup.sh
 - lrelease
 - lupdate
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - mm2gv
 - moc
 - model
 - motif
 - motif_display.py
 - multiBamCov
 - multiIntersectBed
 - muscle
 - ncurses6-config
 - neato
 - nop
 - nucBed
 - obabel
 - obchiral
 - obconformer
 - obdistgen
 - obenergy
 - obfit
 - obgen
 - obgrep
 - obminimize
 - obprobe
 - obprop
 - obrms
 - obrotamer
 - obrotate
 - obspectrophore
 - obsym
 - obtautomer
 - obthermo
 - osage
 - pairToBed
 - pairToPair
 - patchwork
 - pbt_plotting_example.py
 - pdf2dsc
 - pdf2ps
 - peak_pie.py
 - pf2afm
 - pfbtopfa
 - pixeltool
 - plot-roh.py
 - plot-vcfstats
 - pphs
 - printafm
 - prune
 - ps2ascii
 - ps2epsi
 - ps2pdf
 - ps2pdf12
 - ps2pdf13
 - ps2pdf14
 - ps2pdfwr
 - ps2ps
 - ps2ps2
 - pybedtools
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - randomBed
 - rcc
 - roundtrip
 - run-roh.pl
 - sample
 - samtools
 - sccmap
 - sfdp
 - shiftBed
 - shuffleBed
 - sip
 - slopBed
 - sortBed
 - subtractBed
 - syncqt.pl
 - tagBam
 - transformseq
 - tred
 - twopi
 - uic
 - undill
 - unflatten
 - unionBedGraphs
 - unix-lpr.sh
 - vcfutils.pl
 - venn_gchart.py
 - venn_mpl.py
 - weblogo
 - wftopfa
 - windowBed
 - windowMaker
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/eden

If you encounter errors in eden or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eden

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 24,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.*y.000000027.*hfd.000000084.*ecd.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000002.*y.000000027.*hfd.000000084.*ecd.*_.000000002.*zfinal",
          whatis = {
            "Name: eden", "Version: ctr-2.0--py27hfd84ecd_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eden package", "URL: https://quay.io/repository/biocontainers/eden",
          },
        },
      },
    },
    edena = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["edena/ctr-3.131028--0"] = {
          Category = "['De-novo assembly', 'Data handling']",
          Description = "A program for assembling de novo bacterial genomes from short reads produced by Illumina sequencing platform.",
          Name = "edena",
          URL = "https://quay.io/repository/biocontainers/edena",
          Version = "ctr-3.131028--0",
          canonical = "ctr-3.131028--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edena/ctr-3.131028--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edena:3.131028--0, which exposes the
following programs:

 - edena

This container was pulled from:

	https://quay.io/repository/biocontainers/edena

If you encounter errors in edena or need help running the
tools it contains, please contact the developer at

	http://www.genomic.ch/edena

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000131028.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000131028.*zfinal",
          whatis = {
            "Name: edena", "Version: ctr-3.131028--0", "Category: ['De-novo assembly', 'Data handling']", "Keywords: ['Genomics', 'Microbiology', 'Sequence assembly']", "Description: A program for assembling de novo bacterial genomes from short reads produced by Illumina sequencing platform.", "URL: https://quay.io/repository/biocontainers/edena",
          },
        },
        ["edena/ctr-3.131028--h2d50403_1"] = {
          Category = "['De-novo assembly', 'Data handling']",
          Description = "A program for assembling de novo bacterial genomes from short reads produced by Illumina sequencing platform.",
          Name = "edena",
          URL = "https://quay.io/repository/biocontainers/edena",
          Version = "ctr-3.131028--h2d50403_1",
          canonical = "ctr-3.131028--h2d50403_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edena/ctr-3.131028--h2d50403_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edena:3.131028--h2d50403_1, which exposes the
following programs:

 - edena

This container was pulled from:

	https://quay.io/repository/biocontainers/edena

If you encounter errors in edena or need help running the
tools it contains, please contact the developer at

	http://www.genomic.ch/edena

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 25,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000131028.*h.000000002.*d.000050403.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000131028.*h.000000002.*d.000050403.*_.000000001.*zfinal",
          whatis = {
            "Name: edena", "Version: ctr-3.131028--h2d50403_1", "Category: ['De-novo assembly', 'Data handling']", "Keywords: ['Genomics', 'Microbiology', 'Sequence assembly']", "Description: A program for assembling de novo bacterial genomes from short reads produced by Illumina sequencing platform.", "URL: https://quay.io/repository/biocontainers/edena",
          },
        },
      },
    },
    edfbrowser = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["edfbrowser/ctr-v1.58-1b1-deb_cv1"] = {
          Category = "['Bioinformatics']",
          Description = "The edfbrowser package",
          Name = "edfbrowser",
          URL = "https://hub.docker.com/r/biocontainers/edfbrowser",
          Version = "ctr-v1.58-1b1-deb_cv1",
          canonical = "ctr-v1.58-1b1-deb_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edfbrowser/ctr-v1.58-1b1-deb_cv1.lua",
          help = [[This is a module file for the container biocontainers/edfbrowser:v1.58-1b1-deb_cv1, which exposes the
following programs:

 - assistant
 - designer
 - edfbrowser
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - pixeltool
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qdoc3
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qml1plugindump
 - qmlbundle
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qmlviewer
 - qtchooser
 - qtconfig
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - uic
 - uic3
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://hub.docker.com/r/biocontainers/edfbrowser

If you encounter errors in edfbrowser or need help running the
tools it contains, please contact the developer at

	https://hub.docker.com/r/biocontainers/edfbrowser

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000001.000000058.*zfinal-.000000001.*b.000000001.*deb.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000001.000000058.*zfinal-.000000001.*b.000000001.*deb.*_.*cv.000000001.*zfinal",
          whatis = {
            "Name: edfbrowser", "Version: ctr-v1.58-1b1-deb_cv1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edfbrowser package", "URL: https://hub.docker.com/r/biocontainers/edfbrowser",
          },
        },
      },
    },
    editdistance = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["editdistance/ctr-0.3.1--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The editdistance package",
          Name = "editdistance",
          URL = "https://quay.io/repository/biocontainers/editdistance",
          Version = "ctr-0.3.1--py27_0",
          canonical = "ctr-0.3.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/editdistance/ctr-0.3.1--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/editdistance:0.3.1--py27_0, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/editdistance

If you encounter errors in editdistance or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/editdistance

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: editdistance", "Version: ctr-0.3.1--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The editdistance package", "URL: https://quay.io/repository/biocontainers/editdistance",
          },
        },
        ["editdistance/ctr-0.3.1--py34_0"] = {
          Category = "['Bioinformatics']",
          Description = "The editdistance package",
          Name = "editdistance",
          URL = "https://quay.io/repository/biocontainers/editdistance",
          Version = "ctr-0.3.1--py34_0",
          canonical = "ctr-0.3.1--py34_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/editdistance/ctr-0.3.1--py34_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/editdistance:0.3.1--py34_0, which exposes the
following programs:

 - 2to3-3.4
 - easy_install-3.4
 - idle3.4
 - pydoc3.4
 - python3.4
 - python3.4-config
 - python3.4m
 - python3.4m-config
 - pyvenv-3.4

This container was pulled from:

	https://quay.io/repository/biocontainers/editdistance

If you encounter errors in editdistance or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/editdistance

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000034.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000034.*_.*zfinal",
          whatis = {
            "Name: editdistance", "Version: ctr-0.3.1--py34_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The editdistance package", "URL: https://quay.io/repository/biocontainers/editdistance",
          },
        },
        ["editdistance/ctr-0.3.1--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The editdistance package",
          Name = "editdistance",
          URL = "https://quay.io/repository/biocontainers/editdistance",
          Version = "ctr-0.3.1--py35_0",
          canonical = "ctr-0.3.1--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/editdistance/ctr-0.3.1--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/editdistance:0.3.1--py35_0, which exposes the
following programs:

 - easy_install-3.5
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/editdistance

If you encounter errors in editdistance or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/editdistance

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: editdistance", "Version: ctr-0.3.1--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The editdistance package", "URL: https://quay.io/repository/biocontainers/editdistance",
          },
        },
        ["editdistance/ctr-0.3.1--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The editdistance package",
          Name = "editdistance",
          URL = "https://quay.io/repository/biocontainers/editdistance",
          Version = "ctr-0.3.1--py36_0",
          canonical = "ctr-0.3.1--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/editdistance/ctr-0.3.1--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/editdistance:0.3.1--py36_0, which exposes the
following programs:

 - easy_install-3.6

This container was pulled from:

	https://quay.io/repository/biocontainers/editdistance

If you encounter errors in editdistance or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/editdistance

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: editdistance", "Version: ctr-0.3.1--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The editdistance package", "URL: https://quay.io/repository/biocontainers/editdistance",
          },
        },
        ["editdistance/ctr-0.4--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The editdistance package",
          Name = "editdistance",
          URL = "https://quay.io/repository/biocontainers/editdistance",
          Version = "ctr-0.4--py27_0",
          canonical = "ctr-0.4--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/editdistance/ctr-0.4--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/editdistance:0.4--py27_0, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/editdistance

If you encounter errors in editdistance or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/editdistance

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000004.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000004.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: editdistance", "Version: ctr-0.4--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The editdistance package", "URL: https://quay.io/repository/biocontainers/editdistance",
          },
        },
        ["editdistance/ctr-0.4--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The editdistance package",
          Name = "editdistance",
          URL = "https://quay.io/repository/biocontainers/editdistance",
          Version = "ctr-0.4--py35_0",
          canonical = "ctr-0.4--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/editdistance/ctr-0.4--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/editdistance:0.4--py35_0, which exposes the
following programs:

 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/editdistance

If you encounter errors in editdistance or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/editdistance

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000004.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000004.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: editdistance", "Version: ctr-0.4--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The editdistance package", "URL: https://quay.io/repository/biocontainers/editdistance",
          },
        },
        ["editdistance/ctr-0.4--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The editdistance package",
          Name = "editdistance",
          URL = "https://quay.io/repository/biocontainers/editdistance",
          Version = "ctr-0.4--py36_0",
          canonical = "ctr-0.4--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/editdistance/ctr-0.4--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/editdistance:0.4--py36_0, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/editdistance

If you encounter errors in editdistance or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/editdistance

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000004.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000004.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: editdistance", "Version: ctr-0.4--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The editdistance package", "URL: https://quay.io/repository/biocontainers/editdistance",
          },
        },
      },
    },
    edittag = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["edittag/ctr-1.1--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The edittag package",
          Name = "edittag",
          URL = "https://quay.io/repository/biocontainers/edittag",
          Version = "ctr-1.1--py27_0",
          canonical = "ctr-1.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edittag/ctr-1.1--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edittag:1.1--py27_0, which exposes the
following programs:

 - add_tags_to_adapters.py
 - add_tags_to_primers.py
 - design_edit_metric_tags.py
 - estimate_sequencing_error_effects.py
 - get_tag_flows_for_454.py
 - validate_edit_metric_tags.py

This container was pulled from:

	https://quay.io/repository/biocontainers/edittag

If you encounter errors in edittag or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edittag

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: edittag", "Version: ctr-1.1--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edittag package", "URL: https://quay.io/repository/biocontainers/edittag",
          },
        },
        ["edittag/ctr-1.1--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The edittag package",
          Name = "edittag",
          URL = "https://quay.io/repository/biocontainers/edittag",
          Version = "ctr-1.1--py27_1",
          canonical = "ctr-1.1--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edittag/ctr-1.1--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edittag:1.1--py27_1, which exposes the
following programs:

 - add_tags_to_adapters.py
 - add_tags_to_primers.py
 - design_edit_metric_tags.py
 - estimate_sequencing_error_effects.py
 - get_tag_flows_for_454.py
 - validate_edit_metric_tags.py

This container was pulled from:

	https://quay.io/repository/biocontainers/edittag

If you encounter errors in edittag or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edittag

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: edittag", "Version: ctr-1.1--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edittag package", "URL: https://quay.io/repository/biocontainers/edittag",
          },
        },
        ["edittag/ctr-1.1--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The edittag package",
          Name = "edittag",
          URL = "https://quay.io/repository/biocontainers/edittag",
          Version = "ctr-1.1--py35_0",
          canonical = "ctr-1.1--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edittag/ctr-1.1--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edittag:1.1--py35_0, which exposes the
following programs:

 - add_tags_to_adapters.py
 - add_tags_to_primers.py
 - design_edit_metric_tags.py
 - easy_install-3.5
 - estimate_sequencing_error_effects.py
 - get_tag_flows_for_454.py
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - validate_edit_metric_tags.py

This container was pulled from:

	https://quay.io/repository/biocontainers/edittag

If you encounter errors in edittag or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edittag

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: edittag", "Version: ctr-1.1--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edittag package", "URL: https://quay.io/repository/biocontainers/edittag",
          },
        },
        ["edittag/ctr-1.1--py35_1"] = {
          Category = "['Bioinformatics']",
          Description = "The edittag package",
          Name = "edittag",
          URL = "https://quay.io/repository/biocontainers/edittag",
          Version = "ctr-1.1--py35_1",
          canonical = "ctr-1.1--py35_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edittag/ctr-1.1--py35_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edittag:1.1--py35_1, which exposes the
following programs:

 - add_tags_to_adapters.py
 - add_tags_to_primers.py
 - design_edit_metric_tags.py
 - estimate_sequencing_error_effects.py
 - get_tag_flows_for_454.py
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - validate_edit_metric_tags.py

This container was pulled from:

	https://quay.io/repository/biocontainers/edittag

If you encounter errors in edittag or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edittag

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.*y.000000035.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.*y.000000035.*_.000000001.*zfinal",
          whatis = {
            "Name: edittag", "Version: ctr-1.1--py35_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edittag package", "URL: https://quay.io/repository/biocontainers/edittag",
          },
        },
        ["edittag/ctr-1.1--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The edittag package",
          Name = "edittag",
          URL = "https://quay.io/repository/biocontainers/edittag",
          Version = "ctr-1.1--py36_0",
          canonical = "ctr-1.1--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edittag/ctr-1.1--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edittag:1.1--py36_0, which exposes the
following programs:

 - add_tags_to_adapters.py
 - add_tags_to_primers.py
 - design_edit_metric_tags.py
 - easy_install-3.6
 - estimate_sequencing_error_effects.py
 - get_tag_flows_for_454.py
 - validate_edit_metric_tags.py

This container was pulled from:

	https://quay.io/repository/biocontainers/edittag

If you encounter errors in edittag or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edittag

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: edittag", "Version: ctr-1.1--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edittag package", "URL: https://quay.io/repository/biocontainers/edittag",
          },
        },
        ["edittag/ctr-1.1--py36_1"] = {
          Category = "['Bioinformatics']",
          Description = "The edittag package",
          Name = "edittag",
          URL = "https://quay.io/repository/biocontainers/edittag",
          Version = "ctr-1.1--py36_1",
          canonical = "ctr-1.1--py36_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edittag/ctr-1.1--py36_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edittag:1.1--py36_1, which exposes the
following programs:

 - add_tags_to_adapters.py
 - add_tags_to_primers.py
 - design_edit_metric_tags.py
 - estimate_sequencing_error_effects.py
 - get_tag_flows_for_454.py
 - validate_edit_metric_tags.py

This container was pulled from:

	https://quay.io/repository/biocontainers/edittag

If you encounter errors in edittag or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edittag

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.*y.000000036.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.*y.000000036.*_.000000001.*zfinal",
          whatis = {
            "Name: edittag", "Version: ctr-1.1--py36_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edittag package", "URL: https://quay.io/repository/biocontainers/edittag",
          },
        },
      },
    },
    edlib = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["edlib/ctr-1.0.0--0"] = {
          Category = "['Bioinformatics']",
          Description = "The edlib package",
          Name = "edlib",
          URL = "https://quay.io/repository/biocontainers/edlib",
          Version = "ctr-1.0.0--0",
          canonical = "ctr-1.0.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edlib/ctr-1.0.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edlib:1.0.0--0, which exposes the
following programs:

 - aligner

This container was pulled from:

	https://quay.io/repository/biocontainers/edlib

If you encounter errors in edlib or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edlib

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: edlib", "Version: ctr-1.0.0--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edlib package", "URL: https://quay.io/repository/biocontainers/edlib",
          },
        },
        ["edlib/ctr-1.1.2--0"] = {
          Category = "['Bioinformatics']",
          Description = "The edlib package",
          Name = "edlib",
          URL = "https://quay.io/repository/biocontainers/edlib",
          Version = "ctr-1.1.2--0",
          canonical = "ctr-1.1.2--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edlib/ctr-1.1.2--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edlib:1.1.2--0, which exposes the
following programs:

 - edlib-aligner

This container was pulled from:

	https://quay.io/repository/biocontainers/edlib

If you encounter errors in edlib or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edlib

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.000000002.*zfinal",
          whatis = {
            "Name: edlib", "Version: ctr-1.1.2--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edlib package", "URL: https://quay.io/repository/biocontainers/edlib",
          },
        },
        ["edlib/ctr-1.2.0--0"] = {
          Category = "['Bioinformatics']",
          Description = "The edlib package",
          Name = "edlib",
          URL = "https://quay.io/repository/biocontainers/edlib",
          Version = "ctr-1.2.0--0",
          canonical = "ctr-1.2.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edlib/ctr-1.2.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edlib:1.2.0--0, which exposes the
following programs:

 - edlib-aligner

This container was pulled from:

	https://quay.io/repository/biocontainers/edlib

If you encounter errors in edlib or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edlib

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000002.*zfinal",
          whatis = {
            "Name: edlib", "Version: ctr-1.2.0--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edlib package", "URL: https://quay.io/repository/biocontainers/edlib",
          },
        },
        ["edlib/ctr-1.2.1--0"] = {
          Category = "['Bioinformatics']",
          Description = "The edlib package",
          Name = "edlib",
          URL = "https://quay.io/repository/biocontainers/edlib",
          Version = "ctr-1.2.1--0",
          canonical = "ctr-1.2.1--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edlib/ctr-1.2.1--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edlib:1.2.1--0, which exposes the
following programs:

 - edlib-aligner

This container was pulled from:

	https://quay.io/repository/biocontainers/edlib

If you encounter errors in edlib or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edlib

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000002.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000002.000000001.*zfinal",
          whatis = {
            "Name: edlib", "Version: ctr-1.2.1--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edlib package", "URL: https://quay.io/repository/biocontainers/edlib",
          },
        },
        ["edlib/ctr-1.2.3--0"] = {
          Category = "['Bioinformatics']",
          Description = "The edlib package",
          Name = "edlib",
          URL = "https://quay.io/repository/biocontainers/edlib",
          Version = "ctr-1.2.3--0",
          canonical = "ctr-1.2.3--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edlib/ctr-1.2.3--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edlib:1.2.3--0, which exposes the
following programs:

 - edlib-aligner

This container was pulled from:

	https://quay.io/repository/biocontainers/edlib

If you encounter errors in edlib or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edlib

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000002.000000003.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000002.000000003.*zfinal",
          whatis = {
            "Name: edlib", "Version: ctr-1.2.3--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edlib package", "URL: https://quay.io/repository/biocontainers/edlib",
          },
        },
        ["edlib/ctr-1.2.3--h2d50403_1"] = {
          Category = "['Bioinformatics']",
          Description = "The edlib package",
          Name = "edlib",
          URL = "https://quay.io/repository/biocontainers/edlib",
          Version = "ctr-1.2.3--h2d50403_1",
          canonical = "ctr-1.2.3--h2d50403_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edlib/ctr-1.2.3--h2d50403_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/edlib:1.2.3--h2d50403_1, which exposes the
following programs:

 - edlib-aligner

This container was pulled from:

	https://quay.io/repository/biocontainers/edlib

If you encounter errors in edlib or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/edlib

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000002.000000003.*h.000000002.*d.000050403.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000002.000000003.*h.000000002.*d.000050403.*_.000000001.*zfinal",
          whatis = {
            "Name: edlib", "Version: ctr-1.2.3--h2d50403_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edlib package", "URL: https://quay.io/repository/biocontainers/edlib",
          },
        },
      },
    },
    edtsurf = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["edtsurf/ctr-v0.2009-4b1-deb_cv1"] = {
          Category = "['Bioinformatics']",
          Description = "The edtsurf package",
          Name = "edtsurf",
          URL = "https://hub.docker.com/r/biocontainers/edtsurf",
          Version = "ctr-v0.2009-4b1-deb_cv1",
          canonical = "ctr-v0.2009-4b1-deb_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/edtsurf/ctr-v0.2009-4b1-deb_cv1.lua",
          help = [[This is a module file for the container biocontainers/edtsurf:v0.2009-4b1-deb_cv1, which exposes the
following programs:

 - EDTSurf

This container was pulled from:

	https://hub.docker.com/r/biocontainers/edtsurf

If you encounter errors in edtsurf or need help running the
tools it contains, please contact the developer at

	https://hub.docker.com/r/biocontainers/edtsurf

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 24,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000000.000002009.*zfinal-.000000004.*b.000000001.*deb.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000000.000002009.*zfinal-.000000004.*b.000000001.*deb.*_.*cv.000000001.*zfinal",
          whatis = {
            "Name: edtsurf", "Version: ctr-v0.2009-4b1-deb_cv1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The edtsurf package", "URL: https://hub.docker.com/r/biocontainers/edtsurf",
          },
        },
      },
    },
    effectivet3 = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["effectivet3/ctr-1.0.1--py27_0"] = {
          Category = "['Sequence classification']",
          Description = "Prediction of putative Type-III secreted proteins.",
          Name = "effectivet3",
          URL = "https://quay.io/repository/biocontainers/effectivet3",
          Version = "ctr-1.0.1--py27_0",
          canonical = "ctr-1.0.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/effectivet3/ctr-1.0.1--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/effectivet3:1.0.1--py27_0, which exposes the
following programs:

 - appletviewer
 - effectivet3
 - extcheck
 - idlj
 - jar
 - jarsigner
 - java-rmi.cgi
 - javac
 - javadoc
 - javah
 - javap
 - jcmd
 - jconsole
 - jdb
 - jdeps
 - jhat
 - jinfo
 - jmap
 - jps
 - jrunscript
 - jsadebugd
 - jstack
 - jstat
 - jstatd
 - native2ascii
 - rmic
 - schemagen
 - serialver
 - wsgen
 - wsimport
 - xjc

This container was pulled from:

	https://quay.io/repository/biocontainers/effectivet3

If you encounter errors in effectivet3 or need help running the
tools it contains, please contact the developer at

	http://www.effectivedb.org/method/effectivet3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000000.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000000.000000001.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: effectivet3", "Version: ctr-1.0.1--py27_0", "Category: ['Sequence classification']", "Keywords: ['Sequence analysis']", "Description: Prediction of putative Type-III secreted proteins.", "URL: https://quay.io/repository/biocontainers/effectivet3",
          },
        },
        ["effectivet3/ctr-1.0.1--py35_0"] = {
          Category = "['Sequence classification']",
          Description = "Prediction of putative Type-III secreted proteins.",
          Name = "effectivet3",
          URL = "https://quay.io/repository/biocontainers/effectivet3",
          Version = "ctr-1.0.1--py35_0",
          canonical = "ctr-1.0.1--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/effectivet3/ctr-1.0.1--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/effectivet3:1.0.1--py35_0, which exposes the
following programs:

 - appletviewer
 - easy_install-3.5
 - effectivet3
 - extcheck
 - idle3.5
 - idlj
 - jar
 - jarsigner
 - java-rmi.cgi
 - javac
 - javadoc
 - javah
 - javap
 - jcmd
 - jconsole
 - jdb
 - jdeps
 - jhat
 - jinfo
 - jmap
 - jps
 - jrunscript
 - jsadebugd
 - jstack
 - jstat
 - jstatd
 - native2ascii
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - rmic
 - schemagen
 - serialver
 - wsgen
 - wsimport
 - xjc

This container was pulled from:

	https://quay.io/repository/biocontainers/effectivet3

If you encounter errors in effectivet3 or need help running the
tools it contains, please contact the developer at

	http://www.effectivedb.org/method/effectivet3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000000.000000001.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000000.000000001.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: effectivet3", "Version: ctr-1.0.1--py35_0", "Category: ['Sequence classification']", "Keywords: ['Sequence analysis']", "Description: Prediction of putative Type-III secreted proteins.", "URL: https://quay.io/repository/biocontainers/effectivet3",
          },
        },
        ["effectivet3/ctr-1.0.1--py36_0"] = {
          Category = "['Sequence classification']",
          Description = "Prediction of putative Type-III secreted proteins.",
          Name = "effectivet3",
          URL = "https://quay.io/repository/biocontainers/effectivet3",
          Version = "ctr-1.0.1--py36_0",
          canonical = "ctr-1.0.1--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/effectivet3/ctr-1.0.1--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/effectivet3:1.0.1--py36_0, which exposes the
following programs:

 - appletviewer
 - easy_install-3.6
 - effectivet3
 - extcheck
 - idlj
 - jar
 - jarsigner
 - java-rmi.cgi
 - javac
 - javadoc
 - javah
 - javap
 - jcmd
 - jconsole
 - jdb
 - jdeps
 - jhat
 - jinfo
 - jmap
 - jps
 - jrunscript
 - jsadebugd
 - jstack
 - jstat
 - jstatd
 - native2ascii
 - rmic
 - schemagen
 - serialver
 - wsgen
 - wsimport
 - xjc

This container was pulled from:

	https://quay.io/repository/biocontainers/effectivet3

If you encounter errors in effectivet3 or need help running the
tools it contains, please contact the developer at

	http://www.effectivedb.org/method/effectivet3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000000.000000001.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000000.000000001.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: effectivet3", "Version: ctr-1.0.1--py36_0", "Category: ['Sequence classification']", "Keywords: ['Sequence analysis']", "Description: Prediction of putative Type-III secreted proteins.", "URL: https://quay.io/repository/biocontainers/effectivet3",
          },
        },
      },
    },
    egglib = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["egglib/ctr-3.0.0b21--0"] = {
          Category = "['Bioinformatics']",
          Description = "The egglib package",
          Name = "egglib",
          URL = "https://quay.io/repository/biocontainers/egglib",
          Version = "ctr-3.0.0b21--0",
          canonical = "ctr-3.0.0b21--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/egglib/ctr-3.0.0b21--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/egglib:3.0.0b21--0, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/egglib

If you encounter errors in egglib or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/egglib

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.*b.000000021.*zfinal",
          wV = "*ctr.*zfinal-.000000003.*b.000000021.*zfinal",
          whatis = {
            "Name: egglib", "Version: ctr-3.0.0b21--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The egglib package", "URL: https://quay.io/repository/biocontainers/egglib",
          },
        },
        ["egglib/ctr-3.0.0b21--py27h2d50403_1"] = {
          Category = "['Bioinformatics']",
          Description = "The egglib package",
          Name = "egglib",
          URL = "https://quay.io/repository/biocontainers/egglib",
          Version = "ctr-3.0.0b21--py27h2d50403_1",
          canonical = "ctr-3.0.0b21--py27h2d50403_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/egglib/ctr-3.0.0b21--py27h2d50403_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/egglib:3.0.0b21--py27h2d50403_1, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/egglib

If you encounter errors in egglib or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/egglib

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 29,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.*b.000000021.*y.000000027.*h.000000002.*d.000050403.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.*b.000000021.*y.000000027.*h.000000002.*d.000050403.*_.000000001.*zfinal",
          whatis = {
            "Name: egglib", "Version: ctr-3.0.0b21--py27h2d50403_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The egglib package", "URL: https://quay.io/repository/biocontainers/egglib",
          },
        },
      },
    },
    eigensoft = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["eigensoft/ctr-6.0.1--hd535c10_3"] = {
          Category = "['Genetic variation analysis']",
          Description = "Combindes functionality from population genetics methods (Patterson et al. 2006) and the EIGENSTRAT correection method (Price et al. 2006). The EIGENSTRAT methods uses principal component analysis to model ancestry differences between cases and controls along continuous axes of variation. The resulting correction is specific to a candidate markers variation in frequency across ancestral populations, minimizing spurious associations while maximising power to detect true associations.",
          Name = "eigensoft",
          URL = "https://quay.io/repository/biocontainers/eigensoft",
          Version = "ctr-6.0.1--hd535c10_3",
          canonical = "ctr-6.0.1--hd535c10_3",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eigensoft/ctr-6.0.1--hd535c10_3.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eigensoft:6.0.1--hd535c10_3, which exposes the
following programs:

 - baseprog
 - convertf
 - eigenstrat
 - eigenstratQTL
 - mergeit
 - pca
 - pcatoy
 - perl5.26.2
 - smarteigenstrat
 - smartpca
 - smartrel
 - twstats

This container was pulled from:

	https://quay.io/repository/biocontainers/eigensoft

If you encounter errors in eigensoft or need help running the
tools it contains, please contact the developer at

	https://www.hsph.harvard.edu/alkes-price/software/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000006.000000000.000000001.*hd.000000535.*c.000000010.*_.000000003.*zfinal",
          wV = "*ctr.*zfinal-.000000006.000000000.000000001.*hd.000000535.*c.000000010.*_.000000003.*zfinal",
          whatis = {
            "Name: eigensoft", "Version: ctr-6.0.1--hd535c10_3", "Category: ['Genetic variation analysis']", "Keywords: ['Population genetics']", "Description: Combindes functionality from population genetics methods (Patterson et al. 2006) and the EIGENSTRAT correection method (Price et al. 2006). The EIGENSTRAT methods uses principal component analysis to model ancestry differences between cases and controls along continuous axes of variation. The resulting correction is specific to a candidate markers variation in frequency across ancestral populations, minimizing spurious associations while maximising power to detect true associations.", "URL: https://quay.io/repository/biocontainers/eigensoft",
          },
        },
        ["eigensoft/ctr-6.0.1--pl5.22.0_1"] = {
          Category = "['Genetic variation analysis']",
          Description = "Combindes functionality from population genetics methods (Patterson et al. 2006) and the EIGENSTRAT correection method (Price et al. 2006). The EIGENSTRAT methods uses principal component analysis to model ancestry differences between cases and controls along continuous axes of variation. The resulting correction is specific to a candidate markers variation in frequency across ancestral populations, minimizing spurious associations while maximising power to detect true associations.",
          Name = "eigensoft",
          URL = "https://quay.io/repository/biocontainers/eigensoft",
          Version = "ctr-6.0.1--pl5.22.0_1",
          canonical = "ctr-6.0.1--pl5.22.0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eigensoft/ctr-6.0.1--pl5.22.0_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eigensoft:6.0.1--pl5.22.0_1, which exposes the
following programs:

 - baseprog
 - convertf
 - eigenstrat
 - eigenstratQTL
 - evec2pca-ped.perl
 - evec2pca.perl
 - fixgreen
 - gc.perl
 - mergeit
 - oldsmartpca
 - pca
 - pcatoy
 - perl5.22.0
 - ploteig
 - smarteigenstrat
 - smarteigenstrat.perl
 - smartpca
 - smartpca.perl
 - smartrel
 - twstats

This container was pulled from:

	https://quay.io/repository/biocontainers/eigensoft

If you encounter errors in eigensoft or need help running the
tools it contains, please contact the developer at

	https://www.hsph.harvard.edu/alkes-price/software/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000006.000000000.000000001.*l.000000005.000000022.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000006.000000000.000000001.*l.000000005.000000022.*_.000000001.*zfinal",
          whatis = {
            "Name: eigensoft", "Version: ctr-6.0.1--pl5.22.0_1", "Category: ['Genetic variation analysis']", "Keywords: ['Population genetics']", "Description: Combindes functionality from population genetics methods (Patterson et al. 2006) and the EIGENSTRAT correection method (Price et al. 2006). The EIGENSTRAT methods uses principal component analysis to model ancestry differences between cases and controls along continuous axes of variation. The resulting correction is specific to a candidate markers variation in frequency across ancestral populations, minimizing spurious associations while maximising power to detect true associations.", "URL: https://quay.io/repository/biocontainers/eigensoft",
          },
        },
        ["eigensoft/ctr-7.2.1--hd1ccf6d_0"] = {
          Category = "['Genetic variation analysis']",
          Description = "Combindes functionality from population genetics methods (Patterson et al. 2006) and the EIGENSTRAT correection method (Price et al. 2006). The EIGENSTRAT methods uses principal component analysis to model ancestry differences between cases and controls along continuous axes of variation. The resulting correction is specific to a candidate markers variation in frequency across ancestral populations, minimizing spurious associations while maximising power to detect true associations.",
          Name = "eigensoft",
          URL = "https://quay.io/repository/biocontainers/eigensoft",
          Version = "ctr-7.2.1--hd1ccf6d_0",
          canonical = "ctr-7.2.1--hd1ccf6d_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eigensoft/ctr-7.2.1--hd1ccf6d_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eigensoft:7.2.1--hd1ccf6d_0, which exposes the
following programs:

 - baseprog
 - convertf
 - eigenstrat
 - eigenstratQTL
 - mergeit
 - pca
 - pcatoy
 - perl5.26.2
 - smarteigenstrat
 - smartpca
 - smartrel
 - smshrink
 - twstats

This container was pulled from:

	https://quay.io/repository/biocontainers/eigensoft

If you encounter errors in eigensoft or need help running the
tools it contains, please contact the developer at

	https://www.hsph.harvard.edu/alkes-price/software/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000007.000000002.000000001.*hd.000000001.*ccf.000000006.*d.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000007.000000002.000000001.*hd.000000001.*ccf.000000006.*d.*_.*zfinal",
          whatis = {
            "Name: eigensoft", "Version: ctr-7.2.1--hd1ccf6d_0", "Category: ['Genetic variation analysis']", "Keywords: ['Population genetics']", "Description: Combindes functionality from population genetics methods (Patterson et al. 2006) and the EIGENSTRAT correection method (Price et al. 2006). The EIGENSTRAT methods uses principal component analysis to model ancestry differences between cases and controls along continuous axes of variation. The resulting correction is specific to a candidate markers variation in frequency across ancestral populations, minimizing spurious associations while maximising power to detect true associations.", "URL: https://quay.io/repository/biocontainers/eigensoft",
          },
        },
        ["eigensoft/ctr-v6.1.4dfsg-1b1-deb_cv1"] = {
          Category = "['Genetic variation analysis']",
          Description = "Combindes functionality from population genetics methods (Patterson et al. 2006) and the EIGENSTRAT correection method (Price et al. 2006). The EIGENSTRAT methods uses principal component analysis to model ancestry differences between cases and controls along continuous axes of variation. The resulting correction is specific to a candidate markers variation in frequency across ancestral populations, minimizing spurious associations while maximising power to detect true associations.",
          Name = "eigensoft",
          URL = "https://hub.docker.com/r/biocontainers/eigensoft",
          Version = "ctr-v6.1.4dfsg-1b1-deb_cv1",
          canonical = "ctr-v6.1.4dfsg-1b1-deb_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eigensoft/ctr-v6.1.4dfsg-1b1-deb_cv1.lua",
          help = [[This is a module file for the container biocontainers/eigensoft:v6.1.4dfsg-1b1-deb_cv1, which exposes the
following programs:

 - baseprog
 - broadwayd
 - convertf
 - eigenstrat
 - eigenstratQTL
 - evec2pca
 - evec2pca-ped
 - gc-eigensoft
 - glxdemo
 - glxgears
 - glxheads
 - glxinfo
 - gnuplot
 - gnuplot-qt
 - gtk-builder-tool
 - gtk-launch
 - gtk-query-settings
 - libwacom-list-local-devices
 - mergeit
 - pca
 - pcaselection
 - pcatoy
 - ploteig
 - smarteigenstrat
 - smartpca
 - smartrel
 - twstats

This container was pulled from:

	https://hub.docker.com/r/biocontainers/eigensoft

If you encounter errors in eigensoft or need help running the
tools it contains, please contact the developer at

	https://www.hsph.harvard.edu/alkes-price/software/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000006.000000001.000000004.*dfsg.*zfinal-.000000001.*b.000000001.*deb.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000006.000000001.000000004.*dfsg.*zfinal-.000000001.*b.000000001.*deb.*_.*cv.000000001.*zfinal",
          whatis = {
            "Name: eigensoft", "Version: ctr-v6.1.4dfsg-1b1-deb_cv1", "Category: ['Genetic variation analysis']", "Keywords: ['Population genetics']", "Description: Combindes functionality from population genetics methods (Patterson et al. 2006) and the EIGENSTRAT correection method (Price et al. 2006). The EIGENSTRAT methods uses principal component analysis to model ancestry differences between cases and controls along continuous axes of variation. The resulting correction is specific to a candidate markers variation in frequency across ancestral populations, minimizing spurious associations while maximising power to detect true associations.", "URL: https://hub.docker.com/r/biocontainers/eigensoft",
          },
        },
      },
    },
    elasticluster = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["elasticluster/ctr-0.1.3bcbio--py27_11"] = {
          Category = "['Bioinformatics']",
          Description = "The elasticluster package",
          Name = "elasticluster",
          URL = "https://quay.io/repository/biocontainers/elasticluster",
          Version = "ctr-0.1.3bcbio--py27_11",
          canonical = "ctr-0.1.3bcbio--py27_11",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elasticluster/ctr-0.1.3bcbio--py27_11.lua",
          help = [[This is a module file for the container quay.io/biocontainers/elasticluster:0.1.3bcbio--py27_11, which exposes the
following programs:

 - ansible
 - ansible-doc
 - ansible-galaxy
 - ansible-playbook
 - ansible-pull
 - ansible-vault
 - asadmin
 - bundle_image
 - cfadmin
 - cq
 - cwutil
 - dynamodb_dump
 - dynamodb_load
 - elasticluster
 - elbadmin
 - fetch_file
 - gflags2man.py
 - glacier
 - instance_events
 - kill_instance
 - launch_instance
 - list_instances
 - lss3
 - mturk
 - pyami_sendmail
 - pyrsa-decrypt
 - pyrsa-decrypt-bigfile
 - pyrsa-encrypt
 - pyrsa-encrypt-bigfile
 - pyrsa-keygen
 - pyrsa-priv2pub
 - pyrsa-sign
 - pyrsa-verify
 - route53
 - s3put
 - sdbadmin
 - taskadmin

This container was pulled from:

	https://quay.io/repository/biocontainers/elasticluster

If you encounter errors in elasticluster or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/elasticluster

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 24,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000003.*bcbio.*y.000000027.*_.000000011.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000003.*bcbio.*y.000000027.*_.000000011.*zfinal",
          whatis = {
            "Name: elasticluster", "Version: ctr-0.1.3bcbio--py27_11", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The elasticluster package", "URL: https://quay.io/repository/biocontainers/elasticluster",
          },
        },
        ["elasticluster/ctr-0.1.3bcbio--py27_12"] = {
          Category = "['Bioinformatics']",
          Description = "The elasticluster package",
          Name = "elasticluster",
          URL = "https://quay.io/repository/biocontainers/elasticluster",
          Version = "ctr-0.1.3bcbio--py27_12",
          canonical = "ctr-0.1.3bcbio--py27_12",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elasticluster/ctr-0.1.3bcbio--py27_12.lua",
          help = [[This is a module file for the container quay.io/biocontainers/elasticluster:0.1.3bcbio--py27_12, which exposes the
following programs:

 - ansible
 - ansible-doc
 - ansible-galaxy
 - ansible-playbook
 - ansible-pull
 - ansible-vault
 - asadmin
 - bundle_image
 - cfadmin
 - cq
 - cwutil
 - dynamodb_dump
 - dynamodb_load
 - elasticluster
 - elbadmin
 - fetch_file
 - gflags2man.py
 - glacier
 - instance_events
 - kill_instance
 - launch_instance
 - list_instances
 - lss3
 - mturk
 - pyami_sendmail
 - pyrsa-decrypt
 - pyrsa-decrypt-bigfile
 - pyrsa-encrypt
 - pyrsa-encrypt-bigfile
 - pyrsa-keygen
 - pyrsa-priv2pub
 - pyrsa-sign
 - pyrsa-verify
 - route53
 - s3put
 - sdbadmin
 - taskadmin

This container was pulled from:

	https://quay.io/repository/biocontainers/elasticluster

If you encounter errors in elasticluster or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/elasticluster

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 24,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000003.*bcbio.*y.000000027.*_.000000012.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000003.*bcbio.*y.000000027.*_.000000012.*zfinal",
          whatis = {
            "Name: elasticluster", "Version: ctr-0.1.3bcbio--py27_12", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The elasticluster package", "URL: https://quay.io/repository/biocontainers/elasticluster",
          },
        },
      },
    },
    elastix = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["elastix/ctr-v4.8-10-deb_cv1"] = {
          Category = "['Bioinformatics']",
          Description = "The elastix package",
          Name = "elastix",
          URL = "https://hub.docker.com/r/biocontainers/elastix",
          Version = "ctr-v4.8-10-deb_cv1",
          canonical = "ctr-v4.8-10-deb_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elastix/ctr-v4.8-10-deb_cv1.lua",
          help = [[This is a module file for the container biocontainers/elastix:v4.8-10-deb_cv1, which exposes the
following programs:

 - elastix
 - transformix

This container was pulled from:

	https://hub.docker.com/r/biocontainers/elastix

If you encounter errors in elastix or need help running the
tools it contains, please contact the developer at

	https://hub.docker.com/r/biocontainers/elastix

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000004.000000008.*zfinal-.000000010.*deb.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000004.000000008.*zfinal-.000000010.*deb.*_.*cv.000000001.*zfinal",
          whatis = {
            "Name: elastix", "Version: ctr-v4.8-10-deb_cv1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The elastix package", "URL: https://hub.docker.com/r/biocontainers/elastix",
          },
        },
        ["elastix/ctr-v4.8-10b1-deb_cv1"] = {
          Category = "['Bioinformatics']",
          Description = "The elastix package",
          Name = "elastix",
          URL = "https://hub.docker.com/r/biocontainers/elastix",
          Version = "ctr-v4.8-10b1-deb_cv1",
          canonical = "ctr-v4.8-10b1-deb_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elastix/ctr-v4.8-10b1-deb_cv1.lua",
          help = [[This is a module file for the container biocontainers/elastix:v4.8-10b1-deb_cv1, which exposes the
following programs:

 - elastix
 - transformix

This container was pulled from:

	https://hub.docker.com/r/biocontainers/elastix

If you encounter errors in elastix or need help running the
tools it contains, please contact the developer at

	https://hub.docker.com/r/biocontainers/elastix

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000004.000000008.*zfinal-.000000010.*b.000000001.*deb.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000004.000000008.*zfinal-.000000010.*b.000000001.*deb.*_.*cv.000000001.*zfinal",
          whatis = {
            "Name: elastix", "Version: ctr-v4.8-10b1-deb_cv1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The elastix package", "URL: https://hub.docker.com/r/biocontainers/elastix",
          },
        },
      },
    },
    elprep = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["elprep/ctr-3.04--0"] = {
          Category = "['Variant calling']",
          Description = "Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.",
          Name = "elprep",
          URL = "https://quay.io/repository/biocontainers/elprep",
          Version = "ctr-3.04--0",
          canonical = "ctr-3.04--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elprep/ctr-3.04--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/elprep:3.04--0, which exposes the
following programs:

 - elprep
 - elprep-sfm-gnupar.py
 - elprep-sfm.py
 - elprep.py
 - elprep_entrypoint.py
 - elprep_im.py
 - elprep_io_wrapper.py
 - elprep_sfm.py
 - elprep_sfm_gnupar.py

This container was pulled from:

	https://quay.io/repository/biocontainers/elprep

If you encounter errors in elprep or need help running the
tools it contains, please contact the developer at

	https://github.com/ExaScience/elprep

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 12,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000004.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000004.*zfinal",
          whatis = {
            "Name: elprep", "Version: ctr-3.04--0", "Category: ['Variant calling']", "Keywords: ['Sequencing', 'Genetic variation', 'Sequence analysis']", "Description: Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.", "URL: https://quay.io/repository/biocontainers/elprep",
          },
        },
        ["elprep/ctr-3.04--1"] = {
          Category = "['Variant calling']",
          Description = "Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.",
          Name = "elprep",
          URL = "https://quay.io/repository/biocontainers/elprep",
          Version = "ctr-3.04--1",
          canonical = "ctr-3.04--1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elprep/ctr-3.04--1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/elprep:3.04--1, which exposes the
following programs:

 - elprep
 - elprep-sfm-gnupar.py
 - elprep-sfm.py
 - elprep.py
 - elprep_entrypoint.py
 - elprep_im.py
 - elprep_io_wrapper.py
 - elprep_sfm.py
 - elprep_sfm_gnupar.py

This container was pulled from:

	https://quay.io/repository/biocontainers/elprep

If you encounter errors in elprep or need help running the
tools it contains, please contact the developer at

	https://github.com/ExaScience/elprep

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 12,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000004.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000004.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: elprep", "Version: ctr-3.04--1", "Category: ['Variant calling']", "Keywords: ['Sequencing', 'Genetic variation', 'Sequence analysis']", "Description: Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.", "URL: https://quay.io/repository/biocontainers/elprep",
          },
        },
        ["elprep/ctr-4.0.0--0"] = {
          Category = "['Variant calling']",
          Description = "Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.",
          Name = "elprep",
          URL = "https://quay.io/repository/biocontainers/elprep",
          Version = "ctr-4.0.0--0",
          canonical = "ctr-4.0.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elprep/ctr-4.0.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/elprep:4.0.0--0, which exposes the
following programs:

 - elprep

This container was pulled from:

	https://quay.io/repository/biocontainers/elprep

If you encounter errors in elprep or need help running the
tools it contains, please contact the developer at

	https://github.com/ExaScience/elprep

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000004.*zfinal",
          wV = "*ctr.*zfinal-.000000004.*zfinal",
          whatis = {
            "Name: elprep", "Version: ctr-4.0.0--0", "Category: ['Variant calling']", "Keywords: ['Sequencing', 'Genetic variation', 'Sequence analysis']", "Description: Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.", "URL: https://quay.io/repository/biocontainers/elprep",
          },
        },
        ["elprep/ctr-4.0.1--0"] = {
          Category = "['Variant calling']",
          Description = "Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.",
          Name = "elprep",
          URL = "https://quay.io/repository/biocontainers/elprep",
          Version = "ctr-4.0.1--0",
          canonical = "ctr-4.0.1--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elprep/ctr-4.0.1--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/elprep:4.0.1--0, which exposes the
following programs:

 - elprep

This container was pulled from:

	https://quay.io/repository/biocontainers/elprep

If you encounter errors in elprep or need help running the
tools it contains, please contact the developer at

	https://github.com/ExaScience/elprep

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000004.000000000.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000004.000000000.000000001.*zfinal",
          whatis = {
            "Name: elprep", "Version: ctr-4.0.1--0", "Category: ['Variant calling']", "Keywords: ['Sequencing', 'Genetic variation', 'Sequence analysis']", "Description: Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.", "URL: https://quay.io/repository/biocontainers/elprep",
          },
        },
        ["elprep/ctr-4.1.0--0"] = {
          Category = "['Variant calling']",
          Description = "Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.",
          Name = "elprep",
          URL = "https://quay.io/repository/biocontainers/elprep",
          Version = "ctr-4.1.0--0",
          canonical = "ctr-4.1.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elprep/ctr-4.1.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/elprep:4.1.0--0, which exposes the
following programs:

 - elprep

This container was pulled from:

	https://quay.io/repository/biocontainers/elprep

If you encounter errors in elprep or need help running the
tools it contains, please contact the developer at

	https://github.com/ExaScience/elprep

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000004.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000004.000000001.*zfinal",
          whatis = {
            "Name: elprep", "Version: ctr-4.1.0--0", "Category: ['Variant calling']", "Keywords: ['Sequencing', 'Genetic variation', 'Sequence analysis']", "Description: Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.", "URL: https://quay.io/repository/biocontainers/elprep",
          },
        },
        ["elprep/ctr-4.1.1--0"] = {
          Category = "['Variant calling']",
          Description = "Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.",
          Name = "elprep",
          URL = "https://quay.io/repository/biocontainers/elprep",
          Version = "ctr-4.1.1--0",
          canonical = "ctr-4.1.1--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/elprep/ctr-4.1.1--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/elprep:4.1.1--0, which exposes the
following programs:

 - elprep

This container was pulled from:

	https://quay.io/repository/biocontainers/elprep

If you encounter errors in elprep or need help running the
tools it contains, please contact the developer at

	https://github.com/ExaScience/elprep

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000004.000000001.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000004.000000001.000000001.*zfinal",
          whatis = {
            "Name: elprep", "Version: ctr-4.1.1--0", "Category: ['Variant calling']", "Keywords: ['Sequencing', 'Genetic variation', 'Sequence analysis']", "Description: Is a high-performance tool for preparing .sam/.bam/.cram files for variant calling in sequencing pipelines.It can be used as a drop-in replacement for SAMtools/Picard, and was extensively tested with different pipelines for variant analysis with GATK.", "URL: https://quay.io/repository/biocontainers/elprep",
          },
        },
      },
    },
    ema = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ema/ctr-0.6.2--hd28b015_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ema package",
          Name = "ema",
          URL = "https://quay.io/repository/biocontainers/ema",
          Version = "ctr-0.6.2--hd28b015_0",
          canonical = "ctr-0.6.2--hd28b015_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ema/ctr-0.6.2--hd28b015_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ema:0.6.2--hd28b015_0, which exposes the
following programs:

 - ema

This container was pulled from:

	https://quay.io/repository/biocontainers/ema

If you encounter errors in ema or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ema

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000006.000000002.*hd.000000028.*b.000000015.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000006.000000002.*hd.000000028.*b.000000015.*_.*zfinal",
          whatis = {
            "Name: ema", "Version: ctr-0.6.2--hd28b015_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ema package", "URL: https://quay.io/repository/biocontainers/ema",
          },
        },
        ["ema/ctr-v0.6.1--0"] = {
          Category = "['Bioinformatics']",
          Description = "The ema package",
          Name = "ema",
          URL = "https://quay.io/repository/biocontainers/ema",
          Version = "ctr-v0.6.1--0",
          canonical = "ctr-v0.6.1--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ema/ctr-v0.6.1--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ema:v0.6.1--0, which exposes the
following programs:

 - ema

This container was pulled from:

	https://quay.io/repository/biocontainers/ema

If you encounter errors in ema or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ema

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 14,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000000.000000006.000000001.*zfinal",
          wV = "*ctr.*v.000000000.000000006.000000001.*zfinal",
          whatis = {
            "Name: ema", "Version: ctr-v0.6.1--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ema package", "URL: https://quay.io/repository/biocontainers/ema",
          },
        },
        ["ema/ctr-v0.6.1--hd28b015_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ema package",
          Name = "ema",
          URL = "https://quay.io/repository/biocontainers/ema",
          Version = "ctr-v0.6.1--hd28b015_1",
          canonical = "ctr-v0.6.1--hd28b015_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ema/ctr-v0.6.1--hd28b015_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ema:v0.6.1--hd28b015_1, which exposes the
following programs:

 - ema

This container was pulled from:

	https://quay.io/repository/biocontainers/ema

If you encounter errors in ema or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ema

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 23,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000000.000000006.000000001.*hd.000000028.*b.000000015.*_.000000001.*zfinal",
          wV = "*ctr.*v.000000000.000000006.000000001.*hd.000000028.*b.000000015.*_.000000001.*zfinal",
          whatis = {
            "Name: ema", "Version: ctr-v0.6.1--hd28b015_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ema package", "URL: https://quay.io/repository/biocontainers/ema",
          },
        },
        ["ema/ctr-v0.6.2--hd28b015_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ema package",
          Name = "ema",
          URL = "https://quay.io/repository/biocontainers/ema",
          Version = "ctr-v0.6.2--hd28b015_1",
          canonical = "ctr-v0.6.2--hd28b015_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ema/ctr-v0.6.2--hd28b015_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ema:v0.6.2--hd28b015_1, which exposes the
following programs:

 - ema

This container was pulled from:

	https://quay.io/repository/biocontainers/ema

If you encounter errors in ema or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ema

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 23,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000000.000000006.000000002.*hd.000000028.*b.000000015.*_.000000001.*zfinal",
          wV = "*ctr.*v.000000000.000000006.000000002.*hd.000000028.*b.000000015.*_.000000001.*zfinal",
          whatis = {
            "Name: ema", "Version: ctr-v0.6.2--hd28b015_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ema package", "URL: https://quay.io/repository/biocontainers/ema",
          },
        },
      },
    },
    emboss = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["emboss/ctr-5.0.0--0"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://quay.io/repository/biocontainers/emboss",
          Version = "ctr-5.0.0--0",
          canonical = "ctr-5.0.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-5.0.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emboss:5.0.0--0, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - antigenic
 - backtranambig
 - backtranseq
 - banana
 - bdftogd
 - biosed
 - btwisted
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - cons
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbxfasta
 - dbxflat
 - dbxgcg
 - degapseq
 - descseq
 - diffseq
 - digest
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - edialign
 - einverted
 - embossdata
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - fclique
 - fconsense
 - fcontml
 - fcontrast
 - fdiscboot
 - fdnacomp
 - fdnadist
 - fdnainvar
 - fdnaml
 - fdnamlk
 - fdnamove
 - fdnapars
 - fdnapenny
 - fdollop
 - fdolmove
 - fdolpenny
 - fdrawgram
 - fdrawtree
 - ffactor
 - ffitch
 - ffreqboot
 - fgendist
 - findkm
 - fkitsch
 - fmix
 - fmove
 - fneighbor
 - fpars
 - fpenny
 - fproml
 - fpromlk
 - fprotdist
 - fprotpars
 - freak
 - frestboot
 - frestdist
 - frestml
 - fretree
 - fseqboot
 - fseqbootall
 - ftreedist
 - ftreedistpair
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - gd2copypal
 - gd2togif
 - gd2topng
 - gdcmpgif
 - gdlib-config
 - gdparttopng
 - gdtopng
 - geecee
 - getorf
 - giftogd2
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoseq
 - isochore
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - newcpgreport
 - newcpgseek
 - newseq
 - noreturn
 - notseq
 - nthseq
 - octanol
 - oddcomp
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - pngtogd
 - pngtogd2
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - pscan
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - remap
 - restover
 - restrict
 - revseq
 - runJemboss.csh
 - seealso
 - seqmatchall
 - seqret
 - seqretsplit
 - showalign
 - showdb
 - showfeat
 - showorf
 - showseq
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - skipseq
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - tcode
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - twofeat
 - union
 - vectorstrip
 - water
 - webpng
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossname
 - xslt-config
 - xsltproc
 - yank

This container was pulled from:

	https://quay.io/repository/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000005.*zfinal",
          wV = "*ctr.*zfinal-.000000005.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-5.0.0--0", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.", "URL: https://quay.io/repository/biocontainers/emboss",
          },
        },
        ["emboss/ctr-6.5.7"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://hub.docker.com/r/biocontainers/emboss",
          Version = "ctr-6.5.7",
          canonical = "ctr-6.5.7",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-6.5.7.lua",
          help = [[This is a module file for the container biocontainers/emboss:6.5.7, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdgalaxy
 - acdlog
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - aligncopy
 - aligncopypair
 - antigenic
 - assemblyget
 - backtranambig
 - backtranseq
 - banana
 - biosed
 - btwisted
 - cachedas
 - cachedbfetch
 - cacheebeyesearch
 - cacheensembl
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - cons
 - consambig
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbtell
 - dbxcompress
 - dbxedam
 - dbxfasta
 - dbxflat
 - dbxgcg
 - dbxobo
 - dbxreport
 - dbxresource
 - dbxstat
 - dbxtax
 - dbxuncompress
 - degapseq
 - density
 - descseq
 - diffseq
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - drfinddata
 - drfindformat
 - drfindid
 - drfindresource
 - drget
 - drtext
 - edamdef
 - edamhasinput
 - edamhasoutput
 - edamisformat
 - edamisid
 - edamname
 - edialign
 - einverted
 - embossdata
 - embossupdate
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - eprimer32
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - featcopy
 - featmerge
 - featreport
 - feattext
 - findkm
 - freak
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - geecee
 - getorf
 - godef
 - goname
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoassembly
 - infobase
 - inforesidue
 - infoseq
 - isochore
 - jaspextract
 - jaspscan
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskambignuc
 - maskambigprot
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - needleall
 - newcpgreport
 - newcpgseek
 - newseq
 - nohtml
 - noreturn
 - nospace
 - notab
 - notseq
 - nthseq
 - nthseqset
 - octanol
 - oddcomp
 - ontocount
 - ontoget
 - ontogetcommon
 - ontogetdown
 - ontogetobsolete
 - ontogetroot
 - ontogetsibs
 - ontogetup
 - ontoisobsolete
 - ontotext
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepdigest
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - pscan
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - refseqget
 - remap
 - restover
 - restrict
 - revseq
 - runJemboss.sh
 - seealso
 - seqcount
 - seqmatchall
 - seqret
 - seqretsetall
 - seqretsplit
 - seqxref
 - seqxrefget
 - servertell
 - showalign
 - showdb
 - showfeat
 - showorf
 - showpep
 - showseq
 - showserver
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - sizeseq
 - skipredundant
 - skipseq
 - splitsource
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - taxget
 - taxgetdown
 - taxgetrank
 - taxgetspecies
 - taxgetup
 - tcode
 - textget
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - trimspace
 - twofeat
 - union
 - urlget
 - variationget
 - vectorstrip
 - water
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossdata
 - wossinput
 - wossname
 - wossoperation
 - wossoutput
 - wossparam
 - wosstopic
 - yank

This container was pulled from:

	https://hub.docker.com/r/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 10,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000006.000000005.000000007.*zfinal",
          wV = "*ctr.*zfinal-.000000006.000000005.000000007.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-6.5.7", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.", "URL: https://hub.docker.com/r/biocontainers/emboss",
          },
        },
        ["emboss/ctr-6.5.7--2"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://quay.io/repository/biocontainers/emboss",
          Version = "ctr-6.5.7--2",
          canonical = "ctr-6.5.7--2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-6.5.7--2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emboss:6.5.7--2, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdgalaxy
 - acdlog
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - aligncopy
 - aligncopypair
 - antigenic
 - assemblyget
 - backtranambig
 - backtranseq
 - banana
 - biosed
 - btwisted
 - cachedas
 - cachedbfetch
 - cacheebeyesearch
 - cacheensembl
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - cons
 - consambig
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbtell
 - dbxcompress
 - dbxedam
 - dbxfasta
 - dbxflat
 - dbxgcg
 - dbxobo
 - dbxreport
 - dbxresource
 - dbxstat
 - dbxtax
 - dbxuncompress
 - degapseq
 - density
 - descseq
 - diffseq
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - drfinddata
 - drfindformat
 - drfindid
 - drfindresource
 - drget
 - drtext
 - edamdef
 - edamhasinput
 - edamhasoutput
 - edamisformat
 - edamisid
 - edamname
 - edialign
 - einverted
 - embossdata
 - embossupdate
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - eprimer32
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - featcopy
 - featmerge
 - featreport
 - feattext
 - findkm
 - freak
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - geecee
 - getorf
 - godef
 - goname
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoassembly
 - infobase
 - inforesidue
 - infoseq
 - isochore
 - jaspextract
 - jaspscan
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskambignuc
 - maskambigprot
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - needleall
 - newcpgreport
 - newcpgseek
 - newseq
 - nohtml
 - noreturn
 - nospace
 - notab
 - notseq
 - nthseq
 - nthseqset
 - octanol
 - oddcomp
 - ontocount
 - ontoget
 - ontogetcommon
 - ontogetdown
 - ontogetobsolete
 - ontogetroot
 - ontogetsibs
 - ontogetup
 - ontoisobsolete
 - ontotext
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepdigest
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - pscan
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - refseqget
 - remap
 - restover
 - restrict
 - revseq
 - runJemboss.sh
 - seealso
 - seqcount
 - seqmatchall
 - seqret
 - seqretsetall
 - seqretsplit
 - seqxref
 - seqxrefget
 - servertell
 - showalign
 - showdb
 - showfeat
 - showorf
 - showpep
 - showseq
 - showserver
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - sizeseq
 - skipredundant
 - skipseq
 - splitsource
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - taxget
 - taxgetdown
 - taxgetrank
 - taxgetspecies
 - taxgetup
 - tcode
 - textget
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - trimspace
 - twofeat
 - union
 - urlget
 - variationget
 - vectorstrip
 - water
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossdata
 - wossinput
 - wossname
 - wossoperation
 - wossoutput
 - wossparam
 - wosstopic
 - yank

This container was pulled from:

	https://quay.io/repository/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000006.000000005.000000007.*zfinal-.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000006.000000005.000000007.*zfinal-.000000002.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-6.5.7--2", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.", "URL: https://quay.io/repository/biocontainers/emboss",
          },
        },
        ["emboss/ctr-6.5.7--3"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://quay.io/repository/biocontainers/emboss",
          Version = "ctr-6.5.7--3",
          canonical = "ctr-6.5.7--3",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-6.5.7--3.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emboss:6.5.7--3, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdgalaxy
 - acdlog
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - aligncopy
 - aligncopypair
 - antigenic
 - assemblyget
 - backtranambig
 - backtranseq
 - banana
 - biosed
 - btwisted
 - cachedas
 - cachedbfetch
 - cacheebeyesearch
 - cacheensembl
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - cons
 - consambig
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbtell
 - dbxcompress
 - dbxedam
 - dbxfasta
 - dbxflat
 - dbxgcg
 - dbxobo
 - dbxreport
 - dbxresource
 - dbxstat
 - dbxtax
 - dbxuncompress
 - degapseq
 - density
 - descseq
 - diffseq
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - drfinddata
 - drfindformat
 - drfindid
 - drfindresource
 - drget
 - drtext
 - edamdef
 - edamhasinput
 - edamhasoutput
 - edamisformat
 - edamisid
 - edamname
 - edialign
 - einverted
 - embossdata
 - embossupdate
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - eprimer32
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - featcopy
 - featmerge
 - featreport
 - feattext
 - findkm
 - freak
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - geecee
 - getorf
 - godef
 - goname
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoassembly
 - infobase
 - inforesidue
 - infoseq
 - isochore
 - jaspextract
 - jaspscan
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskambignuc
 - maskambigprot
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - needleall
 - newcpgreport
 - newcpgseek
 - newseq
 - nohtml
 - noreturn
 - nospace
 - notab
 - notseq
 - nthseq
 - nthseqset
 - octanol
 - oddcomp
 - ontocount
 - ontoget
 - ontogetcommon
 - ontogetdown
 - ontogetobsolete
 - ontogetroot
 - ontogetsibs
 - ontogetup
 - ontoisobsolete
 - ontotext
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepdigest
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - pscan
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - refseqget
 - remap
 - restover
 - restrict
 - revseq
 - runJemboss.sh
 - seealso
 - seqcount
 - seqmatchall
 - seqret
 - seqretsetall
 - seqretsplit
 - seqxref
 - seqxrefget
 - servertell
 - showalign
 - showdb
 - showfeat
 - showorf
 - showpep
 - showseq
 - showserver
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - sizeseq
 - skipredundant
 - skipseq
 - splitsource
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - taxget
 - taxgetdown
 - taxgetrank
 - taxgetspecies
 - taxgetup
 - tcode
 - textget
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - trimspace
 - twofeat
 - union
 - urlget
 - variationget
 - vectorstrip
 - water
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossdata
 - wossinput
 - wossname
 - wossoperation
 - wossoutput
 - wossparam
 - wosstopic
 - yank

This container was pulled from:

	https://quay.io/repository/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000006.000000005.000000007.*zfinal-.000000003.*zfinal",
          wV = "*ctr.*zfinal-.000000006.000000005.000000007.*zfinal-.000000003.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-6.5.7--3", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.", "URL: https://quay.io/repository/biocontainers/emboss",
          },
        },
        ["emboss/ctr-6.5.7--4"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://quay.io/repository/biocontainers/emboss",
          Version = "ctr-6.5.7--4",
          canonical = "ctr-6.5.7--4",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-6.5.7--4.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emboss:6.5.7--4, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdgalaxy
 - acdlog
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - aligncopy
 - aligncopypair
 - antigenic
 - assemblyget
 - backtranambig
 - backtranseq
 - banana
 - biosed
 - btwisted
 - cachedas
 - cachedbfetch
 - cacheebeyesearch
 - cacheensembl
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - cons
 - consambig
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbtell
 - dbxcompress
 - dbxedam
 - dbxfasta
 - dbxflat
 - dbxgcg
 - dbxobo
 - dbxreport
 - dbxresource
 - dbxstat
 - dbxtax
 - dbxuncompress
 - degapseq
 - density
 - descseq
 - diffseq
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - drfinddata
 - drfindformat
 - drfindid
 - drfindresource
 - drget
 - drtext
 - edamdef
 - edamhasinput
 - edamhasoutput
 - edamisformat
 - edamisid
 - edamname
 - edialign
 - einverted
 - embossdata
 - embossupdate
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - eprimer32
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - featcopy
 - featmerge
 - featreport
 - feattext
 - findkm
 - freak
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - geecee
 - getorf
 - godef
 - goname
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoassembly
 - infobase
 - inforesidue
 - infoseq
 - isochore
 - jaspextract
 - jaspscan
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskambignuc
 - maskambigprot
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - needleall
 - newcpgreport
 - newcpgseek
 - newseq
 - nohtml
 - noreturn
 - nospace
 - notab
 - notseq
 - nthseq
 - nthseqset
 - octanol
 - oddcomp
 - ontocount
 - ontoget
 - ontogetcommon
 - ontogetdown
 - ontogetobsolete
 - ontogetroot
 - ontogetsibs
 - ontogetup
 - ontoisobsolete
 - ontotext
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepdigest
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - pscan
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - refseqget
 - remap
 - restover
 - restrict
 - revseq
 - runJemboss.sh
 - seealso
 - seqcount
 - seqmatchall
 - seqret
 - seqretsetall
 - seqretsplit
 - seqxref
 - seqxrefget
 - servertell
 - showalign
 - showdb
 - showfeat
 - showorf
 - showpep
 - showseq
 - showserver
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - sizeseq
 - skipredundant
 - skipseq
 - splitsource
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - taxget
 - taxgetdown
 - taxgetrank
 - taxgetspecies
 - taxgetup
 - tcode
 - textget
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - trimspace
 - twofeat
 - union
 - urlget
 - variationget
 - vectorstrip
 - water
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossdata
 - wossinput
 - wossname
 - wossoperation
 - wossoutput
 - wossparam
 - wosstopic
 - yank

This container was pulled from:

	https://quay.io/repository/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000006.000000005.000000007.*zfinal-.000000004.*zfinal",
          wV = "*ctr.*zfinal-.000000006.000000005.000000007.*zfinal-.000000004.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-6.5.7--4", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.", "URL: https://quay.io/repository/biocontainers/emboss",
          },
        },
        ["emboss/ctr-6.5.7--ha92aebf_5"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://quay.io/repository/biocontainers/emboss",
          Version = "ctr-6.5.7--ha92aebf_5",
          canonical = "ctr-6.5.7--ha92aebf_5",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-6.5.7--ha92aebf_5.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emboss:6.5.7--ha92aebf_5, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdgalaxy
 - acdlog
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - aligncopy
 - aligncopypair
 - antigenic
 - assemblyget
 - backtranambig
 - backtranseq
 - banana
 - biosed
 - btwisted
 - cachedas
 - cachedbfetch
 - cacheebeyesearch
 - cacheensembl
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - cons
 - consambig
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbtell
 - dbxcompress
 - dbxedam
 - dbxfasta
 - dbxflat
 - dbxgcg
 - dbxobo
 - dbxreport
 - dbxresource
 - dbxstat
 - dbxtax
 - dbxuncompress
 - degapseq
 - density
 - descseq
 - diffseq
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - drfinddata
 - drfindformat
 - drfindid
 - drfindresource
 - drget
 - drtext
 - edamdef
 - edamhasinput
 - edamhasoutput
 - edamisformat
 - edamisid
 - edamname
 - edialign
 - einverted
 - embossdata
 - embossupdate
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - eprimer32
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - featcopy
 - featmerge
 - featreport
 - feattext
 - findkm
 - freak
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - geecee
 - getorf
 - godef
 - goname
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoassembly
 - infobase
 - inforesidue
 - infoseq
 - isochore
 - jaspextract
 - jaspscan
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskambignuc
 - maskambigprot
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - needleall
 - newcpgreport
 - newcpgseek
 - newseq
 - nohtml
 - noreturn
 - nospace
 - notab
 - notseq
 - nthseq
 - nthseqset
 - octanol
 - oddcomp
 - ontocount
 - ontoget
 - ontogetcommon
 - ontogetdown
 - ontogetobsolete
 - ontogetroot
 - ontogetsibs
 - ontogetup
 - ontoisobsolete
 - ontotext
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepdigest
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - pscan
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - refseqget
 - remap
 - restover
 - restrict
 - revseq
 - runJemboss.sh
 - seealso
 - seqcount
 - seqmatchall
 - seqret
 - seqretsetall
 - seqretsplit
 - seqxref
 - seqxrefget
 - servertell
 - showalign
 - showdb
 - showfeat
 - showorf
 - showpep
 - showseq
 - showserver
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - sizeseq
 - skipredundant
 - skipseq
 - splitsource
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - taxget
 - taxgetdown
 - taxgetrank
 - taxgetspecies
 - taxgetup
 - tcode
 - textget
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - trimspace
 - twofeat
 - union
 - urlget
 - variationget
 - vectorstrip
 - water
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossdata
 - wossinput
 - wossname
 - wossoperation
 - wossoutput
 - wossparam
 - wosstopic
 - yank

This container was pulled from:

	https://quay.io/repository/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000006.000000005.000000007.*ha.000000092.*aebf.*_.000000005.*zfinal",
          wV = "*ctr.*zfinal-.000000006.000000005.000000007.*ha.000000092.*aebf.*_.000000005.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-6.5.7--ha92aebf_5", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.", "URL: https://quay.io/repository/biocontainers/emboss",
          },
        },
        ["emboss/ctr-6.6.0--h6debe1e_0"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://quay.io/repository/biocontainers/emboss",
          Version = "ctr-6.6.0--h6debe1e_0",
          canonical = "ctr-6.6.0--h6debe1e_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-6.6.0--h6debe1e_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emboss:6.6.0--h6debe1e_0, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdgalaxy
 - acdlog
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - aligncopy
 - aligncopypair
 - annotate
 - antigenic
 - assemblyget
 - backtranambig
 - backtranseq
 - banana
 - bdftogd
 - biosed
 - btwisted
 - cachedas
 - cachedbfetch
 - cacheebeyesearch
 - cacheensembl
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - cons
 - consambig
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - cwebp
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbtell
 - dbxcompress
 - dbxedam
 - dbxfasta
 - dbxflat
 - dbxgcg
 - dbxobo
 - dbxreport
 - dbxresource
 - dbxstat
 - dbxtax
 - dbxuncompress
 - degapseq
 - density
 - descseq
 - diffseq
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - drfinddata
 - drfindformat
 - drfindid
 - drfindresource
 - drget
 - drtext
 - dwebp
 - edamdef
 - edamhasinput
 - edamhasoutput
 - edamisformat
 - edamisid
 - edamname
 - edialign
 - einverted
 - embossdata
 - embossupdate
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - eprimer32
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - featcopy
 - featmerge
 - featreport
 - feattext
 - findkm
 - freak
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - gd2copypal
 - gd2togif
 - gd2topng
 - gdcmpgif
 - gdlib-config
 - gdparttopng
 - gdtopng
 - geecee
 - getorf
 - gif2rgb
 - gif2webp
 - gifbuild
 - gifclrmp
 - gifecho
 - giffix
 - gifinto
 - giftext
 - giftogd2
 - giftool
 - godef
 - goname
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoassembly
 - infobase
 - inforesidue
 - infoseq
 - isochore
 - jaspextract
 - jaspscan
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskambignuc
 - maskambigprot
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - needleall
 - newcpgreport
 - newcpgseek
 - newseq
 - nohtml
 - noreturn
 - nospace
 - notab
 - notseq
 - nthseq
 - nthseqset
 - octanol
 - oddcomp
 - ontocount
 - ontoget
 - ontogetcommon
 - ontogetdown
 - ontogetobsolete
 - ontogetroot
 - ontogetsibs
 - ontogetup
 - ontoisobsolete
 - ontotext
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepdigest
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - pngtogd
 - pngtogd2
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - pscan
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - refseqget
 - remap
 - restover
 - restrict
 - revseq
 - runJemboss.sh
 - seealso
 - seqcount
 - seqmatchall
 - seqret
 - seqretsetall
 - seqretsplit
 - seqxref
 - seqxrefget
 - servertell
 - showalign
 - showdb
 - showfeat
 - showorf
 - showpep
 - showseq
 - showserver
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - sizeseq
 - skipredundant
 - skipseq
 - splitsource
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - taxget
 - taxgetdown
 - taxgetrank
 - taxgetspecies
 - taxgetup
 - tcode
 - textget
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - trimspace
 - twofeat
 - union
 - urlget
 - variationget
 - vectorstrip
 - water
 - webpmux
 - webpng
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossdata
 - wossinput
 - wossname
 - wossoperation
 - wossoutput
 - wossparam
 - wosstopic
 - xmlget
 - xmltext
 - yank

This container was pulled from:

	https://quay.io/repository/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000006.000000006.*h.000000006.*debe.000000001.*e.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000006.000000006.*h.000000006.*debe.000000001.*e.*_.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-6.6.0--h6debe1e_0", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.", "URL: https://quay.io/repository/biocontainers/emboss",
          },
        },
        ["emboss/ctr-v6.5.7_cv1"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://hub.docker.com/r/biocontainers/emboss",
          Version = "ctr-v6.5.7_cv1",
          canonical = "ctr-v6.5.7_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-v6.5.7_cv1.lua",
          help = [[This is a module file for the container biocontainers/emboss:v6.5.7_cv1, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdgalaxy
 - acdlog
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - aligncopy
 - aligncopypair
 - antigenic
 - assemblyget
 - backtranambig
 - backtranseq
 - banana
 - biosed
 - btwisted
 - cachedas
 - cachedbfetch
 - cacheebeyesearch
 - cacheensembl
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - cons
 - consambig
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbtell
 - dbxcompress
 - dbxedam
 - dbxfasta
 - dbxflat
 - dbxgcg
 - dbxobo
 - dbxreport
 - dbxresource
 - dbxstat
 - dbxtax
 - dbxuncompress
 - degapseq
 - density
 - descseq
 - diffseq
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - drfinddata
 - drfindformat
 - drfindid
 - drfindresource
 - drget
 - drtext
 - edamdef
 - edamhasinput
 - edamhasoutput
 - edamisformat
 - edamisid
 - edamname
 - edialign
 - einverted
 - embossdata
 - embossupdate
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - eprimer32
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - featcopy
 - featmerge
 - featreport
 - feattext
 - findkm
 - freak
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - geecee
 - getorf
 - godef
 - goname
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoassembly
 - infobase
 - inforesidue
 - infoseq
 - isochore
 - jaspextract
 - jaspscan
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskambignuc
 - maskambigprot
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - needleall
 - newcpgreport
 - newcpgseek
 - newseq
 - nohtml
 - noreturn
 - nospace
 - notab
 - notseq
 - nthseq
 - nthseqset
 - octanol
 - oddcomp
 - ontocount
 - ontoget
 - ontogetcommon
 - ontogetdown
 - ontogetobsolete
 - ontogetroot
 - ontogetsibs
 - ontogetup
 - ontoisobsolete
 - ontotext
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepdigest
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - pscan
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - refseqget
 - remap
 - restover
 - restrict
 - revseq
 - runJemboss.sh
 - seealso
 - seqcount
 - seqmatchall
 - seqret
 - seqretsetall
 - seqretsplit
 - seqxref
 - seqxrefget
 - servertell
 - showalign
 - showdb
 - showfeat
 - showorf
 - showpep
 - showseq
 - showserver
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - sizeseq
 - skipredundant
 - skipseq
 - splitsource
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - taxget
 - taxgetdown
 - taxgetrank
 - taxgetspecies
 - taxgetup
 - tcode
 - textget
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - trimspace
 - twofeat
 - union
 - urlget
 - variationget
 - vectorstrip
 - water
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossdata
 - wossinput
 - wossname
 - wossoperation
 - wossoutput
 - wossparam
 - wosstopic
 - yank

This container was pulled from:

	https://hub.docker.com/r/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 15,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000006.000000005.000000007.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000006.000000005.000000007.*_.*cv.000000001.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-v6.5.7_cv1", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.", "URL: https://hub.docker.com/r/biocontainers/emboss",
          },
        },
        ["emboss/ctr-v6.5.7_cv2"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://hub.docker.com/r/biocontainers/emboss",
          Version = "ctr-v6.5.7_cv2",
          canonical = "ctr-v6.5.7_cv2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-v6.5.7_cv2.lua",
          help = [[This is a module file for the container biocontainers/emboss:v6.5.7_cv2, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdgalaxy
 - acdlog
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - aligncopy
 - aligncopypair
 - antigenic
 - assemblyget
 - backtranambig
 - backtranseq
 - banana
 - biosed
 - btwisted
 - cachedas
 - cachedbfetch
 - cacheebeyesearch
 - cacheensembl
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - cons
 - consambig
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbtell
 - dbxcompress
 - dbxedam
 - dbxfasta
 - dbxflat
 - dbxgcg
 - dbxobo
 - dbxreport
 - dbxresource
 - dbxstat
 - dbxtax
 - dbxuncompress
 - degapseq
 - density
 - descseq
 - diffseq
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - drfinddata
 - drfindformat
 - drfindid
 - drfindresource
 - drget
 - drtext
 - edamdef
 - edamhasinput
 - edamhasoutput
 - edamisformat
 - edamisid
 - edamname
 - edialign
 - einverted
 - embossdata
 - embossupdate
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - eprimer32
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - featcopy
 - featmerge
 - featreport
 - feattext
 - findkm
 - freak
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - geecee
 - getorf
 - godef
 - goname
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoassembly
 - infobase
 - inforesidue
 - infoseq
 - isochore
 - jaspextract
 - jaspscan
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskambignuc
 - maskambigprot
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - needleall
 - newcpgreport
 - newcpgseek
 - newseq
 - nohtml
 - noreturn
 - nospace
 - notab
 - notseq
 - nthseq
 - nthseqset
 - octanol
 - oddcomp
 - ontocount
 - ontoget
 - ontogetcommon
 - ontogetdown
 - ontogetobsolete
 - ontogetroot
 - ontogetsibs
 - ontogetup
 - ontoisobsolete
 - ontotext
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepdigest
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - pscan
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - refseqget
 - remap
 - restover
 - restrict
 - revseq
 - runJemboss.sh
 - seealso
 - seqcount
 - seqmatchall
 - seqret
 - seqretsetall
 - seqretsplit
 - seqxref
 - seqxrefget
 - servertell
 - showalign
 - showdb
 - showfeat
 - showorf
 - showpep
 - showseq
 - showserver
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - sizeseq
 - skipredundant
 - skipseq
 - splitsource
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - taxget
 - taxgetdown
 - taxgetrank
 - taxgetspecies
 - taxgetup
 - tcode
 - textget
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - trimspace
 - twofeat
 - union
 - urlget
 - variationget
 - vectorstrip
 - water
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossdata
 - wossinput
 - wossname
 - wossoperation
 - wossoutput
 - wossparam
 - wosstopic
 - yank

This container was pulled from:

	https://hub.docker.com/r/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 15,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000006.000000005.000000007.*_.*cv.000000002.*zfinal",
          wV = "*ctr.*v.000000006.000000005.000000007.*_.*cv.000000002.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-v6.5.7_cv2", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analagous to GCG; context-sensitive help for each tool.", "URL: https://hub.docker.com/r/biocontainers/emboss",
          },
        },
        ["emboss/ctr-v6.6.0dfsg-6-deb_cv1"] = {
          Category = "['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']",
          Description = "Diverse suite of tools for sequence analysis; many programs analogous to GCG; context-sensitive help for each tool.",
          Name = "emboss",
          URL = "https://hub.docker.com/r/biocontainers/emboss",
          Version = "ctr-v6.6.0dfsg-6-deb_cv1",
          canonical = "ctr-v6.6.0dfsg-6-deb_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emboss/ctr-v6.6.0dfsg-6-deb_cv1.lua",
          help = [[This is a module file for the container biocontainers/emboss:v6.6.0dfsg-6-deb_cv1, which exposes the
following programs:

 - aaindexextract
 - abiview
 - acdc
 - acdgalaxy
 - acdlog
 - acdpretty
 - acdtable
 - acdtrace
 - acdvalid
 - aligncopy
 - aligncopypair
 - antigenic
 - assemblyget
 - backtranambig
 - backtranseq
 - banana
 - biosed
 - btwisted
 - cachedas
 - cachedbfetch
 - cacheebeyesearch
 - cacheensembl
 - cai
 - chaos
 - charge
 - checktrans
 - chips
 - cirdna
 - codcmp
 - codcopy
 - coderet
 - compseq
 - consambig
 - cpgplot
 - cpgreport
 - cusp
 - cutgextract
 - cutseq
 - dan
 - dbiblast
 - dbifasta
 - dbiflat
 - dbigcg
 - dbtell
 - dbxcompress
 - dbxedam
 - dbxfasta
 - dbxflat
 - dbxgcg
 - dbxobo
 - dbxreport
 - dbxresource
 - dbxstat
 - dbxtax
 - dbxuncompress
 - degapseq
 - density
 - descseq
 - dialign2-2
 - diffseq
 - distmat
 - dotmatcher
 - dotpath
 - dottup
 - dreg
 - drfinddata
 - drfindformat
 - drfindid
 - drfindresource
 - drget
 - drtext
 - edamdef
 - edamhasinput
 - edamhasoutput
 - edamisformat
 - edamisid
 - edamname
 - edialign
 - einverted
 - em_cons
 - em_pscan
 - embossdata
 - embossupdate
 - embossversion
 - emma
 - emowse
 - entret
 - epestfind
 - eprimer3
 - eprimer32
 - equicktandem
 - est2genome
 - etandem
 - extractalign
 - extractfeat
 - extractseq
 - featcopy
 - featmerge
 - featreport
 - feattext
 - findkm
 - freak
 - fuzznuc
 - fuzzpro
 - fuzztran
 - garnier
 - geecee
 - getorf
 - godef
 - goname
 - helixturnhelix
 - hmoment
 - iep
 - infoalign
 - infoassembly
 - infobase
 - inforesidue
 - infoseq
 - isochore
 - jaspextract
 - jaspscan
 - jembossctl
 - lindna
 - listor
 - makenucseq
 - makeprotseq
 - marscan
 - maskambignuc
 - maskambigprot
 - maskfeat
 - maskseq
 - matcher
 - megamerger
 - merger
 - msbar
 - mwcontam
 - mwfilter
 - needle
 - needleall
 - newcpgreport
 - newcpgseek
 - newseq
 - nohtml
 - noreturn
 - nospace
 - notab
 - notseq
 - ntdpal
 - nthseq
 - nthseqset
 - ntthal
 - octanol
 - oddcomp
 - oligotm
 - ontocount
 - ontoget
 - ontogetcommon
 - ontogetdown
 - ontogetobsolete
 - ontogetroot
 - ontogetsibs
 - ontogetup
 - ontoisobsolete
 - ontotext
 - palindrome
 - pasteseq
 - patmatdb
 - patmatmotifs
 - pepcoil
 - pepdigest
 - pepinfo
 - pepnet
 - pepstats
 - pepwheel
 - pepwindow
 - pepwindowall
 - plotcon
 - plotorf
 - polydot
 - preg
 - prettyplot
 - prettyseq
 - primer3_core
 - primersearch
 - printsextract
 - profit
 - prophecy
 - prophet
 - prosextract
 - psiphi
 - rebaseextract
 - recoder
 - redata
 - refseqget
 - remap
 - restover
 - restrict
 - revseq
 - seealso
 - seqcount
 - seqmatchall
 - seqret
 - seqretsetall
 - seqretsplit
 - seqxref
 - seqxrefget
 - servertell
 - showalign
 - showdb
 - showfeat
 - showorf
 - showpep
 - showseq
 - showserver
 - shuffleseq
 - sigcleave
 - silent
 - sirna
 - sixpack
 - sizeseq
 - skipredundant
 - skipseq
 - splitsource
 - splitter
 - stretcher
 - stssearch
 - supermatcher
 - syco
 - taxget
 - taxgetdown
 - taxgetrank
 - taxgetspecies
 - taxgetup
 - tcode
 - textget
 - textsearch
 - tfextract
 - tfm
 - tfscan
 - tmap
 - tranalign
 - transeq
 - trimest
 - trimseq
 - trimspace
 - twofeat
 - union
 - urlget
 - variationget
 - vectorstrip
 - water
 - whichdb
 - wobble
 - wordcount
 - wordfinder
 - wordmatch
 - wossdata
 - wossinput
 - wossname
 - wossoperation
 - wossoutput
 - wossparam
 - wosstopic
 - xmlget
 - xmltext
 - yank

This container was pulled from:

	https://hub.docker.com/r/biocontainers/emboss

If you encounter errors in emboss or need help running the
tools it contains, please contact the developer at

	http://emboss.bioinformatics.nl/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 25,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000006.000000006.*dfsg.*zfinal-.000000006.*deb.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000006.000000006.*dfsg.*zfinal-.000000006.*deb.*_.*cv.000000001.*zfinal",
          whatis = {
            "Name: emboss", "Version: ctr-v6.6.0dfsg-6-deb_cv1", "Category: ['Sequence analysis', 'Local alignment', 'Sequence alignment analysis', 'Global alignment', 'Sequence alignment']", "Keywords: ['Molecular biology', 'Sequence analysis', 'Biology']", "Description: Diverse suite of tools for sequence analysis; many programs analogous to GCG; context-sensitive help for each tool.", "URL: https://hub.docker.com/r/biocontainers/emboss",
          },
        },
      },
    },
    emirge = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["emirge/ctr-0.61.1--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The emirge package",
          Name = "emirge",
          URL = "https://quay.io/repository/biocontainers/emirge",
          Version = "ctr-0.61.1--py27_0",
          canonical = "ctr-0.61.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emirge/ctr-0.61.1--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emirge:0.61.1--py27_0, which exposes the
following programs:

 - aclocal.bak
 - automake.bak
 - bcftools
 - bmp2tiff
 - bowtie
 - bowtie-align-l
 - bowtie-align-s
 - bowtie-build
 - bowtie-build-l
 - bowtie-build-s
 - bowtie-inspect
 - bowtie-inspect-l
 - bowtie-inspect-s
 - color-chrs.pl
 - createfontdatachunk.py
 - emirge.py
 - emirge_amplicon.py
 - emirge_makedb.py
 - emirge_rename_fasta.py
 - enhancer.py
 - explode.py
 - gif2tiff
 - gifmaker.py
 - painter.py
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - plot-vcfstats
 - ras2tiff
 - rgb2ycbcr
 - sample
 - samtools
 - thresholder.py
 - thumbnail
 - vcfutils.pl
 - viewer.py
 - vsearch

This container was pulled from:

	https://quay.io/repository/biocontainers/emirge

If you encounter errors in emirge or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/emirge

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000061.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000061.000000001.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: emirge", "Version: ctr-0.61.1--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The emirge package", "URL: https://quay.io/repository/biocontainers/emirge",
          },
        },
        ["emirge/ctr-0.61.1--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The emirge package",
          Name = "emirge",
          URL = "https://quay.io/repository/biocontainers/emirge",
          Version = "ctr-0.61.1--py27_1",
          canonical = "ctr-0.61.1--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emirge/ctr-0.61.1--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emirge:0.61.1--py27_1, which exposes the
following programs:

 - aclocal.bak
 - autoheader.bak
 - autom4te.bak
 - automake.bak
 - autoreconf.bak
 - autoscan.bak
 - autoupdate.bak
 - bcftools
 - bowtie
 - bowtie-align-l
 - bowtie-align-s
 - bowtie-build
 - bowtie-build-l
 - bowtie-build-s
 - bowtie-inspect
 - bowtie-inspect-l
 - bowtie-inspect-s
 - color-chrs.pl
 - createfontdatachunk.py
 - emirge.py
 - emirge_amplicon.py
 - emirge_makedb.py
 - emirge_rename_fasta.py
 - enhancer.py
 - explode.py
 - gifmaker.py
 - guess-ploidy.py
 - ifnames.bak
 - ksu
 - painter.py
 - perl5.22.0
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - plot-roh.py
 - plot-vcfstats
 - run-roh.pl
 - sample
 - samtools
 - thresholder.py
 - vcfutils.pl
 - viewer.py
 - vsearch

This container was pulled from:

	https://quay.io/repository/biocontainers/emirge

If you encounter errors in emirge or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/emirge

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000061.000000001.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000061.000000001.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: emirge", "Version: ctr-0.61.1--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The emirge package", "URL: https://quay.io/repository/biocontainers/emirge",
          },
        },
        ["emirge/ctr-0.61.1--py27h355e19c_2"] = {
          Category = "['Bioinformatics']",
          Description = "The emirge package",
          Name = "emirge",
          URL = "https://quay.io/repository/biocontainers/emirge",
          Version = "ctr-0.61.1--py27h355e19c_2",
          canonical = "ctr-0.61.1--py27h355e19c_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emirge/ctr-0.61.1--py27h355e19c_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emirge:0.61.1--py27h355e19c_2, which exposes the
following programs:

 - bcftools
 - bowtie
 - bowtie-align-l
 - bowtie-align-s
 - bowtie-build
 - bowtie-build-l
 - bowtie-build-s
 - bowtie-inspect
 - bowtie-inspect-l
 - bowtie-inspect-s
 - color-chrs.pl
 - emirge.py
 - emirge_amplicon.py
 - emirge_makedb.py
 - emirge_rename_fasta.py
 - guess-ploidy.py
 - perl5.26.2
 - plot-roh.py
 - plot-vcfstats
 - run-roh.pl
 - sample
 - samtools
 - vcfutils.pl
 - vsearch

This container was pulled from:

	https://quay.io/repository/biocontainers/emirge

If you encounter errors in emirge or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/emirge

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000061.000000001.*y.000000027.*h.000000355.*e.000000019.*c.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000061.000000001.*y.000000027.*h.000000355.*e.000000019.*c.*_.000000002.*zfinal",
          whatis = {
            "Name: emirge", "Version: ctr-0.61.1--py27h355e19c_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The emirge package", "URL: https://quay.io/repository/biocontainers/emirge",
          },
        },
      },
    },
    emmix = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["emmix/ctr-1.3--0"] = {
          Category = "['Bioinformatics']",
          Description = "The emmix package",
          Name = "emmix",
          URL = "https://quay.io/repository/biocontainers/emmix",
          Version = "ctr-1.3--0",
          canonical = "ctr-1.3--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emmix/ctr-1.3--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emmix:1.3--0, which exposes the
following programs:

 - EMMIX

This container was pulled from:

	https://quay.io/repository/biocontainers/emmix

If you encounter errors in emmix or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/emmix

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.*zfinal",
          whatis = {
            "Name: emmix", "Version: ctr-1.3--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The emmix package", "URL: https://quay.io/repository/biocontainers/emmix",
          },
        },
        ["emmix/ctr-1.3--1"] = {
          Category = "['Bioinformatics']",
          Description = "The emmix package",
          Name = "emmix",
          URL = "https://quay.io/repository/biocontainers/emmix",
          Version = "ctr-1.3--1",
          canonical = "ctr-1.3--1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emmix/ctr-1.3--1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emmix:1.3--1, which exposes the
following programs:

 - EMMIX

This container was pulled from:

	https://quay.io/repository/biocontainers/emmix

If you encounter errors in emmix or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/emmix

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: emmix", "Version: ctr-1.3--1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The emmix package", "URL: https://quay.io/repository/biocontainers/emmix",
          },
        },
        ["emmix/ctr-1.3--h470a237_2"] = {
          Category = "['Bioinformatics']",
          Description = "The emmix package",
          Name = "emmix",
          URL = "https://quay.io/repository/biocontainers/emmix",
          Version = "ctr-1.3--h470a237_2",
          canonical = "ctr-1.3--h470a237_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emmix/ctr-1.3--h470a237_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emmix:1.3--h470a237_2, which exposes the
following programs:

 - EMMIX

This container was pulled from:

	https://quay.io/repository/biocontainers/emmix

If you encounter errors in emmix or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/emmix

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.*h.000000470.*a.000000237.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.*h.000000470.*a.000000237.*_.000000002.*zfinal",
          whatis = {
            "Name: emmix", "Version: ctr-1.3--h470a237_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The emmix package", "URL: https://quay.io/repository/biocontainers/emmix",
          },
        },
      },
    },
    emperor = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["emperor/ctr-0.9.3--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The emperor package",
          Name = "emperor",
          URL = "https://quay.io/repository/biocontainers/emperor",
          Version = "ctr-0.9.3--py27_0",
          canonical = "ctr-0.9.3--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emperor/ctr-0.9.3--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emperor:0.9.3--py27_0, which exposes the
following programs:

 - make_emperor.py
 - qcli_make_rst
 - qcli_make_script

This container was pulled from:

	https://quay.io/repository/biocontainers/emperor

If you encounter errors in emperor or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/emperor

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000003.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000003.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: emperor", "Version: ctr-0.9.3--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The emperor package", "URL: https://quay.io/repository/biocontainers/emperor",
          },
        },
        ["emperor/ctr-0.9.3--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The emperor package",
          Name = "emperor",
          URL = "https://quay.io/repository/biocontainers/emperor",
          Version = "ctr-0.9.3--py27_1",
          canonical = "ctr-0.9.3--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/emperor/ctr-0.9.3--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/emperor:0.9.3--py27_1, which exposes the
following programs:

 - make_emperor.py
 - qcli_make_rst
 - qcli_make_script

This container was pulled from:

	https://quay.io/repository/biocontainers/emperor

If you encounter errors in emperor or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/emperor

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000003.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000003.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: emperor", "Version: ctr-0.9.3--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The emperor package", "URL: https://quay.io/repository/biocontainers/emperor",
          },
        },
      },
    },
    enabrowsertools = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["enabrowsertools/ctr-1.5.4--0"] = {
          Category = "['Bioinformatics']",
          Description = "The enabrowsertools package",
          Name = "enabrowsertools",
          URL = "https://quay.io/repository/biocontainers/enabrowsertools",
          Version = "ctr-1.5.4--0",
          canonical = "ctr-1.5.4--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enabrowsertools/ctr-1.5.4--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enabrowsertools:1.5.4--0, which exposes the
following programs:

 - 2to3-3.7
 - assemblyGet.py
 - enaDataGet
 - enaDataGet.py
 - enaGroupGet
 - enaGroupGet.py
 - idle3.7
 - ncurses6-config
 - pydoc3.7
 - python3.7
 - python3.7-config
 - python3.7m
 - python3.7m-config
 - pyvenv-3.7
 - readGet.py
 - sequenceGet.py
 - utils.py
 - utils_py2.py

This container was pulled from:

	https://quay.io/repository/biocontainers/enabrowsertools

If you encounter errors in enabrowsertools or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enabrowsertools

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000005.000000004.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000005.000000004.*zfinal",
          whatis = {
            "Name: enabrowsertools", "Version: ctr-1.5.4--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enabrowsertools package", "URL: https://quay.io/repository/biocontainers/enabrowsertools",
          },
        },
      },
    },
    enasearch = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["enasearch/ctr-0.0.4--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.0.4--py27_0",
          canonical = "ctr-0.0.4--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.0.4--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.0.4--py27_0, which exposes the
following programs:

 - bmp2tiff
 - createfontdatachunk.py
 - enasearch
 - enhancer.py
 - explode.py
 - flake8
 - gif2tiff
 - gifmaker.py
 - painter.py
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - pycodestyle
 - pyflakes
 - ras2tiff
 - rgb2ycbcr
 - sample
 - thresholder.py
 - thumbnail
 - viewer.py

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000004.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000004.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.0.4--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
        ["enasearch/ctr-0.0.4--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.0.4--py27_1",
          canonical = "ctr-0.0.4--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.0.4--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.0.4--py27_1, which exposes the
following programs:

 - enasearch
 - flake8
 - pycodestyle
 - pyflakes
 - sample

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000004.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000004.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.0.4--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
        ["enasearch/ctr-0.0.5--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.0.5--py27_0",
          canonical = "ctr-0.0.5--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.0.5--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.0.5--py27_0, which exposes the
following programs:

 - bmp2tiff
 - createfontdatachunk.py
 - enasearch
 - enhancer.py
 - explode.py
 - flake8
 - gif2tiff
 - gifmaker.py
 - painter.py
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - pycodestyle
 - pyflakes
 - ras2tiff
 - rgb2ycbcr
 - sample
 - thresholder.py
 - thumbnail
 - viewer.py

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000005.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000005.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.0.5--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
        ["enasearch/ctr-0.0.5--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.0.5--py27_1",
          canonical = "ctr-0.0.5--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.0.5--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.0.5--py27_1, which exposes the
following programs:

 - enasearch
 - flake8
 - pycodestyle
 - pyflakes
 - sample

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000005.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000005.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.0.5--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
        ["enasearch/ctr-0.0.6--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.0.6--py27_0",
          canonical = "ctr-0.0.6--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.0.6--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.0.6--py27_0, which exposes the
following programs:

 - bmp2tiff
 - createfontdatachunk.py
 - enasearch
 - enhancer.py
 - explode.py
 - flake8
 - gif2tiff
 - gifmaker.py
 - painter.py
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - pycodestyle
 - pyflakes
 - ras2tiff
 - rgb2ycbcr
 - sample
 - thresholder.py
 - thumbnail
 - viewer.py

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000006.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000006.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.0.6--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
        ["enasearch/ctr-0.0.6--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.0.6--py27_1",
          canonical = "ctr-0.0.6--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.0.6--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.0.6--py27_1, which exposes the
following programs:

 - enasearch
 - flake8
 - pycodestyle
 - pyflakes
 - sample

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000006.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000006.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.0.6--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
        ["enasearch/ctr-0.1.1--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.1.1--py27_0",
          canonical = "ctr-0.1.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.1.1--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.1.1--py27_0, which exposes the
following programs:

 - bmp2tiff
 - createfontdatachunk.py
 - enasearch
 - enhancer.py
 - explode.py
 - flake8
 - gif2tiff
 - gifmaker.py
 - painter.py
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - pycodestyle
 - pyflakes
 - ras2tiff
 - rgb2ycbcr
 - sample
 - thresholder.py
 - thumbnail
 - viewer.py

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000001.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.1.1--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
        ["enasearch/ctr-0.1.1--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.1.1--py27_1",
          canonical = "ctr-0.1.1--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.1.1--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.1.1--py27_1, which exposes the
following programs:

 - enasearch
 - flake8
 - pycodestyle
 - pyflakes
 - sample

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000001.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000001.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.1.1--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
        ["enasearch/ctr-0.2.2--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.2.2--py27_0",
          canonical = "ctr-0.2.2--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.2.2--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.2.2--py27_0, which exposes the
following programs:

 - enasearch
 - enhancer.py
 - explode.py
 - flake8
 - gifmaker.py
 - painter.py
 - pilconvert.py
 - pildriver.py
 - pilfile.py
 - pilfont.py
 - pilprint.py
 - player.py
 - pycodestyle
 - pyflakes
 - sample
 - thresholder.py
 - viewer.py

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.2.2--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
        ["enasearch/ctr-0.2.2--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The enasearch package",
          Name = "enasearch",
          URL = "https://quay.io/repository/biocontainers/enasearch",
          Version = "ctr-0.2.2--py27_1",
          canonical = "ctr-0.2.2--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/enasearch/ctr-0.2.2--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/enasearch:0.2.2--py27_1, which exposes the
following programs:

 - enasearch
 - flake8
 - pycodestyle
 - pyflakes
 - sample

This container was pulled from:

	https://quay.io/repository/biocontainers/enasearch

If you encounter errors in enasearch or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/enasearch

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: enasearch", "Version: ctr-0.2.2--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The enasearch package", "URL: https://quay.io/repository/biocontainers/enasearch",
          },
        },
      },
    },
    eoulsan = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["eoulsan/ctr-2.0_beta4--1"] = {
          Category = "['Bioinformatics']",
          Description = "The eoulsan package",
          Name = "eoulsan",
          URL = "https://quay.io/repository/biocontainers/eoulsan",
          Version = "ctr-2.0_beta4--1",
          canonical = "ctr-2.0_beta4--1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eoulsan/ctr-2.0_beta4--1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eoulsan:2.0_beta4--1, which exposes the
following programs:

 - .java-jdk-post-link.sh
 - .java-jdk-pre-unlink.sh
 - appletviewer
 - eoulsan.sh
 - extcheck
 - idlj
 - jar
 - jarsigner
 - java-rmi.cgi
 - javac
 - javadoc
 - javah
 - javap
 - jcmd
 - jconsole
 - jdb
 - jdeps
 - jhat
 - jinfo
 - jmap
 - jps
 - jrunscript
 - jsadebugd
 - jstack
 - jstat
 - jstatd
 - native2ascii
 - rmic
 - schemagen
 - serialver
 - wsgen
 - wsimport
 - xjc

This container was pulled from:

	https://quay.io/repository/biocontainers/eoulsan

If you encounter errors in eoulsan or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eoulsan

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 17,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.*_.*beta.000000004.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000002.*_.*beta.000000004.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: eoulsan", "Version: ctr-2.0_beta4--1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eoulsan package", "URL: https://quay.io/repository/biocontainers/eoulsan",
          },
        },
        ["eoulsan/ctr-2.3--0"] = {
          Category = "['Bioinformatics']",
          Description = "The eoulsan package",
          Name = "eoulsan",
          URL = "https://quay.io/repository/biocontainers/eoulsan",
          Version = "ctr-2.3--0",
          canonical = "ctr-2.3--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eoulsan/ctr-2.3--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eoulsan:2.3--0, which exposes the
following programs:

 - appletviewer
 - eoulsan.sh
 - extcheck
 - idlj
 - jar
 - jarsigner
 - java-rmi.cgi
 - javac
 - javadoc
 - javah
 - javap
 - jcmd
 - jconsole
 - jdb
 - jdeps
 - jhat
 - jinfo
 - jmap
 - jps
 - jrunscript
 - jsadebugd
 - jstack
 - jstat
 - jstatd
 - native2ascii
 - rmic
 - schemagen
 - serialver
 - wsgen
 - wsimport
 - xjc

This container was pulled from:

	https://quay.io/repository/biocontainers/eoulsan

If you encounter errors in eoulsan or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eoulsan

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000003.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000003.*zfinal",
          whatis = {
            "Name: eoulsan", "Version: ctr-2.3--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eoulsan package", "URL: https://quay.io/repository/biocontainers/eoulsan",
          },
        },
      },
    },
    ephemeris = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ephemeris/ctr-0.8.0--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ephemeris package",
          Name = "ephemeris",
          URL = "https://quay.io/repository/biocontainers/ephemeris",
          Version = "ctr-0.8.0--py27_0",
          canonical = "ctr-0.8.0--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ephemeris/ctr-0.8.0--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ephemeris:0.8.0--py27_0, which exposes the
following programs:

 - asadmin
 - bioblend-galaxy-tests
 - bundle_image
 - cfadmin
 - cq
 - cwutil
 - dynamodb_dump
 - dynamodb_load
 - elbadmin
 - fetch_file
 - galaxy-wait
 - get-tool-list
 - glacier
 - instance_events
 - kill_instance
 - launch_instance
 - list_instances
 - lss3
 - mturk
 - pyami_sendmail
 - route53
 - run-data-managers
 - s3put
 - sdbadmin
 - setup-data-libraries
 - shed-tools
 - taskadmin
 - workflow-install
 - workflow-to-tools

This container was pulled from:

	https://quay.io/repository/biocontainers/ephemeris

If you encounter errors in ephemeris or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ephemeris

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000008.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000008.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: ephemeris", "Version: ctr-0.8.0--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ephemeris package", "URL: https://quay.io/repository/biocontainers/ephemeris",
          },
        },
        ["ephemeris/ctr-0.8.0--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ephemeris package",
          Name = "ephemeris",
          URL = "https://quay.io/repository/biocontainers/ephemeris",
          Version = "ctr-0.8.0--py35_0",
          canonical = "ctr-0.8.0--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ephemeris/ctr-0.8.0--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ephemeris:0.8.0--py35_0, which exposes the
following programs:

 - asadmin
 - bioblend-galaxy-tests
 - bundle_image
 - cfadmin
 - cq
 - cwutil
 - dynamodb_dump
 - dynamodb_load
 - elbadmin
 - fetch_file
 - galaxy-wait
 - get-tool-list
 - glacier
 - idle3.5
 - instance_events
 - kill_instance
 - launch_instance
 - list_instances
 - lss3
 - mturk
 - pyami_sendmail
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - route53
 - run-data-managers
 - s3put
 - sdbadmin
 - setup-data-libraries
 - shed-tools
 - taskadmin
 - workflow-install
 - workflow-to-tools

This container was pulled from:

	https://quay.io/repository/biocontainers/ephemeris

If you encounter errors in ephemeris or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ephemeris

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000008.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000008.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: ephemeris", "Version: ctr-0.8.0--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ephemeris package", "URL: https://quay.io/repository/biocontainers/ephemeris",
          },
        },
        ["ephemeris/ctr-0.8.0--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ephemeris package",
          Name = "ephemeris",
          URL = "https://quay.io/repository/biocontainers/ephemeris",
          Version = "ctr-0.8.0--py36_0",
          canonical = "ctr-0.8.0--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ephemeris/ctr-0.8.0--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ephemeris:0.8.0--py36_0, which exposes the
following programs:

 - asadmin
 - bioblend-galaxy-tests
 - bundle_image
 - cfadmin
 - cq
 - cwutil
 - dynamodb_dump
 - dynamodb_load
 - elbadmin
 - fetch_file
 - galaxy-wait
 - get-tool-list
 - glacier
 - instance_events
 - kill_instance
 - launch_instance
 - list_instances
 - lss3
 - mturk
 - pyami_sendmail
 - route53
 - run-data-managers
 - s3put
 - sdbadmin
 - setup-data-libraries
 - shed-tools
 - taskadmin
 - workflow-install
 - workflow-to-tools

This container was pulled from:

	https://quay.io/repository/biocontainers/ephemeris

If you encounter errors in ephemeris or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ephemeris

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000008.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000008.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: ephemeris", "Version: ctr-0.8.0--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ephemeris package", "URL: https://quay.io/repository/biocontainers/ephemeris",
          },
        },
        ["ephemeris/ctr-0.8.0--py_2"] = {
          Category = "['Bioinformatics']",
          Description = "The ephemeris package",
          Name = "ephemeris",
          URL = "https://quay.io/repository/biocontainers/ephemeris",
          Version = "ctr-0.8.0--py_2",
          canonical = "ctr-0.8.0--py_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ephemeris/ctr-0.8.0--py_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ephemeris:0.8.0--py_2, which exposes the
following programs:

 - asadmin
 - bioblend-galaxy-tests
 - bundle_image
 - cfadmin
 - cq
 - cwutil
 - dynamodb_dump
 - dynamodb_load
 - elbadmin
 - fetch_file
 - galaxy-wait
 - get-tool-list
 - glacier
 - instance_events
 - kill_instance
 - launch_instance
 - list_instances
 - lss3
 - mturk
 - pyami_sendmail
 - route53
 - run-data-managers
 - s3put
 - sdbadmin
 - setup-data-libraries
 - shed-tools
 - taskadmin
 - workflow-install
 - workflow-to-tools

This container was pulled from:

	https://quay.io/repository/biocontainers/ephemeris

If you encounter errors in ephemeris or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ephemeris

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 16,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000008.*y.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000008.*y.*_.000000002.*zfinal",
          whatis = {
            "Name: ephemeris", "Version: ctr-0.8.0--py_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ephemeris package", "URL: https://quay.io/repository/biocontainers/ephemeris",
          },
        },
      },
    },
    epic = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["epic/ctr-0.1.17--py27_0"] = {
          Version = "ctr-0.1.17--py27_0",
          canonical = "ctr-0.1.17--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.17--py27_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000017.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000017.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.1.17--py34_0"] = {
          Version = "ctr-0.1.17--py34_0",
          canonical = "ctr-0.1.17--py34_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.17--py34_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000017.*y.000000034.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000017.*y.000000034.*_.*zfinal",
        },
        ["epic/ctr-0.1.17--py35_0"] = {
          Version = "ctr-0.1.17--py35_0",
          canonical = "ctr-0.1.17--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.17--py35_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000017.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000017.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.1.20--py35_0"] = {
          Version = "ctr-0.1.20--py35_0",
          canonical = "ctr-0.1.20--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.20--py35_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000020.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000020.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.1.24--py27_0"] = {
          Version = "ctr-0.1.24--py27_0",
          canonical = "ctr-0.1.24--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.24--py27_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000024.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000024.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.1.24--py34_0"] = {
          Version = "ctr-0.1.24--py34_0",
          canonical = "ctr-0.1.24--py34_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.24--py34_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000024.*y.000000034.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000024.*y.000000034.*_.*zfinal",
        },
        ["epic/ctr-0.1.24--py35_0"] = {
          Version = "ctr-0.1.24--py35_0",
          canonical = "ctr-0.1.24--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.24--py35_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000024.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000024.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.1.24--py36_0"] = {
          Version = "ctr-0.1.24--py36_0",
          canonical = "ctr-0.1.24--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.24--py36_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000024.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000024.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.1.25--py27_0"] = {
          Version = "ctr-0.1.25--py27_0",
          canonical = "ctr-0.1.25--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.25--py27_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000025.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000025.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.1.25--py35_0"] = {
          Version = "ctr-0.1.25--py35_0",
          canonical = "ctr-0.1.25--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.25--py35_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000025.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000025.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.1.25--py36_0"] = {
          Version = "ctr-0.1.25--py36_0",
          canonical = "ctr-0.1.25--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.25--py36_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000025.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000025.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.1.26--py35_0"] = {
          Version = "ctr-0.1.26--py35_0",
          canonical = "ctr-0.1.26--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.26--py35_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000026.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000026.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.1.26--py36_0"] = {
          Version = "ctr-0.1.26--py36_0",
          canonical = "ctr-0.1.26--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.26--py36_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000026.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000026.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.1.27--py27_0"] = {
          Version = "ctr-0.1.27--py27_0",
          canonical = "ctr-0.1.27--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.27--py27_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000027.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000027.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.1.27--py35_0"] = {
          Version = "ctr-0.1.27--py35_0",
          canonical = "ctr-0.1.27--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.27--py35_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000027.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000027.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.1.27--py36_0"] = {
          Version = "ctr-0.1.27--py36_0",
          canonical = "ctr-0.1.27--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.27--py36_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000027.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000027.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.1.28--py27_0"] = {
          Version = "ctr-0.1.28--py27_0",
          canonical = "ctr-0.1.28--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.28--py27_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000028.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000028.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.1.28--py35_0"] = {
          Version = "ctr-0.1.28--py35_0",
          canonical = "ctr-0.1.28--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.28--py35_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000028.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000028.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.1.28--py36_0"] = {
          Version = "ctr-0.1.28--py36_0",
          canonical = "ctr-0.1.28--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.1.28--py36_0.lua",
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.000000028.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.000000028.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.2.1--py27_0"] = {
          Version = "ctr-0.2.1--py27_0",
          canonical = "ctr-0.2.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.1--py27_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.2.1--py35_0"] = {
          Version = "ctr-0.2.1--py35_0",
          canonical = "ctr-0.2.1--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.1--py35_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.2.1--py36_0"] = {
          Version = "ctr-0.2.1--py36_0",
          canonical = "ctr-0.2.1--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.1--py36_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.2.12--py27h24bf2e0_0"] = {
          Version = "ctr-0.2.12--py27h24bf2e0_0",
          canonical = "ctr-0.2.12--py27h24bf2e0_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.12--py27h24bf2e0_0.lua",
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000027.*h.000000024.*bf.000000002.*e.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000027.*h.000000024.*bf.000000002.*e.*_.*zfinal",
        },
        ["epic/ctr-0.2.12--py27h24bf2e0_1"] = {
          Version = "ctr-0.2.12--py27h24bf2e0_1",
          canonical = "ctr-0.2.12--py27h24bf2e0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.12--py27h24bf2e0_1.lua",
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
        },
        ["epic/ctr-0.2.12--py35h24bf2e0_0"] = {
          Version = "ctr-0.2.12--py35h24bf2e0_0",
          canonical = "ctr-0.2.12--py35h24bf2e0_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.12--py35h24bf2e0_0.lua",
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000035.*h.000000024.*bf.000000002.*e.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000035.*h.000000024.*bf.000000002.*e.*_.*zfinal",
        },
        ["epic/ctr-0.2.12--py35h24bf2e0_1"] = {
          Version = "ctr-0.2.12--py35h24bf2e0_1",
          canonical = "ctr-0.2.12--py35h24bf2e0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.12--py35h24bf2e0_1.lua",
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000035.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000035.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
        },
        ["epic/ctr-0.2.12--py36h24bf2e0_0"] = {
          Version = "ctr-0.2.12--py36h24bf2e0_0",
          canonical = "ctr-0.2.12--py36h24bf2e0_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.12--py36h24bf2e0_0.lua",
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000036.*h.000000024.*bf.000000002.*e.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000036.*h.000000024.*bf.000000002.*e.*_.*zfinal",
        },
        ["epic/ctr-0.2.12--py36h24bf2e0_1"] = {
          Version = "ctr-0.2.12--py36h24bf2e0_1",
          canonical = "ctr-0.2.12--py36h24bf2e0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.12--py36h24bf2e0_1.lua",
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000036.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000012.*y.000000036.*h.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
        },
        ["epic/ctr-0.2.2--py27_0"] = {
          Version = "ctr-0.2.2--py27_0",
          canonical = "ctr-0.2.2--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.2--py27_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.2.2--py35_0"] = {
          Version = "ctr-0.2.2--py35_0",
          canonical = "ctr-0.2.2--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.2--py35_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.2.2--py36_0"] = {
          Version = "ctr-0.2.2--py36_0",
          canonical = "ctr-0.2.2--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.2--py36_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000002.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.2.4--py27_0"] = {
          Version = "ctr-0.2.4--py27_0",
          canonical = "ctr-0.2.4--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.4--py27_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000004.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000004.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.2.4--py35_0"] = {
          Version = "ctr-0.2.4--py35_0",
          canonical = "ctr-0.2.4--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.4--py35_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000004.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000004.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.2.4--py36_0"] = {
          Version = "ctr-0.2.4--py36_0",
          canonical = "ctr-0.2.4--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.4--py36_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000004.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000004.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.2.5--py27_0"] = {
          Version = "ctr-0.2.5--py27_0",
          canonical = "ctr-0.2.5--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.5--py27_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000005.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000005.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.2.5--py35_0"] = {
          Version = "ctr-0.2.5--py35_0",
          canonical = "ctr-0.2.5--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.5--py35_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000005.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000005.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.2.5--py36_0"] = {
          Version = "ctr-0.2.5--py36_0",
          canonical = "ctr-0.2.5--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.5--py36_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000005.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000005.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.2.7--py27_0"] = {
          Version = "ctr-0.2.7--py27_0",
          canonical = "ctr-0.2.7--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.7--py27_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000007.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000007.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.2.7--py35_0"] = {
          Version = "ctr-0.2.7--py35_0",
          canonical = "ctr-0.2.7--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.7--py35_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000007.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000007.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.2.7--py36_0"] = {
          Version = "ctr-0.2.7--py36_0",
          canonical = "ctr-0.2.7--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.7--py36_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000007.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000007.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.2.8--py27_0"] = {
          Version = "ctr-0.2.8--py27_0",
          canonical = "ctr-0.2.8--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.8--py27_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000008.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000008.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.2.8--py35_0"] = {
          Version = "ctr-0.2.8--py35_0",
          canonical = "ctr-0.2.8--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.8--py35_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000008.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000008.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.2.8--py36_0"] = {
          Version = "ctr-0.2.8--py36_0",
          canonical = "ctr-0.2.8--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.8--py36_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000008.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000008.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.2.9--py27_0"] = {
          Version = "ctr-0.2.9--py27_0",
          canonical = "ctr-0.2.9--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.9--py27_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000009.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000009.*y.000000027.*_.*zfinal",
        },
        ["epic/ctr-0.2.9--py35_0"] = {
          Version = "ctr-0.2.9--py35_0",
          canonical = "ctr-0.2.9--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.9--py35_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000009.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000009.*y.000000035.*_.*zfinal",
        },
        ["epic/ctr-0.2.9--py36_0"] = {
          Version = "ctr-0.2.9--py36_0",
          canonical = "ctr-0.2.9--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.9--py36_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000009.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000009.*y.000000036.*_.*zfinal",
        },
        ["epic/ctr-0.2.9--pyh24bf2e0_1"] = {
          Version = "ctr-0.2.9--pyh24bf2e0_1",
          canonical = "ctr-0.2.9--pyh24bf2e0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic/ctr-0.2.9--pyh24bf2e0_1.lua",
          luaExt = 24,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000009.*yh.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000009.*yh.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
        },
      },
    },
    epic2 = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["epic2/ctr-0.0.15--py27h4e20536_0"] = {
          Category = "['Bioinformatics']",
          Description = "The epic2 package",
          Name = "epic2",
          URL = "https://quay.io/repository/biocontainers/epic2",
          Version = "ctr-0.0.15--py27h4e20536_0",
          canonical = "ctr-0.0.15--py27h4e20536_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic2/ctr-0.0.15--py27h4e20536_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/epic2:0.0.15--py27h4e20536_0, which exposes the
following programs:

 - bcftools
 - color-chrs.pl
 - cygdb
 - cython
 - cythonize
 - epic2
 - guess-ploidy.py
 - natsort
 - ncurses6-config
 - plot-roh.py
 - plot-vcfstats
 - run-roh.pl
 - samtools
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/epic2

If you encounter errors in epic2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/epic2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000015.*y.000000027.*h.000000004.*e.000020536.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000015.*y.000000027.*h.000000004.*e.000020536.*_.*zfinal",
          whatis = {
            "Name: epic2", "Version: ctr-0.0.15--py27h4e20536_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The epic2 package", "URL: https://quay.io/repository/biocontainers/epic2",
          },
        },
        ["epic2/ctr-0.0.15--py35h4e20536_0"] = {
          Category = "['Bioinformatics']",
          Description = "The epic2 package",
          Name = "epic2",
          URL = "https://quay.io/repository/biocontainers/epic2",
          Version = "ctr-0.0.15--py35h4e20536_0",
          canonical = "ctr-0.0.15--py35h4e20536_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic2/ctr-0.0.15--py35h4e20536_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/epic2:0.0.15--py35h4e20536_0, which exposes the
following programs:

 - bcftools
 - color-chrs.pl
 - cygdb
 - cython
 - cythonize
 - epic2
 - guess-ploidy.py
 - idle3.5
 - natsort
 - ncurses6-config
 - plot-roh.py
 - plot-vcfstats
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - run-roh.pl
 - samtools
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/epic2

If you encounter errors in epic2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/epic2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000015.*y.000000035.*h.000000004.*e.000020536.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000015.*y.000000035.*h.000000004.*e.000020536.*_.*zfinal",
          whatis = {
            "Name: epic2", "Version: ctr-0.0.15--py35h4e20536_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The epic2 package", "URL: https://quay.io/repository/biocontainers/epic2",
          },
        },
        ["epic2/ctr-0.0.15--py36h4e20536_0"] = {
          Category = "['Bioinformatics']",
          Description = "The epic2 package",
          Name = "epic2",
          URL = "https://quay.io/repository/biocontainers/epic2",
          Version = "ctr-0.0.15--py36h4e20536_0",
          canonical = "ctr-0.0.15--py36h4e20536_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic2/ctr-0.0.15--py36h4e20536_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/epic2:0.0.15--py36h4e20536_0, which exposes the
following programs:

 - bcftools
 - color-chrs.pl
 - cygdb
 - cython
 - cythonize
 - epic2
 - guess-ploidy.py
 - natsort
 - ncurses6-config
 - plot-roh.py
 - plot-vcfstats
 - run-roh.pl
 - samtools
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/epic2

If you encounter errors in epic2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/epic2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000015.*y.000000036.*h.000000004.*e.000020536.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000015.*y.000000036.*h.000000004.*e.000020536.*_.*zfinal",
          whatis = {
            "Name: epic2", "Version: ctr-0.0.15--py36h4e20536_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The epic2 package", "URL: https://quay.io/repository/biocontainers/epic2",
          },
        },
        ["epic2/ctr-0.0.16--py27h4e20536_0"] = {
          Category = "['Bioinformatics']",
          Description = "The epic2 package",
          Name = "epic2",
          URL = "https://quay.io/repository/biocontainers/epic2",
          Version = "ctr-0.0.16--py27h4e20536_0",
          canonical = "ctr-0.0.16--py27h4e20536_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic2/ctr-0.0.16--py27h4e20536_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/epic2:0.0.16--py27h4e20536_0, which exposes the
following programs:

 - bcftools
 - color-chrs.pl
 - cygdb
 - cython
 - cythonize
 - epic2
 - guess-ploidy.py
 - natsort
 - ncurses6-config
 - plot-roh.py
 - plot-vcfstats
 - run-roh.pl
 - samtools
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/epic2

If you encounter errors in epic2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/epic2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000016.*y.000000027.*h.000000004.*e.000020536.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000016.*y.000000027.*h.000000004.*e.000020536.*_.*zfinal",
          whatis = {
            "Name: epic2", "Version: ctr-0.0.16--py27h4e20536_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The epic2 package", "URL: https://quay.io/repository/biocontainers/epic2",
          },
        },
        ["epic2/ctr-0.0.16--py35h4e20536_0"] = {
          Category = "['Bioinformatics']",
          Description = "The epic2 package",
          Name = "epic2",
          URL = "https://quay.io/repository/biocontainers/epic2",
          Version = "ctr-0.0.16--py35h4e20536_0",
          canonical = "ctr-0.0.16--py35h4e20536_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic2/ctr-0.0.16--py35h4e20536_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/epic2:0.0.16--py35h4e20536_0, which exposes the
following programs:

 - bcftools
 - color-chrs.pl
 - cygdb
 - cython
 - cythonize
 - epic2
 - guess-ploidy.py
 - idle3.5
 - natsort
 - ncurses6-config
 - plot-roh.py
 - plot-vcfstats
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - run-roh.pl
 - samtools
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/epic2

If you encounter errors in epic2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/epic2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000016.*y.000000035.*h.000000004.*e.000020536.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000016.*y.000000035.*h.000000004.*e.000020536.*_.*zfinal",
          whatis = {
            "Name: epic2", "Version: ctr-0.0.16--py35h4e20536_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The epic2 package", "URL: https://quay.io/repository/biocontainers/epic2",
          },
        },
        ["epic2/ctr-0.0.16--py36h4e20536_0"] = {
          Category = "['Bioinformatics']",
          Description = "The epic2 package",
          Name = "epic2",
          URL = "https://quay.io/repository/biocontainers/epic2",
          Version = "ctr-0.0.16--py36h4e20536_0",
          canonical = "ctr-0.0.16--py36h4e20536_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epic2/ctr-0.0.16--py36h4e20536_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/epic2:0.0.16--py36h4e20536_0, which exposes the
following programs:

 - bcftools
 - color-chrs.pl
 - cygdb
 - cython
 - cythonize
 - epic2
 - guess-ploidy.py
 - natsort
 - ncurses6-config
 - plot-roh.py
 - plot-vcfstats
 - run-roh.pl
 - samtools
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/epic2

If you encounter errors in epic2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/epic2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000016.*y.000000036.*h.000000004.*e.000020536.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000016.*y.000000036.*h.000000004.*e.000020536.*_.*zfinal",
          whatis = {
            "Name: epic2", "Version: ctr-0.0.16--py36h4e20536_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The epic2 package", "URL: https://quay.io/repository/biocontainers/epic2",
          },
        },
      },
    },
    epicseg = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["epicseg/ctr-1.0--r3.4.1_0"] = {
          Category = "['Bioinformatics']",
          Description = "The epicseg package",
          Name = "epicseg",
          URL = "https://quay.io/repository/biocontainers/epicseg",
          Version = "ctr-1.0--r3.4.1_0",
          canonical = "ctr-1.0--r3.4.1_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epicseg/ctr-1.0--r3.4.1_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/epicseg:1.0--r3.4.1_0, which exposes the
following programs:

 - .bioconductor-genomeinfodbdata-post-link.sh
 - .bioconductor-genomeinfodbdata-pre-unlink.sh
 - .r-base-post-link.sh
 - R
 - Rscript
 - ksu

This container was pulled from:

	https://quay.io/repository/biocontainers/epicseg

If you encounter errors in epicseg or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/epicseg

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*r.000000003.000000004.000000001.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*r.000000003.000000004.000000001.*_.*zfinal",
          whatis = {
            "Name: epicseg", "Version: ctr-1.0--r3.4.1_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The epicseg package", "URL: https://quay.io/repository/biocontainers/epicseg",
          },
        },
        ["epicseg/ctr-1.0--r341h470a237_1"] = {
          Category = "['Bioinformatics']",
          Description = "The epicseg package",
          Name = "epicseg",
          URL = "https://quay.io/repository/biocontainers/epicseg",
          Version = "ctr-1.0--r341h470a237_1",
          canonical = "ctr-1.0--r341h470a237_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epicseg/ctr-1.0--r341h470a237_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/epicseg:1.0--r341h470a237_1, which exposes the
following programs:

 - .bioconductor-genomeinfodbdata-post-link.sh
 - .bioconductor-genomeinfodbdata-pre-unlink.sh
 - .r-base-post-link.sh
 - R
 - Rscript
 - fc-conflist
 - hb-subset

This container was pulled from:

	https://quay.io/repository/biocontainers/epicseg

If you encounter errors in epicseg or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/epicseg

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 24,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*r.000000341.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*r.000000341.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          whatis = {
            "Name: epicseg", "Version: ctr-1.0--r341h470a237_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The epicseg package", "URL: https://quay.io/repository/biocontainers/epicseg",
          },
        },
        ["epicseg/ctr-1.0--r351h470a237_1"] = {
          Category = "['Bioinformatics']",
          Description = "The epicseg package",
          Name = "epicseg",
          URL = "https://quay.io/repository/biocontainers/epicseg",
          Version = "ctr-1.0--r351h470a237_1",
          canonical = "ctr-1.0--r351h470a237_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epicseg/ctr-1.0--r351h470a237_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/epicseg:1.0--r351h470a237_1, which exposes the
following programs:

 - .bioconductor-genomeinfodbdata-post-link.sh
 - .bioconductor-genomeinfodbdata-pre-unlink.sh
 - .r-base-post-link.sh
 - R
 - Rscript
 - fc-conflist
 - hb-subset
 - ncurses6-config

This container was pulled from:

	https://quay.io/repository/biocontainers/epicseg

If you encounter errors in epicseg or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/epicseg

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 24,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*r.000000351.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*r.000000351.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          whatis = {
            "Name: epicseg", "Version: ctr-1.0--r351h470a237_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The epicseg package", "URL: https://quay.io/repository/biocontainers/epicseg",
          },
        },
      },
    },
    epigrass = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["epigrass/ctr-v2.4.7-1-deb_cv1"] = {
          Version = "ctr-v2.4.7-1-deb_cv1",
          canonical = "ctr-v2.4.7-1-deb_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epigrass/ctr-v2.4.7-1-deb_cv1.lua",
          luaExt = 21,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000002.000000004.000000007.*zfinal-.000000001.*deb.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000002.000000004.000000007.*zfinal-.000000001.*deb.*_.*cv.000000001.*zfinal",
        },
      },
    },
    epydoc = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["epydoc/ctr-3.0.1--py27_0"] = {
          Version = "ctr-3.0.1--py27_0",
          canonical = "ctr-3.0.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epydoc/ctr-3.0.1--py27_0.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000000.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000000.000000001.*y.000000027.*_.*zfinal",
        },
        ["epydoc/ctr-3.0.1--py27_1"] = {
          Version = "ctr-3.0.1--py27_1",
          canonical = "ctr-3.0.1--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/epydoc/ctr-3.0.1--py27_1.lua",
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000000.000000001.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000000.000000001.*y.000000027.*_.000000001.*zfinal",
        },
      },
    },
    eqtlbma = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["eqtlbma/ctr-1.3.1--r3.3.1_0"] = {
          Version = "ctr-1.3.1--r3.3.1_0",
          canonical = "ctr-1.3.1--r3.3.1_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eqtlbma/ctr-1.3.1--r3.3.1_0.lua",
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000003.000000001.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000003.000000001.*_.*zfinal",
        },
        ["eqtlbma/ctr-1.3.1--r3.3.1_1"] = {
          Version = "ctr-1.3.1--r3.3.1_1",
          canonical = "ctr-1.3.1--r3.3.1_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eqtlbma/ctr-1.3.1--r3.3.1_1.lua",
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000003.000000001.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000003.000000001.*_.000000001.*zfinal",
        },
        ["eqtlbma/ctr-1.3.1--r3.3.2_1"] = {
          Version = "ctr-1.3.1--r3.3.2_1",
          canonical = "ctr-1.3.1--r3.3.2_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eqtlbma/ctr-1.3.1--r3.3.2_1.lua",
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000003.000000002.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000003.000000002.*_.000000001.*zfinal",
        },
        ["eqtlbma/ctr-1.3.1--r3.4.1_1"] = {
          Version = "ctr-1.3.1--r3.4.1_1",
          canonical = "ctr-1.3.1--r3.4.1_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eqtlbma/ctr-1.3.1--r3.4.1_1.lua",
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000004.000000001.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000004.000000001.*_.000000001.*zfinal",
        },
        ["eqtlbma/ctr-1.3.1--r3.4.1_2"] = {
          Version = "ctr-1.3.1--r3.4.1_2",
          canonical = "ctr-1.3.1--r3.4.1_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eqtlbma/ctr-1.3.1--r3.4.1_2.lua",
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000004.000000001.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000004.000000001.*_.000000002.*zfinal",
        },
        ["eqtlbma/ctr-1.3.1--r3.4.1_3"] = {
          Version = "ctr-1.3.1--r3.4.1_3",
          canonical = "ctr-1.3.1--r3.4.1_3",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eqtlbma/ctr-1.3.1--r3.4.1_3.lua",
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000004.000000001.*_.000000003.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.000000001.*r.000000003.000000004.000000001.*_.000000003.*zfinal",
        },
        ["eqtlbma/ctr-1.3.3--h3dbd7e7_0"] = {
          Version = "ctr-1.3.3--h3dbd7e7_0",
          canonical = "ctr-1.3.3--h3dbd7e7_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eqtlbma/ctr-1.3.3--h3dbd7e7_0.lua",
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000003.000000003.*h.000000003.*dbd.000000007.*e.000000007.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000003.000000003.*h.000000003.*dbd.000000007.*e.000000007.*_.*zfinal",
        },
      },
    },
    erds = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["erds/ctr-1.1--pl5.22.0_0"] = {
          Category = "['Variant calling', 'Sequence alignment', 'Statistical calculation']",
          Description = "ERDS is a free, open-source software, designed for detection of copy number variants (CNVs) on human genomes from next generation sequence data. It uses paired Hidden Markov models (PHMM) based on the expected distribution of read depth of short reads and the presence of heterozygous sites. ERDS is NOT good for whole exome data.",
          Name = "erds",
          URL = "https://quay.io/repository/biocontainers/erds",
          Version = "ctr-1.1--pl5.22.0_0",
          canonical = "ctr-1.1--pl5.22.0_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/erds/ctr-1.1--pl5.22.0_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/erds:1.1--pl5.22.0_0, which exposes the
following programs:

 - bcftools
 - erds_pipeline
 - ncurses6-config
 - perl5.22.0
 - samtools
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/erds

If you encounter errors in erds or need help running the
tools it contains, please contact the developer at

	http://www.utahresearch.org/mingfuzhu/erds/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.*l.000000005.000000022.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.*l.000000005.000000022.*_.*zfinal",
          whatis = {
            "Name: erds", "Version: ctr-1.1--pl5.22.0_0", "Category: ['Variant calling', 'Sequence alignment', 'Statistical calculation']", "Keywords: ['Sequence analysis', 'Genetic variation', 'Genomics']", "Description: ERDS is a free, open-source software, designed for detection of copy number variants (CNVs) on human genomes from next generation sequence data. It uses paired Hidden Markov models (PHMM) based on the expected distribution of read depth of short reads and the presence of heterozygous sites. ERDS is NOT good for whole exome data.", "URL: https://quay.io/repository/biocontainers/erds",
          },
        },
        ["erds/ctr-1.1--pl5.22.0_1"] = {
          Category = "['Variant calling', 'Sequence alignment', 'Statistical calculation']",
          Description = "ERDS is a free, open-source software, designed for detection of copy number variants (CNVs) on human genomes from next generation sequence data. It uses paired Hidden Markov models (PHMM) based on the expected distribution of read depth of short reads and the presence of heterozygous sites. ERDS is NOT good for whole exome data.",
          Name = "erds",
          URL = "https://quay.io/repository/biocontainers/erds",
          Version = "ctr-1.1--pl5.22.0_1",
          canonical = "ctr-1.1--pl5.22.0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/erds/ctr-1.1--pl5.22.0_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/erds:1.1--pl5.22.0_1, which exposes the
following programs:

 - bcftools
 - erds_pipeline
 - ncurses6-config
 - perl5.22.0
 - samtools
 - vcfutils.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/erds

If you encounter errors in erds or need help running the
tools it contains, please contact the developer at

	http://www.utahresearch.org/mingfuzhu/erds/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.*l.000000005.000000022.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.*l.000000005.000000022.*_.000000001.*zfinal",
          whatis = {
            "Name: erds", "Version: ctr-1.1--pl5.22.0_1", "Category: ['Variant calling', 'Sequence alignment', 'Statistical calculation']", "Keywords: ['Sequence analysis', 'Genetic variation', 'Genomics']", "Description: ERDS is a free, open-source software, designed for detection of copy number variants (CNVs) on human genomes from next generation sequence data. It uses paired Hidden Markov models (PHMM) based on the expected distribution of read depth of short reads and the presence of heterozygous sites. ERDS is NOT good for whole exome data.", "URL: https://quay.io/repository/biocontainers/erds",
          },
        },
      },
    },
    ericscript = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ericscript/ctr-0.5.5--3"] = {
          Category = "['Bioinformatics']",
          Description = "The ericscript package",
          Name = "ericscript",
          URL = "https://quay.io/repository/biocontainers/ericscript",
          Version = "ctr-0.5.5--3",
          canonical = "ctr-0.5.5--3",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ericscript/ctr-0.5.5--3.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ericscript:0.5.5--3, which exposes the
following programs:

 - R
 - Rscript
 - annotateBed
 - bamToBed
 - bamToFastq
 - bcftools
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - blat
 - bwa
 - closestBed
 - clusterBed
 - complementBed
 - coverageBed
 - ericscript.pl
 - expandCols
 - faToNib
 - faToTwoBit
 - fastaFromBed
 - flankBed
 - genomeCoverageBed
 - getOverlap
 - gfClient
 - gfServer
 - groupBy
 - intersectBed
 - linksBed
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - multiBamCov
 - multiIntersectBed
 - nibFrag
 - nucBed
 - pairToBed
 - pairToPair
 - perl5.26.2
 - pslPretty
 - pslReps
 - pslSort
 - randomBed
 - samtools
 - seqtk
 - shiftBed
 - shuffleBed
 - slopBed
 - sortBed
 - subtractBed
 - tagBam
 - twoBitInfo
 - twoBitToFa
 - unionBedGraphs
 - vcfutils.pl
 - windowBed
 - windowMaker

This container was pulled from:

	https://quay.io/repository/biocontainers/ericscript

If you encounter errors in ericscript or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ericscript

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000005.000000005.*zfinal-.000000003.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000005.000000005.*zfinal-.000000003.*zfinal",
          whatis = {
            "Name: ericscript", "Version: ctr-0.5.5--3", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ericscript package", "URL: https://quay.io/repository/biocontainers/ericscript",
          },
        },
        ["ericscript/ctr-0.5.5--pl5.22.0r3.3.1_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ericscript package",
          Name = "ericscript",
          URL = "https://quay.io/repository/biocontainers/ericscript",
          Version = "ctr-0.5.5--pl5.22.0r3.3.1_1",
          canonical = "ctr-0.5.5--pl5.22.0r3.3.1_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ericscript/ctr-0.5.5--pl5.22.0r3.3.1_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ericscript:0.5.5--pl5.22.0r3.3.1_1, which exposes the
following programs:

 - R
 - Rscript
 - annotateBed
 - bamToBed
 - bamToFastq
 - bcftools
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - blat
 - bwa
 - closestBed
 - clusterBed
 - complementBed
 - coverageBed
 - ericscript.pl
 - expandCols
 - faToNib
 - faToTwoBit
 - fastaFromBed
 - flankBed
 - genccode
 - gencmn
 - gennorm2
 - genomeCoverageBed
 - gensprep
 - getOverlap
 - gfClient
 - gfServer
 - groupBy
 - icupkg
 - intersectBed
 - linksBed
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - multiBamCov
 - multiIntersectBed
 - nibFrag
 - nucBed
 - pairToBed
 - pairToPair
 - perl5.22.0
 - pslPretty
 - pslReps
 - pslSort
 - randomBed
 - samtools
 - seqtk
 - shiftBed
 - shuffleBed
 - slopBed
 - sortBed
 - subtractBed
 - tagBam
 - twoBitInfo
 - twoBitToFa
 - uconv
 - unionBedGraphs
 - vcfutils.pl
 - windowBed
 - windowMaker

This container was pulled from:

	https://quay.io/repository/biocontainers/ericscript

If you encounter errors in ericscript or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ericscript

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 28,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000005.000000005.*l.000000005.000000022.*r.000000003.000000003.000000001.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000005.000000005.*l.000000005.000000022.*r.000000003.000000003.000000001.*_.000000001.*zfinal",
          whatis = {
            "Name: ericscript", "Version: ctr-0.5.5--pl5.22.0r3.3.1_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ericscript package", "URL: https://quay.io/repository/biocontainers/ericscript",
          },
        },
        ["ericscript/ctr-0.5.5--pl5.22.0r3.3.2_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ericscript package",
          Name = "ericscript",
          URL = "https://quay.io/repository/biocontainers/ericscript",
          Version = "ctr-0.5.5--pl5.22.0r3.3.2_1",
          canonical = "ctr-0.5.5--pl5.22.0r3.3.2_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ericscript/ctr-0.5.5--pl5.22.0r3.3.2_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ericscript:0.5.5--pl5.22.0r3.3.2_1, which exposes the
following programs:

 - R
 - Rscript
 - annotateBed
 - bamToBed
 - bamToFastq
 - bcftools
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - blat
 - bwa
 - closestBed
 - clusterBed
 - complementBed
 - coverageBed
 - ericscript.pl
 - expandCols
 - faToNib
 - faToTwoBit
 - fastaFromBed
 - flankBed
 - genccode
 - gencmn
 - gennorm2
 - genomeCoverageBed
 - gensprep
 - getOverlap
 - gfClient
 - gfServer
 - groupBy
 - icupkg
 - intersectBed
 - linksBed
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - multiBamCov
 - multiIntersectBed
 - nibFrag
 - nucBed
 - pairToBed
 - pairToPair
 - perl5.22.0
 - pslPretty
 - pslReps
 - pslSort
 - randomBed
 - samtools
 - seqtk
 - shiftBed
 - shuffleBed
 - slopBed
 - sortBed
 - subtractBed
 - tagBam
 - twoBitInfo
 - twoBitToFa
 - uconv
 - unionBedGraphs
 - vcfutils.pl
 - windowBed
 - windowMaker

This container was pulled from:

	https://quay.io/repository/biocontainers/ericscript

If you encounter errors in ericscript or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ericscript

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 28,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000005.000000005.*l.000000005.000000022.*r.000000003.000000003.000000002.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000005.000000005.*l.000000005.000000022.*r.000000003.000000003.000000002.*_.000000001.*zfinal",
          whatis = {
            "Name: ericscript", "Version: ctr-0.5.5--pl5.22.0r3.3.2_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ericscript package", "URL: https://quay.io/repository/biocontainers/ericscript",
          },
        },
        ["ericscript/ctr-0.5.5--pl5.22.0r3.4.1_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ericscript package",
          Name = "ericscript",
          URL = "https://quay.io/repository/biocontainers/ericscript",
          Version = "ctr-0.5.5--pl5.22.0r3.4.1_1",
          canonical = "ctr-0.5.5--pl5.22.0r3.4.1_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ericscript/ctr-0.5.5--pl5.22.0r3.4.1_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ericscript:0.5.5--pl5.22.0r3.4.1_1, which exposes the
following programs:

 - .r-base-post-link.sh
 - R
 - Rscript
 - annotateBed
 - bamToBed
 - bamToFastq
 - bcftools
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - blat
 - bmp2tiff
 - bwa
 - closestBed
 - clusterBed
 - complementBed
 - coverageBed
 - ericscript.pl
 - expandCols
 - faToNib
 - faToTwoBit
 - fastaFromBed
 - flankBed
 - genomeCoverageBed
 - getOverlap
 - gfClient
 - gfServer
 - gif2tiff
 - groupBy
 - intersectBed
 - ksu
 - linksBed
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - multiBamCov
 - multiIntersectBed
 - nibFrag
 - nucBed
 - pairToBed
 - pairToPair
 - perl5.22.0
 - pslPretty
 - pslReps
 - pslSort
 - qualfa2fq.pl
 - randomBed
 - ras2tiff
 - rgb2ycbcr
 - samtools
 - seqtk
 - shiftBed
 - shuffleBed
 - slopBed
 - sortBed
 - subtractBed
 - tagBam
 - thumbnail
 - twoBitInfo
 - twoBitToFa
 - unionBedGraphs
 - vcfutils.pl
 - windowBed
 - windowMaker
 - xa2multi.pl

This container was pulled from:

	https://quay.io/repository/biocontainers/ericscript

If you encounter errors in ericscript or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ericscript

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 28,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000005.000000005.*l.000000005.000000022.*r.000000003.000000004.000000001.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000005.000000005.*l.000000005.000000022.*r.000000003.000000004.000000001.*_.000000001.*zfinal",
          whatis = {
            "Name: ericscript", "Version: ctr-0.5.5--pl5.22.0r3.4.1_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ericscript package", "URL: https://quay.io/repository/biocontainers/ericscript",
          },
        },
        ["ericscript/ctr-0.5.5--pl5.22.0r3.4.1_2"] = {
          Category = "['Bioinformatics']",
          Description = "The ericscript package",
          Name = "ericscript",
          URL = "https://quay.io/repository/biocontainers/ericscript",
          Version = "ctr-0.5.5--pl5.22.0r3.4.1_2",
          canonical = "ctr-0.5.5--pl5.22.0r3.4.1_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ericscript/ctr-0.5.5--pl5.22.0r3.4.1_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ericscript:0.5.5--pl5.22.0r3.4.1_2, which exposes the
following programs:

 - .r-base-post-link.sh
 - R
 - Rscript
 - annotateBed
 - bamToBed
 - bamToFastq
 - bcftools
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - blat
 - bwa
 - closestBed
 - clusterBed
 - complementBed
 - coverageBed
 - ericscript.pl
 - expandCols
 - faToNib
 - faToTwoBit
 - fastaFromBed
 - flankBed
 - genomeCoverageBed
 - getOverlap
 - gfClient
 - gfServer
 - groupBy
 - intersectBed
 - ksu
 - linksBed
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - multiBamCov
 - multiIntersectBed
 - nibFrag
 - nucBed
 - pairToBed
 - pairToPair
 - perl5.22.0
 - pslPretty
 - pslReps
 - pslSort
 - randomBed
 - samtools
 - seqtk
 - shiftBed
 - shuffleBed
 - slopBed
 - sortBed
 - subtractBed
 - tagBam
 - twoBitInfo
 - twoBitToFa
 - unionBedGraphs
 - vcfutils.pl
 - windowBed
 - windowMaker

This container was pulled from:

	https://quay.io/repository/biocontainers/ericscript

If you encounter errors in ericscript or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ericscript

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 28,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000005.000000005.*l.000000005.000000022.*r.000000003.000000004.000000001.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000005.000000005.*l.000000005.000000022.*r.000000003.000000004.000000001.*_.000000002.*zfinal",
          whatis = {
            "Name: ericscript", "Version: ctr-0.5.5--pl5.22.0r3.4.1_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ericscript package", "URL: https://quay.io/repository/biocontainers/ericscript",
          },
        },
        ["ericscript/ctr-0.5.5--r3.3.1_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ericscript package",
          Name = "ericscript",
          URL = "https://quay.io/repository/biocontainers/ericscript",
          Version = "ctr-0.5.5--r3.3.1_0",
          canonical = "ctr-0.5.5--r3.3.1_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ericscript/ctr-0.5.5--r3.3.1_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ericscript:0.5.5--r3.3.1_0, which exposes the
following programs:

 - R
 - Rscript
 - annotateBed
 - bamToBed
 - bamToFastq
 - bcftools
 - bed12ToBed6
 - bedToBam
 - bedToIgv
 - bedpeToBam
 - bedtools
 - blat
 - bmp2tiff
 - bwa
 - closestBed
 - clusterBed
 - complementBed
 - coverageBed
 - ericscript.pl
 - expandCols
 - faToNib
 - faToTwoBit
 - fastaFromBed
 - flankBed
 - genccode
 - gencmn
 - gennorm2
 - genomeCoverageBed
 - gensprep
 - getOverlap
 - gfClient
 - gfServer
 - gif2tiff
 - groupBy
 - icupkg
 - intersectBed
 - linksBed
 - mapBed
 - maskFastaFromBed
 - mergeBed
 - multiBamCov
 - multiIntersectBed
 - nibFrag
 - nucBed
 - pairToBed
 - pairToPair
 - perl5.22.0
 - pslPretty
 - pslReps
 - pslSort
 - randomBed
 - ras2tiff
 - rgb2ycbcr
 - samtools
 - seqtk
 - shiftBed
 - shuffleBed
 - slopBed
 - sortBed
 - subtractBed
 - tagBam
 - thumbnail
 - twoBitInfo
 - twoBitToFa
 - uconv
 - unionBedGraphs
 - vcfutils.pl
 - windowBed
 - windowMaker

This container was pulled from:

	https://quay.io/repository/biocontainers/ericscript

If you encounter errors in ericscript or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ericscript

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000005.000000005.*r.000000003.000000003.000000001.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000005.000000005.*r.000000003.000000003.000000001.*_.*zfinal",
          whatis = {
            "Name: ericscript", "Version: ctr-0.5.5--r3.3.1_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ericscript package", "URL: https://quay.io/repository/biocontainers/ericscript",
          },
        },
      },
    },
    erne = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["erne/ctr-2.1.1--boost1.61_0"] = {
          Category = "['Read mapping', 'Bisulfite mapping']",
          Description = "Extended Randomized Numerical alignEr for accurate alignment of NGS reads. It can map bisulfite-treated reads.",
          Name = "erne",
          URL = "https://quay.io/repository/biocontainers/erne",
          Version = "ctr-2.1.1--boost1.61_0",
          canonical = "ctr-2.1.1--boost1.61_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/erne/ctr-2.1.1--boost1.61_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/erne:2.1.1--boost1.61_0, which exposes the
following programs:

 - easy_install-3.6
 - erne-bs5
 - erne-create
 - erne-filter
 - erne-map
 - erne-meth
 - genccode
 - gencmn
 - gennorm2
 - gensprep
 - icupkg
 - uconv

This container was pulled from:

	https://quay.io/repository/biocontainers/erne

If you encounter errors in erne or need help running the
tools it contains, please contact the developer at

	http://erne.sourceforge.net

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 23,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000001.000000001.*boost.000000001.000000061.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000001.000000001.*boost.000000001.000000061.*_.*zfinal",
          whatis = {
            "Name: erne", "Version: ctr-2.1.1--boost1.61_0", "Category: ['Read mapping', 'Bisulfite mapping']", "Keywords: ['Genomics', 'Sequencing', 'Epigenetics']", "Description: Extended Randomized Numerical alignEr for accurate alignment of NGS reads. It can map bisulfite-treated reads.", "URL: https://quay.io/repository/biocontainers/erne",
          },
        },
        ["erne/ctr-2.1.1--boost1.64_0"] = {
          Category = "['Read mapping', 'Bisulfite mapping']",
          Description = "Extended Randomized Numerical alignEr for accurate alignment of NGS reads. It can map bisulfite-treated reads.",
          Name = "erne",
          URL = "https://quay.io/repository/biocontainers/erne",
          Version = "ctr-2.1.1--boost1.64_0",
          canonical = "ctr-2.1.1--boost1.64_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/erne/ctr-2.1.1--boost1.64_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/erne:2.1.1--boost1.64_0, which exposes the
following programs:

 - easy_install-3.6
 - erne-bs5
 - erne-create
 - erne-filter
 - erne-map
 - erne-meth

This container was pulled from:

	https://quay.io/repository/biocontainers/erne

If you encounter errors in erne or need help running the
tools it contains, please contact the developer at

	http://erne.sourceforge.net

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 23,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000001.000000001.*boost.000000001.000000064.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000001.000000001.*boost.000000001.000000064.*_.*zfinal",
          whatis = {
            "Name: erne", "Version: ctr-2.1.1--boost1.64_0", "Category: ['Read mapping', 'Bisulfite mapping']", "Keywords: ['Genomics', 'Sequencing', 'Epigenetics']", "Description: Extended Randomized Numerical alignEr for accurate alignment of NGS reads. It can map bisulfite-treated reads.", "URL: https://quay.io/repository/biocontainers/erne",
          },
        },
        ["erne/ctr-2.1.1--boost1.64_1"] = {
          Category = "['Read mapping', 'Bisulfite mapping']",
          Description = "Extended Randomized Numerical alignEr for accurate alignment of NGS reads. It can map bisulfite-treated reads.",
          Name = "erne",
          URL = "https://quay.io/repository/biocontainers/erne",
          Version = "ctr-2.1.1--boost1.64_1",
          canonical = "ctr-2.1.1--boost1.64_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/erne/ctr-2.1.1--boost1.64_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/erne:2.1.1--boost1.64_1, which exposes the
following programs:

 - erne-bs5
 - erne-create
 - erne-filter
 - erne-map
 - erne-meth

This container was pulled from:

	https://quay.io/repository/biocontainers/erne

If you encounter errors in erne or need help running the
tools it contains, please contact the developer at

	http://erne.sourceforge.net

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 23,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000001.000000001.*boost.000000001.000000064.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000001.000000001.*boost.000000001.000000064.*_.000000001.*zfinal",
          whatis = {
            "Name: erne", "Version: ctr-2.1.1--boost1.64_1", "Category: ['Read mapping', 'Bisulfite mapping']", "Keywords: ['Genomics', 'Sequencing', 'Epigenetics']", "Description: Extended Randomized Numerical alignEr for accurate alignment of NGS reads. It can map bisulfite-treated reads.", "URL: https://quay.io/repository/biocontainers/erne",
          },
        },
        ["erne/ctr-2.1.1--heaa942f_2"] = {
          Category = "['Read mapping', 'Bisulfite mapping']",
          Description = "Extended Randomized Numerical alignEr for accurate alignment of NGS reads. It can map bisulfite-treated reads.",
          Name = "erne",
          URL = "https://quay.io/repository/biocontainers/erne",
          Version = "ctr-2.1.1--heaa942f_2",
          canonical = "ctr-2.1.1--heaa942f_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/erne/ctr-2.1.1--heaa942f_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/erne:2.1.1--heaa942f_2, which exposes the
following programs:

 - erne-bs5
 - erne-create
 - erne-filter
 - erne-map
 - erne-meth

This container was pulled from:

	https://quay.io/repository/biocontainers/erne

If you encounter errors in erne or need help running the
tools it contains, please contact the developer at

	http://erne.sourceforge.net

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000001.000000001.*heaa.000000942.*f.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000001.000000001.*heaa.000000942.*f.*_.000000002.*zfinal",
          whatis = {
            "Name: erne", "Version: ctr-2.1.1--heaa942f_2", "Category: ['Read mapping', 'Bisulfite mapping']", "Keywords: ['Genomics', 'Sequencing', 'Epigenetics']", "Description: Extended Randomized Numerical alignEr for accurate alignment of NGS reads. It can map bisulfite-treated reads.", "URL: https://quay.io/repository/biocontainers/erne",
          },
        },
      },
    },
    esimsa = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["esimsa/ctr-1.0--0"] = {
          Category = "['Deisotoping']",
          Description = "Simple deconvolution of electrospray ionization peak lists.",
          Name = "esimsa",
          URL = "https://quay.io/repository/biocontainers/esimsa",
          Version = "ctr-1.0--0",
          canonical = "ctr-1.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/esimsa/ctr-1.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/esimsa:1.0--0, which exposes the
following programs:

 - esimsa

This container was pulled from:

	https://quay.io/repository/biocontainers/esimsa

If you encounter errors in esimsa or need help running the
tools it contains, please contact the developer at

	http://www.ms-utils.org/esimsa.html

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: esimsa", "Version: ctr-1.0--0", "Category: ['Deisotoping']", "Keywords: ['Proteomics', 'Proteomics experiment']", "Description: Simple deconvolution of electrospray ionization peak lists.", "URL: https://quay.io/repository/biocontainers/esimsa",
          },
        },
        ["esimsa/ctr-1.0--h470a237_1"] = {
          Category = "['Deisotoping']",
          Description = "Simple deconvolution of electrospray ionization peak lists.",
          Name = "esimsa",
          URL = "https://quay.io/repository/biocontainers/esimsa",
          Version = "ctr-1.0--h470a237_1",
          canonical = "ctr-1.0--h470a237_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/esimsa/ctr-1.0--h470a237_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/esimsa:1.0--h470a237_1, which exposes the
following programs:

 - esimsa

This container was pulled from:

	https://quay.io/repository/biocontainers/esimsa

If you encounter errors in esimsa or need help running the
tools it contains, please contact the developer at

	http://www.ms-utils.org/esimsa.html

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          whatis = {
            "Name: esimsa", "Version: ctr-1.0--h470a237_1", "Category: ['Deisotoping']", "Keywords: ['Proteomics', 'Proteomics experiment']", "Description: Simple deconvolution of electrospray ionization peak lists.", "URL: https://quay.io/repository/biocontainers/esimsa",
          },
        },
      },
    },
    esmre = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["esmre/ctr-0.3.1--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The esmre package",
          Name = "esmre",
          URL = "https://quay.io/repository/biocontainers/esmre",
          Version = "ctr-0.3.1--py27_0",
          canonical = "ctr-0.3.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/esmre/ctr-0.3.1--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/esmre:0.3.1--py27_0, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/esmre

If you encounter errors in esmre or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/esmre

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: esmre", "Version: ctr-0.3.1--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The esmre package", "URL: https://quay.io/repository/biocontainers/esmre",
          },
        },
        ["esmre/ctr-0.3.1--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The esmre package",
          Name = "esmre",
          URL = "https://quay.io/repository/biocontainers/esmre",
          Version = "ctr-0.3.1--py27_1",
          canonical = "ctr-0.3.1--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/esmre/ctr-0.3.1--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/esmre:0.3.1--py27_1, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/esmre

If you encounter errors in esmre or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/esmre

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000003.000000001.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: esmre", "Version: ctr-0.3.1--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The esmre package", "URL: https://quay.io/repository/biocontainers/esmre",
          },
        },
      },
    },
    estmapper = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["estmapper/ctr-2008--py27pl5.22.0_0"] = {
          Category = "['Bioinformatics']",
          Description = "The estmapper package",
          Name = "estmapper",
          URL = "https://quay.io/repository/biocontainers/estmapper",
          Version = "ctr-2008--py27pl5.22.0_0",
          canonical = "ctr-2008--py27pl5.22.0_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/estmapper/ctr-2008--py27pl5.22.0_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/estmapper:2008--py27pl5.22.0_0, which exposes the
following programs:

 - ESTmapper.pl
 - cleanPolishes
 - comparePolishes
 - configureESTmapper.pl
 - convertPolishes
 - convertToAtac
 - convertToExtent
 - depthOfPolishes
 - detectChimera
 - existDB
 - filterEST
 - filterESTsimple
 - filterMRNA
 - filterNULL
 - filterPolishes
 - filtertest
 - fixPolishesIID
 - headPolishes
 - hitConverter
 - kmer-mask
 - leaff
 - mapMers
 - mapMers-depth
 - mappedCoverage
 - mergeCounts
 - mergePolishes
 - meryl
 - mt19937ar-test
 - parseSNP
 - perl5.22.0
 - pickBestPair
 - pickBestPolish
 - pickUniquePolish
 - plotCoverageVsIdentity
 - positionDB
 - realignPolishes
 - removeDuplicate
 - removeRedundant
 - reportAlignmentDifferences
 - runConcurrently.pl
 - seagen
 - sim4db
 - simple
 - sortHits
 - sortPolishes
 - summarizePolishes
 - terminate
 - test-merStream
 - test-seqCache
 - test-seqStream
 - uniqPolishes
 - vennPolishes

This container was pulled from:

	https://quay.io/repository/biocontainers/estmapper

If you encounter errors in estmapper or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/estmapper

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 25,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000002008.*y.000000027.*pl.000000005.000000022.*_.*zfinal",
          wV = "*ctr.*zfinal-.000002008.*y.000000027.*pl.000000005.000000022.*_.*zfinal",
          whatis = {
            "Name: estmapper", "Version: ctr-2008--py27pl5.22.0_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The estmapper package", "URL: https://quay.io/repository/biocontainers/estmapper",
          },
        },
        ["estmapper/ctr-2008--py27pl526h9d27c22_2"] = {
          Category = "['Bioinformatics']",
          Description = "The estmapper package",
          Name = "estmapper",
          URL = "https://quay.io/repository/biocontainers/estmapper",
          Version = "ctr-2008--py27pl526h9d27c22_2",
          canonical = "ctr-2008--py27pl526h9d27c22_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/estmapper/ctr-2008--py27pl526h9d27c22_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/estmapper:2008--py27pl526h9d27c22_2, which exposes the
following programs:

 - ESTmapper.pl
 - cleanPolishes
 - comparePolishes
 - configureESTmapper.pl
 - convertPolishes
 - convertToAtac
 - convertToExtent
 - depthOfPolishes
 - detectChimera
 - existDB
 - filterEST
 - filterESTsimple
 - filterMRNA
 - filterNULL
 - filterPolishes
 - filtertest
 - fixPolishesIID
 - headPolishes
 - hitConverter
 - kmer-mask
 - leaff
 - mapMers
 - mapMers-depth
 - mappedCoverage
 - mergeCounts
 - mergePolishes
 - meryl
 - mt19937ar-test
 - ncurses6-config
 - parseSNP
 - perl5.26.2
 - pickBestPair
 - pickBestPolish
 - pickUniquePolish
 - plotCoverageVsIdentity
 - positionDB
 - realignPolishes
 - removeDuplicate
 - removeRedundant
 - reportAlignmentDifferences
 - runConcurrently.pl
 - seagen
 - sim4db
 - simple
 - sortHits
 - sortPolishes
 - summarizePolishes
 - terminate
 - test-merStream
 - test-seqCache
 - test-seqStream
 - uniqPolishes
 - vennPolishes

This container was pulled from:

	https://quay.io/repository/biocontainers/estmapper

If you encounter errors in estmapper or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/estmapper

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 30,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000002008.*y.000000027.*pl.000000526.*h.000000009.*d.000000027.*c.000000022.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000002008.*y.000000027.*pl.000000526.*h.000000009.*d.000000027.*c.000000022.*_.000000002.*zfinal",
          whatis = {
            "Name: estmapper", "Version: ctr-2008--py27pl526h9d27c22_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The estmapper package", "URL: https://quay.io/repository/biocontainers/estmapper",
          },
        },
        ["estmapper/ctr-2008--py34pl5.22.0_0"] = {
          Category = "['Bioinformatics']",
          Description = "The estmapper package",
          Name = "estmapper",
          URL = "https://quay.io/repository/biocontainers/estmapper",
          Version = "ctr-2008--py34pl5.22.0_0",
          canonical = "ctr-2008--py34pl5.22.0_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/estmapper/ctr-2008--py34pl5.22.0_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/estmapper:2008--py34pl5.22.0_0, which exposes the
following programs:

 - 2to3-3.4
 - ESTmapper.pl
 - cleanPolishes
 - comparePolishes
 - configureESTmapper.pl
 - convertPolishes
 - convertToAtac
 - convertToExtent
 - depthOfPolishes
 - detectChimera
 - easy_install-3.4
 - existDB
 - filterEST
 - filterESTsimple
 - filterMRNA
 - filterNULL
 - filterPolishes
 - filtertest
 - fixPolishesIID
 - headPolishes
 - hitConverter
 - idle3.4
 - kmer-mask
 - leaff
 - mapMers
 - mapMers-depth
 - mappedCoverage
 - mergeCounts
 - mergePolishes
 - meryl
 - mt19937ar-test
 - parseSNP
 - perl5.22.0
 - pickBestPair
 - pickBestPolish
 - pickUniquePolish
 - plotCoverageVsIdentity
 - positionDB
 - pydoc3.4
 - python3.4
 - python3.4-config
 - python3.4m
 - python3.4m-config
 - pyvenv-3.4
 - realignPolishes
 - removeDuplicate
 - removeRedundant
 - reportAlignmentDifferences
 - runConcurrently.pl
 - seagen
 - sim4db
 - simple
 - sortHits
 - sortPolishes
 - summarizePolishes
 - terminate
 - test-merStream
 - test-seqCache
 - test-seqStream
 - uniqPolishes
 - vennPolishes

This container was pulled from:

	https://quay.io/repository/biocontainers/estmapper

If you encounter errors in estmapper or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/estmapper

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 25,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000002008.*y.000000034.*pl.000000005.000000022.*_.*zfinal",
          wV = "*ctr.*zfinal-.000002008.*y.000000034.*pl.000000005.000000022.*_.*zfinal",
          whatis = {
            "Name: estmapper", "Version: ctr-2008--py34pl5.22.0_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The estmapper package", "URL: https://quay.io/repository/biocontainers/estmapper",
          },
        },
        ["estmapper/ctr-2008--py35pl5.22.0_0"] = {
          Category = "['Bioinformatics']",
          Description = "The estmapper package",
          Name = "estmapper",
          URL = "https://quay.io/repository/biocontainers/estmapper",
          Version = "ctr-2008--py35pl5.22.0_0",
          canonical = "ctr-2008--py35pl5.22.0_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/estmapper/ctr-2008--py35pl5.22.0_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/estmapper:2008--py35pl5.22.0_0, which exposes the
following programs:

 - ESTmapper.pl
 - cleanPolishes
 - comparePolishes
 - configureESTmapper.pl
 - convertPolishes
 - convertToAtac
 - convertToExtent
 - depthOfPolishes
 - detectChimera
 - easy_install-3.5
 - existDB
 - filterEST
 - filterESTsimple
 - filterMRNA
 - filterNULL
 - filterPolishes
 - filtertest
 - fixPolishesIID
 - headPolishes
 - hitConverter
 - idle3.5
 - kmer-mask
 - leaff
 - mapMers
 - mapMers-depth
 - mappedCoverage
 - mergeCounts
 - mergePolishes
 - meryl
 - mt19937ar-test
 - parseSNP
 - perl5.22.0
 - pickBestPair
 - pickBestPolish
 - pickUniquePolish
 - plotCoverageVsIdentity
 - positionDB
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - realignPolishes
 - removeDuplicate
 - removeRedundant
 - reportAlignmentDifferences
 - runConcurrently.pl
 - seagen
 - sim4db
 - simple
 - sortHits
 - sortPolishes
 - summarizePolishes
 - terminate
 - test-merStream
 - test-seqCache
 - test-seqStream
 - uniqPolishes
 - vennPolishes

This container was pulled from:

	https://quay.io/repository/biocontainers/estmapper

If you encounter errors in estmapper or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/estmapper

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 25,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000002008.*y.000000035.*pl.000000005.000000022.*_.*zfinal",
          wV = "*ctr.*zfinal-.000002008.*y.000000035.*pl.000000005.000000022.*_.*zfinal",
          whatis = {
            "Name: estmapper", "Version: ctr-2008--py35pl5.22.0_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The estmapper package", "URL: https://quay.io/repository/biocontainers/estmapper",
          },
        },
        ["estmapper/ctr-2008--py36pl5.22.0_0"] = {
          Category = "['Bioinformatics']",
          Description = "The estmapper package",
          Name = "estmapper",
          URL = "https://quay.io/repository/biocontainers/estmapper",
          Version = "ctr-2008--py36pl5.22.0_0",
          canonical = "ctr-2008--py36pl5.22.0_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/estmapper/ctr-2008--py36pl5.22.0_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/estmapper:2008--py36pl5.22.0_0, which exposes the
following programs:

 - ESTmapper.pl
 - cleanPolishes
 - comparePolishes
 - configureESTmapper.pl
 - convertPolishes
 - convertToAtac
 - convertToExtent
 - depthOfPolishes
 - detectChimera
 - easy_install-3.6
 - existDB
 - filterEST
 - filterESTsimple
 - filterMRNA
 - filterNULL
 - filterPolishes
 - filtertest
 - fixPolishesIID
 - headPolishes
 - hitConverter
 - kmer-mask
 - leaff
 - mapMers
 - mapMers-depth
 - mappedCoverage
 - mergeCounts
 - mergePolishes
 - meryl
 - mt19937ar-test
 - parseSNP
 - perl5.22.0
 - pickBestPair
 - pickBestPolish
 - pickUniquePolish
 - plotCoverageVsIdentity
 - positionDB
 - realignPolishes
 - removeDuplicate
 - removeRedundant
 - reportAlignmentDifferences
 - runConcurrently.pl
 - seagen
 - sim4db
 - simple
 - sortHits
 - sortPolishes
 - summarizePolishes
 - terminate
 - test-merStream
 - test-seqCache
 - test-seqStream
 - uniqPolishes
 - vennPolishes

This container was pulled from:

	https://quay.io/repository/biocontainers/estmapper

If you encounter errors in estmapper or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/estmapper

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 25,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000002008.*y.000000036.*pl.000000005.000000022.*_.*zfinal",
          wV = "*ctr.*zfinal-.000002008.*y.000000036.*pl.000000005.000000022.*_.*zfinal",
          whatis = {
            "Name: estmapper", "Version: ctr-2008--py36pl5.22.0_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The estmapper package", "URL: https://quay.io/repository/biocontainers/estmapper",
          },
        },
      },
    },
    estscan = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["estscan/ctr-3.0--1"] = {
          Category = "['Annotation']",
          Description = "ESTScan can detect coding regions (CDS) in DNA sequences, even if they are of low quality.  It  also detects/corrects sequencing errors that lead to frameshifts.  ESTScan is not a gene prediction program , nor is it an open reading frame detector.  In fact, its strength lies in the fact that it does not require an open reading frame to detect a coding region.  The program may miss a few translated amino acids at termini, but detects coding regions with high selectivity and sensitivity.",
          Name = "estscan",
          URL = "https://quay.io/repository/biocontainers/estscan",
          Version = "ctr-3.0--1",
          canonical = "ctr-3.0--1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/estscan/ctr-3.0--1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/estscan:3.0--1, which exposes the
following programs:

 - estscan

This container was pulled from:

	https://quay.io/repository/biocontainers/estscan

If you encounter errors in estscan or need help running the
tools it contains, please contact the developer at

	http://myhits.isb-sib.ch/cgi-bin/estscan

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: estscan", "Version: ctr-3.0--1", "Category: ['Annotation']", "Keywords: ['Genomics', 'Transcriptomics']", "Description: ESTScan can detect coding regions (CDS) in DNA sequences, even if they are of low quality.  It  also detects/corrects sequencing errors that lead to frameshifts.  ESTScan is not a gene prediction program , nor is it an open reading frame detector.  In fact, its strength lies in the fact that it does not require an open reading frame to detect a coding region.  The program may miss a few translated amino acids at termini, but detects coding regions with high selectivity and sensitivity.", "URL: https://quay.io/repository/biocontainers/estscan",
          },
        },
        ["estscan/ctr-3.0--h470a237_2"] = {
          Category = "['Annotation']",
          Description = "ESTScan can detect coding regions (CDS) in DNA sequences, even if they are of low quality.  It  also detects/corrects sequencing errors that lead to frameshifts.  ESTScan is not a gene prediction program , nor is it an open reading frame detector.  In fact, its strength lies in the fact that it does not require an open reading frame to detect a coding region.  The program may miss a few translated amino acids at termini, but detects coding regions with high selectivity and sensitivity.",
          Name = "estscan",
          URL = "https://quay.io/repository/biocontainers/estscan",
          Version = "ctr-3.0--h470a237_2",
          canonical = "ctr-3.0--h470a237_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/estscan/ctr-3.0--h470a237_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/estscan:3.0--h470a237_2, which exposes the
following programs:

 - estscan

This container was pulled from:

	https://quay.io/repository/biocontainers/estscan

If you encounter errors in estscan or need help running the
tools it contains, please contact the developer at

	http://myhits.isb-sib.ch/cgi-bin/estscan

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.*h.000000470.*a.000000237.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000003.*h.000000470.*a.000000237.*_.000000002.*zfinal",
          whatis = {
            "Name: estscan", "Version: ctr-3.0--h470a237_2", "Category: ['Annotation']", "Keywords: ['Genomics', 'Transcriptomics']", "Description: ESTScan can detect coding regions (CDS) in DNA sequences, even if they are of low quality.  It  also detects/corrects sequencing errors that lead to frameshifts.  ESTScan is not a gene prediction program , nor is it an open reading frame detector.  In fact, its strength lies in the fact that it does not require an open reading frame to detect a coding region.  The program may miss a few translated amino acids at termini, but detects coding regions with high selectivity and sensitivity.", "URL: https://quay.io/repository/biocontainers/estscan",
          },
        },
        ["estscan/ctr-3.0--r3.3.1_0"] = {
          Category = "['Annotation']",
          Description = "ESTScan can detect coding regions (CDS) in DNA sequences, even if they are of low quality.  It  also detects/corrects sequencing errors that lead to frameshifts.  ESTScan is not a gene prediction program , nor is it an open reading frame detector.  In fact, its strength lies in the fact that it does not require an open reading frame to detect a coding region.  The program may miss a few translated amino acids at termini, but detects coding regions with high selectivity and sensitivity.",
          Name = "estscan",
          URL = "https://quay.io/repository/biocontainers/estscan",
          Version = "ctr-3.0--r3.3.1_0",
          canonical = "ctr-3.0--r3.3.1_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/estscan/ctr-3.0--r3.3.1_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/estscan:3.0--r3.3.1_0, which exposes the
following programs:

 - ESTScan
 - ESTScan1
 - R
 - Rscript
 - annotate
 - bdftogd
 - bl2seq
 - blastall
 - blastclust
 - blastpgp
 - bmp2tiff
 - copymat
 - estscan
 - fastacmd
 - fetch
 - formatdb
 - formatrpsdb
 - gd2copypal
 - gd2togif
 - gd2topng
 - gdcmpgif
 - gdlib-config
 - gdparttopng
 - gdtopng
 - genccode
 - gencmn
 - gennorm2
 - gensprep
 - gif2tiff
 - giftogd2
 - gnuplot
 - icupkg
 - impala
 - indexer
 - makemat
 - megablast
 - netfetch
 - perl5.22.0
 - pngtogd
 - pngtogd2
 - ras2tiff
 - rgb2ycbcr
 - rpsblast
 - seedtop
 - thumbnail
 - uconv
 - webpng

This container was pulled from:

	https://quay.io/repository/biocontainers/estscan

If you encounter errors in estscan or need help running the
tools it contains, please contact the developer at

	http://myhits.isb-sib.ch/cgi-bin/estscan

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.*r.000000003.000000003.000000001.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000003.*r.000000003.000000003.000000001.*_.*zfinal",
          whatis = {
            "Name: estscan", "Version: ctr-3.0--r3.3.1_0", "Category: ['Annotation']", "Keywords: ['Genomics', 'Transcriptomics']", "Description: ESTScan can detect coding regions (CDS) in DNA sequences, even if they are of low quality.  It  also detects/corrects sequencing errors that lead to frameshifts.  ESTScan is not a gene prediction program , nor is it an open reading frame detector.  In fact, its strength lies in the fact that it does not require an open reading frame to detect a coding region.  The program may miss a few translated amino acids at termini, but detects coding regions with high selectivity and sensitivity.", "URL: https://quay.io/repository/biocontainers/estscan",
          },
        },
      },
    },
    eta = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["eta/ctr-0.9.7--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The eta package",
          Name = "eta",
          URL = "https://quay.io/repository/biocontainers/eta",
          Version = "ctr-0.9.7--py27_0",
          canonical = "ctr-0.9.7--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eta/ctr-0.9.7--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eta:0.9.7--py27_0, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/eta

If you encounter errors in eta or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eta

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: eta", "Version: ctr-0.9.7--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eta package", "URL: https://quay.io/repository/biocontainers/eta",
          },
        },
        ["eta/ctr-0.9.7--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The eta package",
          Name = "eta",
          URL = "https://quay.io/repository/biocontainers/eta",
          Version = "ctr-0.9.7--py27_1",
          canonical = "ctr-0.9.7--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eta/ctr-0.9.7--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eta:0.9.7--py27_1, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/eta

If you encounter errors in eta or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eta

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: eta", "Version: ctr-0.9.7--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eta package", "URL: https://quay.io/repository/biocontainers/eta",
          },
        },
        ["eta/ctr-0.9.7--py34_0"] = {
          Category = "['Bioinformatics']",
          Description = "The eta package",
          Name = "eta",
          URL = "https://quay.io/repository/biocontainers/eta",
          Version = "ctr-0.9.7--py34_0",
          canonical = "ctr-0.9.7--py34_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eta/ctr-0.9.7--py34_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eta:0.9.7--py34_0, which exposes the
following programs:

 - 2to3-3.4
 - easy_install-3.4
 - idle3.4
 - pydoc3.4
 - python3.4
 - python3.4-config
 - python3.4m
 - python3.4m-config
 - pyvenv-3.4

This container was pulled from:

	https://quay.io/repository/biocontainers/eta

If you encounter errors in eta or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eta

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000034.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000034.*_.*zfinal",
          whatis = {
            "Name: eta", "Version: ctr-0.9.7--py34_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eta package", "URL: https://quay.io/repository/biocontainers/eta",
          },
        },
        ["eta/ctr-0.9.7--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The eta package",
          Name = "eta",
          URL = "https://quay.io/repository/biocontainers/eta",
          Version = "ctr-0.9.7--py35_0",
          canonical = "ctr-0.9.7--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eta/ctr-0.9.7--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eta:0.9.7--py35_0, which exposes the
following programs:

 - easy_install-3.5
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/eta

If you encounter errors in eta or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eta

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: eta", "Version: ctr-0.9.7--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eta package", "URL: https://quay.io/repository/biocontainers/eta",
          },
        },
        ["eta/ctr-0.9.7--py35_1"] = {
          Category = "['Bioinformatics']",
          Description = "The eta package",
          Name = "eta",
          URL = "https://quay.io/repository/biocontainers/eta",
          Version = "ctr-0.9.7--py35_1",
          canonical = "ctr-0.9.7--py35_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eta/ctr-0.9.7--py35_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eta:0.9.7--py35_1, which exposes the
following programs:

 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/eta

If you encounter errors in eta or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eta

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000035.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000035.*_.000000001.*zfinal",
          whatis = {
            "Name: eta", "Version: ctr-0.9.7--py35_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eta package", "URL: https://quay.io/repository/biocontainers/eta",
          },
        },
        ["eta/ctr-0.9.7--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The eta package",
          Name = "eta",
          URL = "https://quay.io/repository/biocontainers/eta",
          Version = "ctr-0.9.7--py36_0",
          canonical = "ctr-0.9.7--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eta/ctr-0.9.7--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eta:0.9.7--py36_0, which exposes the
following programs:

 - easy_install-3.6

This container was pulled from:

	https://quay.io/repository/biocontainers/eta

If you encounter errors in eta or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eta

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: eta", "Version: ctr-0.9.7--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eta package", "URL: https://quay.io/repository/biocontainers/eta",
          },
        },
        ["eta/ctr-0.9.7--py36_1"] = {
          Category = "['Bioinformatics']",
          Description = "The eta package",
          Name = "eta",
          URL = "https://quay.io/repository/biocontainers/eta",
          Version = "ctr-0.9.7--py36_1",
          canonical = "ctr-0.9.7--py36_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/eta/ctr-0.9.7--py36_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/eta:0.9.7--py36_1, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/eta

If you encounter errors in eta or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/eta

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000036.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000009.000000007.*y.000000036.*_.000000001.*zfinal",
          whatis = {
            "Name: eta", "Version: ctr-0.9.7--py36_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The eta package", "URL: https://quay.io/repository/biocontainers/eta",
          },
        },
      },
    },
    ete2 = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ete2/ctr-2.3.10--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ete2 package",
          Name = "ete2",
          URL = "https://quay.io/repository/biocontainers/ete2",
          Version = "ctr-2.3.10--py27_1",
          canonical = "ctr-2.3.10--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete2/ctr-2.3.10--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete2:2.3.10--py27_1, which exposes the
following programs:

 - assistant-qt4
 - designer-qt4
 - ete
 - lconvert-qt4
 - linguist-qt4
 - lrelease-qt4
 - lupdate-qt4
 - moc-qt4
 - pixeltool-qt4
 - pylupdate4
 - pyrcc4
 - pyuic4
 - qcollectiongenerator-qt4
 - qdbus-qt4
 - qdbuscpp2xml-qt4
 - qdbusviewer-qt4
 - qdbusxml2cpp-qt4
 - qdoc3-qt4
 - qhelpconverter-qt4
 - qhelpgenerator-qt4
 - qmake-qt4
 - qmlplugindump-qt4
 - qmlviewer-qt4
 - qt3to4-qt4
 - qtconfig-qt4
 - qttracereplay-qt4
 - rcc-qt4
 - sip
 - uic-qt4
 - xmlpatterns-qt4
 - xmlpatternsvalidator-qt4
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete2

If you encounter errors in ete2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000003.000000010.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000003.000000010.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: ete2", "Version: ctr-2.3.10--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete2 package", "URL: https://quay.io/repository/biocontainers/ete2",
          },
        },
        ["ete2/ctr-2.3.10--py27_2"] = {
          Category = "['Bioinformatics']",
          Description = "The ete2 package",
          Name = "ete2",
          URL = "https://quay.io/repository/biocontainers/ete2",
          Version = "ctr-2.3.10--py27_2",
          canonical = "ctr-2.3.10--py27_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete2/ctr-2.3.10--py27_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete2:2.3.10--py27_2, which exposes the
following programs:

 - assistant-qt4
 - designer-qt4
 - ete
 - lconvert-qt4
 - linguist-qt4
 - lrelease-qt4
 - lupdate-qt4
 - moc-qt4
 - pixeltool-qt4
 - pylupdate4
 - pyrcc4
 - pyuic4
 - qcollectiongenerator-qt4
 - qdbus-qt4
 - qdbuscpp2xml-qt4
 - qdbusviewer-qt4
 - qdbusxml2cpp-qt4
 - qdoc3-qt4
 - qhelpconverter-qt4
 - qhelpgenerator-qt4
 - qmake-qt4
 - qmlplugindump-qt4
 - qmlviewer-qt4
 - qt3to4-qt4
 - qtconfig-qt4
 - qttracereplay-qt4
 - rcc-qt4
 - sip
 - uic-qt4
 - xmlpatterns-qt4
 - xmlpatternsvalidator-qt4
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete2

If you encounter errors in ete2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 19,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000003.000000010.*y.000000027.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000003.000000010.*y.000000027.*_.000000002.*zfinal",
          whatis = {
            "Name: ete2", "Version: ctr-2.3.10--py27_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete2 package", "URL: https://quay.io/repository/biocontainers/ete2",
          },
        },
        ["ete2/ctr-2.3.10--py27h24bf2e0_3"] = {
          Category = "['Bioinformatics']",
          Description = "The ete2 package",
          Name = "ete2",
          URL = "https://quay.io/repository/biocontainers/ete2",
          Version = "ctr-2.3.10--py27h24bf2e0_3",
          canonical = "ctr-2.3.10--py27h24bf2e0_3",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete2/ctr-2.3.10--py27h24bf2e0_3.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete2:2.3.10--py27h24bf2e0_3, which exposes the
following programs:

 - assistant
 - designer
 - ete
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - pixeltool
 - pylupdate4
 - pyrcc4
 - pyuic4
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc3
 - qhelpconverter
 - qhelpgenerator
 - qmake
 - qmlplugindump
 - qmlviewer
 - qt3to4
 - qtconfig
 - qttracereplay
 - rcc
 - sip
 - uic
 - xmlpatterns
 - xmlpatternsvalidator
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete2

If you encounter errors in ete2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 27,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000003.000000010.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000003.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000003.000000010.*y.000000027.*h.000000024.*bf.000000002.*e.*_.000000003.*zfinal",
          whatis = {
            "Name: ete2", "Version: ctr-2.3.10--py27h24bf2e0_3", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete2 package", "URL: https://quay.io/repository/biocontainers/ete2",
          },
        },
      },
    },
    ete3 = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["ete3/ctr-3.0.0b35--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ete3 package",
          Name = "ete3",
          URL = "https://quay.io/repository/biocontainers/ete3",
          Version = "ctr-3.0.0b35--py27_1",
          canonical = "ctr-3.0.0b35--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete3/ctr-3.0.0b35--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete3:3.0.0b35--py27_1, which exposes the
following programs:

 - assistant
 - designer
 - ete3
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - pixeltool
 - pylupdate4
 - pyrcc4
 - pyuic4
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc3
 - qhelpconverter
 - qhelpgenerator
 - qmake
 - qmlplugindump
 - qmlviewer
 - qt3to4
 - qtconfig
 - qttracereplay
 - rcc
 - sip
 - uic
 - xmlpatterns
 - xmlpatternsvalidator
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete3

If you encounter errors in ete3 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 21,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.*b.000000035.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.*b.000000035.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: ete3", "Version: ctr-3.0.0b35--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete3 package", "URL: https://quay.io/repository/biocontainers/ete3",
          },
        },
        ["ete3/ctr-3.0.0b35--py34_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ete3 package",
          Name = "ete3",
          URL = "https://quay.io/repository/biocontainers/ete3",
          Version = "ctr-3.0.0b35--py34_1",
          canonical = "ctr-3.0.0b35--py34_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete3/ctr-3.0.0b35--py34_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete3:3.0.0b35--py34_1, which exposes the
following programs:

 - 2to3-3.4
 - assistant
 - designer
 - easy_install-3.4
 - ete3
 - idle3.4
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - pixeltool
 - pydoc3.4
 - pylupdate4
 - pyrcc4
 - python3.4
 - python3.4-config
 - python3.4m
 - python3.4m-config
 - pyuic4
 - pyvenv-3.4
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc3
 - qhelpconverter
 - qhelpgenerator
 - qmake
 - qmlplugindump
 - qmlviewer
 - qt3to4
 - qtconfig
 - qttracereplay
 - rcc
 - sip
 - uic
 - xmlpatterns
 - xmlpatternsvalidator
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete3

If you encounter errors in ete3 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 21,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.*b.000000035.*y.000000034.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.*b.000000035.*y.000000034.*_.000000001.*zfinal",
          whatis = {
            "Name: ete3", "Version: ctr-3.0.0b35--py34_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete3 package", "URL: https://quay.io/repository/biocontainers/ete3",
          },
        },
        ["ete3/ctr-3.0.0b35--py35_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ete3 package",
          Name = "ete3",
          URL = "https://quay.io/repository/biocontainers/ete3",
          Version = "ctr-3.0.0b35--py35_1",
          canonical = "ctr-3.0.0b35--py35_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete3/ctr-3.0.0b35--py35_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete3:3.0.0b35--py35_1, which exposes the
following programs:

 - assistant
 - bmp2tiff
 - designer
 - easy_install-3.5
 - ete3
 - gif2tiff
 - idle3.5
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - pixeltool
 - pylupdate4
 - pyrcc4
 - python3.5-config
 - python3.5m-config
 - pyuic4
 - pyvenv-3.5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc3
 - qhelpconverter
 - qhelpgenerator
 - qmake
 - qmlplugindump
 - qmlviewer
 - qt3to4
 - qtconfig
 - qttracereplay
 - ras2tiff
 - rcc
 - rgb2ycbcr
 - sip
 - thumbnail
 - uic
 - xmlpatterns
 - xmlpatternsvalidator
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete3

If you encounter errors in ete3 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 21,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.*b.000000035.*y.000000035.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.*b.000000035.*y.000000035.*_.000000001.*zfinal",
          whatis = {
            "Name: ete3", "Version: ctr-3.0.0b35--py35_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete3 package", "URL: https://quay.io/repository/biocontainers/ete3",
          },
        },
        ["ete3/ctr-3.0.0b35--py36_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ete3 package",
          Name = "ete3",
          URL = "https://quay.io/repository/biocontainers/ete3",
          Version = "ctr-3.0.0b35--py36_1",
          canonical = "ctr-3.0.0b35--py36_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete3/ctr-3.0.0b35--py36_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete3:3.0.0b35--py36_1, which exposes the
following programs:

 - assistant
 - designer
 - easy_install-3.6
 - ete3
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - pixeltool
 - pylupdate4
 - pyrcc4
 - pyuic4
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc3
 - qhelpconverter
 - qhelpgenerator
 - qmake
 - qmlplugindump
 - qmlviewer
 - qt3to4
 - qtconfig
 - qttracereplay
 - rcc
 - sip
 - uic
 - xmlpatterns
 - xmlpatternsvalidator
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete3

If you encounter errors in ete3 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 21,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.*b.000000035.*y.000000036.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.*b.000000035.*y.000000036.*_.000000001.*zfinal",
          whatis = {
            "Name: ete3", "Version: ctr-3.0.0b35--py36_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete3 package", "URL: https://quay.io/repository/biocontainers/ete3",
          },
        },
        ["ete3/ctr-3.1.1--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ete3 package",
          Name = "ete3",
          URL = "https://quay.io/repository/biocontainers/ete3",
          Version = "ctr-3.1.1--py27_0",
          canonical = "ctr-3.1.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete3/ctr-3.1.1--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete3:3.1.1--py27_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - dbus-launch
 - designer
 - ete3
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete3

If you encounter errors in ete3 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000001.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000001.000000001.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: ete3", "Version: ctr-3.1.1--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete3 package", "URL: https://quay.io/repository/biocontainers/ete3",
          },
        },
        ["ete3/ctr-3.1.1--py35_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ete3 package",
          Name = "ete3",
          URL = "https://quay.io/repository/biocontainers/ete3",
          Version = "ctr-3.1.1--py35_0",
          canonical = "ctr-3.1.1--py35_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete3/ctr-3.1.1--py35_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete3:3.1.1--py35_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - dbus-launch
 - designer
 - ete3
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - idle3.5
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - python3.5-config
 - python3.5m-config
 - pyuic5
 - pyvenv-3.5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete3

If you encounter errors in ete3 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000001.000000001.*y.000000035.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000001.000000001.*y.000000035.*_.*zfinal",
          whatis = {
            "Name: ete3", "Version: ctr-3.1.1--py35_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete3 package", "URL: https://quay.io/repository/biocontainers/ete3",
          },
        },
        ["ete3/ctr-3.1.1--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The ete3 package",
          Name = "ete3",
          URL = "https://quay.io/repository/biocontainers/ete3",
          Version = "ctr-3.1.1--py36_0",
          canonical = "ctr-3.1.1--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete3/ctr-3.1.1--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete3:3.1.1--py36_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - dbus-launch
 - designer
 - ete3
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete3

If you encounter errors in ete3 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000001.000000001.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000001.000000001.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: ete3", "Version: ctr-3.1.1--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete3 package", "URL: https://quay.io/repository/biocontainers/ete3",
          },
        },
        ["ete3/ctr-3.1.1--pyh24bf2e0_1"] = {
          Category = "['Bioinformatics']",
          Description = "The ete3 package",
          Name = "ete3",
          URL = "https://quay.io/repository/biocontainers/ete3",
          Version = "ctr-3.1.1--pyh24bf2e0_1",
          canonical = "ctr-3.1.1--pyh24bf2e0_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/ete3/ctr-3.1.1--pyh24bf2e0_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/ete3:3.1.1--pyh24bf2e0_1, which exposes the
following programs:

 - assistant
 - designer
 - ete3
 - insserv
 - install_packages
 - lconvert
 - linguist
 - locale-gen
 - lrelease
 - lupdate
 - moc
 - perl5.20.2
 - pixeltool
 - pylupdate4
 - pyrcc4
 - pyuic4
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc3
 - qhelpconverter
 - qhelpgenerator
 - qmake
 - qmlplugindump
 - qmlviewer
 - qt3to4
 - qtconfig
 - qttracereplay
 - rcc
 - sip
 - uic
 - update-locale
 - validlocale
 - xmlpatterns
 - xmlpatternsvalidator
 - xslt-config
 - xsltproc

This container was pulled from:

	https://quay.io/repository/biocontainers/ete3

If you encounter errors in ete3 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/ete3

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 24,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000001.000000001.*yh.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000001.000000001.*yh.000000024.*bf.000000002.*e.*_.000000001.*zfinal",
          whatis = {
            "Name: ete3", "Version: ctr-3.1.1--pyh24bf2e0_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The ete3 package", "URL: https://quay.io/repository/biocontainers/ete3",
          },
        },
      },
    },
    evidencemodeler = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["evidencemodeler/ctr-1.1.1--0"] = {
          Category = "['Bioinformatics']",
          Description = "The evidencemodeler package",
          Name = "evidencemodeler",
          URL = "https://quay.io/repository/biocontainers/evidencemodeler",
          Version = "ctr-1.1.1--0",
          canonical = "ctr-1.1.1--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/evidencemodeler/ctr-1.1.1--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/evidencemodeler:1.1.1--0, which exposes the
following programs:

 - evidence_modeler.pl
 - perl5.26.2

This container was pulled from:

	https://quay.io/repository/biocontainers/evidencemodeler

If you encounter errors in evidencemodeler or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/evidencemodeler

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000001.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000001.000000001.*zfinal",
          whatis = {
            "Name: evidencemodeler", "Version: ctr-1.1.1--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The evidencemodeler package", "URL: https://quay.io/repository/biocontainers/evidencemodeler",
          },
        },
        ["evidencemodeler/ctr-v1.1.1--0"] = {
          Category = "['Bioinformatics']",
          Description = "The evidencemodeler package",
          Name = "evidencemodeler",
          URL = "https://quay.io/repository/biocontainers/evidencemodeler",
          Version = "ctr-v1.1.1--0",
          canonical = "ctr-v1.1.1--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/evidencemodeler/ctr-v1.1.1--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/evidencemodeler:v1.1.1--0, which exposes the
following programs:

 - evidence_modeler.pl
 - perl5.26.2

This container was pulled from:

	https://quay.io/repository/biocontainers/evidencemodeler

If you encounter errors in evidencemodeler or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/evidencemodeler

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 14,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000001.000000001.000000001.*zfinal",
          wV = "*ctr.*v.000000001.000000001.000000001.*zfinal",
          whatis = {
            "Name: evidencemodeler", "Version: ctr-v1.1.1--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The evidencemodeler package", "URL: https://quay.io/repository/biocontainers/evidencemodeler",
          },
        },
      },
    },
    evofold2 = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["evofold2/ctr-0.1--0"] = {
          Category = "['Bioinformatics']",
          Description = "The evofold2 package",
          Name = "evofold2",
          URL = "https://quay.io/repository/biocontainers/evofold2",
          Version = "ctr-0.1--0",
          canonical = "ctr-0.1--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/evofold2/ctr-0.1--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/evofold2:0.1--0, which exposes the
following programs:

 - EvoFoldV2
 - EvoFoldV2.sh
 - dfgEval
 - dfgTrain
 - grammarTrain
 - multinomial

This container was pulled from:

	https://quay.io/repository/biocontainers/evofold2

If you encounter errors in evofold2 or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/evofold2

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000001.*zfinal",
          whatis = {
            "Name: evofold2", "Version: ctr-0.1--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The evofold2 package", "URL: https://quay.io/repository/biocontainers/evofold2",
          },
        },
      },
    },
    exonerate = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["exonerate/ctr-2.2.0--0"] = {
          Category = "['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']",
          Description = "A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.",
          Name = "exonerate",
          URL = "https://quay.io/repository/biocontainers/exonerate",
          Version = "ctr-2.2.0--0",
          canonical = "ctr-2.2.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/exonerate/ctr-2.2.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/exonerate:2.2.0--0, which exposes the
following programs:

 - esd2esi
 - exonerate
 - exonerate-server
 - fasta2esd
 - fastaannotatecdna
 - fastachecksum
 - fastaclean
 - fastaclip
 - fastacomposition
 - fastadiff
 - fastaexplode
 - fastafetch
 - fastahardmask
 - fastaindex
 - fastalength
 - fastanrdb
 - fastaoverlap
 - fastareformat
 - fastaremove
 - fastarevcomp
 - fastasoftmask
 - fastasort
 - fastasplit
 - fastasubseq
 - fastatranslate
 - fastavalidcds
 - ipcress

This container was pulled from:

	https://quay.io/repository/biocontainers/exonerate

If you encounter errors in exonerate or need help running the
tools it contains, please contact the developer at

	http://www.ebi.ac.uk/%7Eguy/exonerate

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000002.*zfinal",
          whatis = {
            "Name: exonerate", "Version: ctr-2.2.0--0", "Category: ['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']", "Keywords: ['Sequence analysis', 'Sequence sites, features and motifs', 'Molecular interactions, pathways and networks']", "Description: A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.", "URL: https://quay.io/repository/biocontainers/exonerate",
          },
        },
        ["exonerate/ctr-2.2.0--1"] = {
          Category = "['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']",
          Description = "A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.",
          Name = "exonerate",
          URL = "https://quay.io/repository/biocontainers/exonerate",
          Version = "ctr-2.2.0--1",
          canonical = "ctr-2.2.0--1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/exonerate/ctr-2.2.0--1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/exonerate:2.2.0--1, which exposes the
following programs:

 - esd2esi
 - exonerate
 - exonerate-server
 - fasta2esd
 - fastaannotatecdna
 - fastachecksum
 - fastaclean
 - fastaclip
 - fastacomposition
 - fastadiff
 - fastaexplode
 - fastafetch
 - fastahardmask
 - fastaindex
 - fastalength
 - fastanrdb
 - fastaoverlap
 - fastareformat
 - fastaremove
 - fastarevcomp
 - fastasoftmask
 - fastasort
 - fastasplit
 - fastasubseq
 - fastatranslate
 - fastavalidcds
 - ipcress

This container was pulled from:

	https://quay.io/repository/biocontainers/exonerate

If you encounter errors in exonerate or need help running the
tools it contains, please contact the developer at

	http://www.ebi.ac.uk/%7Eguy/exonerate

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000002.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000002.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: exonerate", "Version: ctr-2.2.0--1", "Category: ['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']", "Keywords: ['Sequence analysis', 'Sequence sites, features and motifs', 'Molecular interactions, pathways and networks']", "Description: A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.", "URL: https://quay.io/repository/biocontainers/exonerate",
          },
        },
        ["exonerate/ctr-2.4.0--0"] = {
          Category = "['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']",
          Description = "A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.",
          Name = "exonerate",
          URL = "https://quay.io/repository/biocontainers/exonerate",
          Version = "ctr-2.4.0--0",
          canonical = "ctr-2.4.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/exonerate/ctr-2.4.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/exonerate:2.4.0--0, which exposes the
following programs:

 - esd2esi
 - exonerate
 - exonerate-server
 - fasta2esd
 - fastaannotatecdna
 - fastachecksum
 - fastaclean
 - fastaclip
 - fastacomposition
 - fastadiff
 - fastaexplode
 - fastafetch
 - fastahardmask
 - fastaindex
 - fastalength
 - fastanrdb
 - fastaoverlap
 - fastareformat
 - fastaremove
 - fastarevcomp
 - fastasoftmask
 - fastasort
 - fastasplit
 - fastasubseq
 - fastatranslate
 - fastavalidcds
 - ipcress

This container was pulled from:

	https://quay.io/repository/biocontainers/exonerate

If you encounter errors in exonerate or need help running the
tools it contains, please contact the developer at

	http://www.ebi.ac.uk/%7Eguy/exonerate

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000004.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000004.*zfinal",
          whatis = {
            "Name: exonerate", "Version: ctr-2.4.0--0", "Category: ['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']", "Keywords: ['Sequence analysis', 'Sequence sites, features and motifs', 'Molecular interactions, pathways and networks']", "Description: A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.", "URL: https://quay.io/repository/biocontainers/exonerate",
          },
        },
        ["exonerate/ctr-2.4.0--hb9387b2_1"] = {
          Category = "['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']",
          Description = "A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.",
          Name = "exonerate",
          URL = "https://quay.io/repository/biocontainers/exonerate",
          Version = "ctr-2.4.0--hb9387b2_1",
          canonical = "ctr-2.4.0--hb9387b2_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/exonerate/ctr-2.4.0--hb9387b2_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/exonerate:2.4.0--hb9387b2_1, which exposes the
following programs:

 - esd2esi
 - exonerate
 - exonerate-server
 - fasta2esd
 - fastaannotatecdna
 - fastachecksum
 - fastaclean
 - fastaclip
 - fastacomposition
 - fastadiff
 - fastaexplode
 - fastafetch
 - fastahardmask
 - fastaindex
 - fastalength
 - fastanrdb
 - fastaoverlap
 - fastareformat
 - fastaremove
 - fastarevcomp
 - fastasoftmask
 - fastasort
 - fastasplit
 - fastasubseq
 - fastatranslate
 - fastavalidcds
 - ipcress

This container was pulled from:

	https://quay.io/repository/biocontainers/exonerate

If you encounter errors in exonerate or need help running the
tools it contains, please contact the developer at

	http://www.ebi.ac.uk/%7Eguy/exonerate

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000004.*hb.000009387.*b.000000002.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000004.*hb.000009387.*b.000000002.*_.000000001.*zfinal",
          whatis = {
            "Name: exonerate", "Version: ctr-2.4.0--hb9387b2_1", "Category: ['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']", "Keywords: ['Sequence analysis', 'Sequence sites, features and motifs', 'Molecular interactions, pathways and networks']", "Description: A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.", "URL: https://quay.io/repository/biocontainers/exonerate",
          },
        },
        ["exonerate/ctr-v2.4.0-2-deb_cv1"] = {
          Category = "['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']",
          Description = "A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.",
          Name = "exonerate",
          URL = "https://hub.docker.com/r/biocontainers/exonerate",
          Version = "ctr-v2.4.0-2-deb_cv1",
          canonical = "ctr-v2.4.0-2-deb_cv1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/exonerate/ctr-v2.4.0-2-deb_cv1.lua",
          help = [[This is a module file for the container biocontainers/exonerate:v2.4.0-2-deb_cv1, which exposes the
following programs:

 - esd2esi
 - exonerate
 - exonerate-server
 - fasta2esd
 - fastaannotatecdna
 - fastachecksum
 - fastaclean
 - fastaclip
 - fastacomposition
 - fastadiff
 - fastaexplode
 - fastafetch
 - fastahardmask
 - fastaindex
 - fastalength
 - fastanrdb
 - fastaoverlap
 - fastareformat
 - fastaremove
 - fastarevcomp
 - fastasoftmask
 - fastasort
 - fastasplit
 - fastasubseq
 - fastatranslate
 - fastavalidcds
 - ipcress

This container was pulled from:

	https://hub.docker.com/r/biocontainers/exonerate

If you encounter errors in exonerate or need help running the
tools it contains, please contact the developer at

	http://www.ebi.ac.uk/%7Eguy/exonerate

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 21,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*v.000000002.000000004.*zfinal-.000000002.*deb.*_.*cv.000000001.*zfinal",
          wV = "*ctr.*v.000000002.000000004.*zfinal-.000000002.*deb.*_.*cv.000000001.*zfinal",
          whatis = {
            "Name: exonerate", "Version: ctr-v2.4.0-2-deb_cv1", "Category: ['Pairwise sequence alignment', 'Protein threading', 'Genome alignment']", "Keywords: ['Sequence analysis', 'Sequence sites, features and motifs', 'Molecular interactions, pathways and networks']", "Description: A tool for pairwise sequence alignment. It enables alignment for DNA-DNA and DNA-protein pairs and also gapped and ungapped alignment.", "URL: https://hub.docker.com/r/biocontainers/exonerate",
          },
        },
      },
    },
    expansionhunter = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["expansionhunter/ctr-2.0.6--0"] = {
          Category = "['Bioinformatics']",
          Description = "The expansionhunter package",
          Name = "expansionhunter",
          URL = "https://quay.io/repository/biocontainers/expansionhunter",
          Version = "ctr-2.0.6--0",
          canonical = "ctr-2.0.6--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/expansionhunter/ctr-2.0.6--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/expansionhunter:2.0.6--0, which exposes the
following programs:

 - ExpansionHunter
 - easy_install-3.5
 - idle3.5
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5

This container was pulled from:

	https://quay.io/repository/biocontainers/expansionhunter

If you encounter errors in expansionhunter or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/expansionhunter

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000000.000000006.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000000.000000006.*zfinal",
          whatis = {
            "Name: expansionhunter", "Version: ctr-2.0.6--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The expansionhunter package", "URL: https://quay.io/repository/biocontainers/expansionhunter",
          },
        },
        ["expansionhunter/ctr-2.0.8--0"] = {
          Category = "['Bioinformatics']",
          Description = "The expansionhunter package",
          Name = "expansionhunter",
          URL = "https://quay.io/repository/biocontainers/expansionhunter",
          Version = "ctr-2.0.8--0",
          canonical = "ctr-2.0.8--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/expansionhunter/ctr-2.0.8--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/expansionhunter:2.0.8--0, which exposes the
following programs:

 - ExpansionHunter
 - easy_install-3.6
 - genccode
 - gencmn
 - gennorm2
 - gensprep
 - icupkg
 - uconv

This container was pulled from:

	https://quay.io/repository/biocontainers/expansionhunter

If you encounter errors in expansionhunter or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/expansionhunter

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000000.000000008.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000000.000000008.*zfinal",
          whatis = {
            "Name: expansionhunter", "Version: ctr-2.0.8--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The expansionhunter package", "URL: https://quay.io/repository/biocontainers/expansionhunter",
          },
        },
        ["expansionhunter/ctr-2.0.8--hf66f9d2_1"] = {
          Category = "['Bioinformatics']",
          Description = "The expansionhunter package",
          Name = "expansionhunter",
          URL = "https://quay.io/repository/biocontainers/expansionhunter",
          Version = "ctr-2.0.8--hf66f9d2_1",
          canonical = "ctr-2.0.8--hf66f9d2_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/expansionhunter/ctr-2.0.8--hf66f9d2_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/expansionhunter:2.0.8--hf66f9d2_1, which exposes the
following programs:

 - ExpansionHunter

This container was pulled from:

	https://quay.io/repository/biocontainers/expansionhunter

If you encounter errors in expansionhunter or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/expansionhunter

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000000.000000008.*hf.000000066.*f.000000009.*d.000000002.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000000.000000008.*hf.000000066.*f.000000009.*d.000000002.*_.000000001.*zfinal",
          whatis = {
            "Name: expansionhunter", "Version: ctr-2.0.8--hf66f9d2_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The expansionhunter package", "URL: https://quay.io/repository/biocontainers/expansionhunter",
          },
        },
      },
    },
    exparna = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["exparna/ctr-1.0.1--0"] = {
          Category = "['Structure-based sequence alignment']",
          Description = "Structure motif based alignment generation.",
          Name = "exparna",
          URL = "https://quay.io/repository/biocontainers/exparna",
          Version = "ctr-1.0.1--0",
          canonical = "ctr-1.0.1--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/exparna/ctr-1.0.1--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/exparna:1.0.1--0, which exposes the
following programs:

 - ExpaRNA
 - Kinfold
 - RNA2Dfold
 - RNALalifold
 - RNALfold
 - RNAPKplex
 - RNAaliduplex
 - RNAalifold
 - RNAcofold
 - RNAdistance
 - RNAduplex
 - RNAeval
 - RNAfold
 - RNAforester
 - RNAheat
 - RNAinverse
 - RNApaln
 - RNAparconv
 - RNApdist
 - RNAplex
 - RNAplfold
 - RNAplot
 - RNApvmin
 - RNAsnoop
 - RNAsubopt
 - RNAup
 - aclocal.bak
 - autoheader.bak
 - autom4te.bak
 - automake.bak
 - autoreconf.bak
 - autoscan.bak
 - autoupdate.bak
 - b2ct
 - ct2db
 - ifnames.bak
 - kinwalker
 - popt

This container was pulled from:

	https://quay.io/repository/biocontainers/exparna

If you encounter errors in exparna or need help running the
tools it contains, please contact the developer at

	http://www.bioinf.uni-freiburg.de/Software/#ExpaRNA-download

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000000.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000000.000000001.*zfinal",
          whatis = {
            "Name: exparna", "Version: ctr-1.0.1--0", "Category: ['Structure-based sequence alignment']", "Keywords: ['Sequence analysis']", "Description: Structure motif based alignment generation.", "URL: https://quay.io/repository/biocontainers/exparna",
          },
        },
        ["exparna/ctr-1.0.1--h470a237_1"] = {
          Category = "['Structure-based sequence alignment']",
          Description = "Structure motif based alignment generation.",
          Name = "exparna",
          URL = "https://quay.io/repository/biocontainers/exparna",
          Version = "ctr-1.0.1--h470a237_1",
          canonical = "ctr-1.0.1--h470a237_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/exparna/ctr-1.0.1--h470a237_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/exparna:1.0.1--h470a237_1, which exposes the
following programs:

 - AnalyseDists
 - AnalyseSeqs
 - ExpaRNA
 - Kinfold
 - RNA2Dfold
 - RNALalifold
 - RNALfold
 - RNAPKplex
 - RNAaliduplex
 - RNAalifold
 - RNAcofold
 - RNAdistance
 - RNAduplex
 - RNAeval
 - RNAfold
 - RNAforester
 - RNAheat
 - RNAinverse
 - RNAlocmin
 - RNApaln
 - RNAparconv
 - RNApdist
 - RNAplex
 - RNAplfold
 - RNAplot
 - RNApvmin
 - RNAsnoop
 - RNAsubopt
 - RNAup
 - b2ct
 - ct2db
 - kinwalker
 - popt

This container was pulled from:

	https://quay.io/repository/biocontainers/exparna

If you encounter errors in exparna or need help running the
tools it contains, please contact the developer at

	http://www.bioinf.uni-freiburg.de/Software/#ExpaRNA-download

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000000.000000001.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000000.000000001.*h.000000470.*a.000000237.*_.000000001.*zfinal",
          whatis = {
            "Name: exparna", "Version: ctr-1.0.1--h470a237_1", "Category: ['Structure-based sequence alignment']", "Keywords: ['Sequence analysis']", "Description: Structure motif based alignment generation.", "URL: https://quay.io/repository/biocontainers/exparna",
          },
        },
      },
    },
    expat = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["expat/ctr-2.1.0--0"] = {
          Category = "['Bioinformatics']",
          Description = "Expat is a stream oriented XML parser.  This means that you register handlers with the parser prior to starting the parse.  These handlers are called when the parser discovers the associated structures in the document being parsed.  A start tag is an example of the kind of structures for which you may register handlers.",
          Name = "expat",
          URL = "https://quay.io/repository/biocontainers/expat",
          Version = "ctr-2.1.0--0",
          canonical = "ctr-2.1.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/expat/ctr-2.1.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/expat:2.1.0--0, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/expat

If you encounter errors in expat or need help running the
tools it contains, please contact the developer at

	http://www.libexpat.org/

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000002.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000002.000000001.*zfinal",
          whatis = {
            "Name: expat", "Version: ctr-2.1.0--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: Expat is a stream oriented XML parser.  This means that you register handlers with the parser prior to starting the parse.  These handlers are called when the parser discovers the associated structures in the document being parsed.  A start tag is an example of the kind of structures for which you may register handlers.", "URL: https://quay.io/repository/biocontainers/expat",
          },
        },
      },
    },
    export2graphlan = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["export2graphlan/ctr-0.19--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The export2graphlan package",
          Name = "export2graphlan",
          URL = "https://quay.io/repository/biocontainers/export2graphlan",
          Version = "ctr-0.19--py27_0",
          canonical = "ctr-0.19--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/export2graphlan/ctr-0.19--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/export2graphlan:0.19--py27_0, which exposes the
following programs:

 - assistant
 - createpackage
 - designer
 - elf2e32_qtwrapper
 - export2graphlan.py
 - findtr
 - lconvert
 - libpng15-config
 - linguist
 - lrelease
 - lupdate
 - moc
 - patch_capabilities
 - pixeltool
 - pylupdate4
 - pyrcc4
 - pyuic4
 - qcollectiongenerator
 - qhelpconverter
 - qhelpgenerator
 - qmake
 - qmlplugindump
 - qmlviewer
 - qtconfig
 - qttracereplay
 - rcc
 - sip
 - syncqt
 - uic
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/export2graphlan

If you encounter errors in export2graphlan or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/export2graphlan

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 17,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000019.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000019.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: export2graphlan", "Version: ctr-0.19--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The export2graphlan package", "URL: https://quay.io/repository/biocontainers/export2graphlan",
          },
        },
        ["export2graphlan/ctr-0.19--py27_2"] = {
          Category = "['Bioinformatics']",
          Description = "The export2graphlan package",
          Name = "export2graphlan",
          URL = "https://quay.io/repository/biocontainers/export2graphlan",
          Version = "ctr-0.19--py27_2",
          canonical = "ctr-0.19--py27_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/export2graphlan/ctr-0.19--py27_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/export2graphlan:0.19--py27_2, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - dbus-launch
 - designer
 - export2graphlan.py
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - hclust2.py
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/export2graphlan

If you encounter errors in export2graphlan or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/export2graphlan

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 17,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000019.*y.000000027.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000019.*y.000000027.*_.000000002.*zfinal",
          whatis = {
            "Name: export2graphlan", "Version: ctr-0.19--py27_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The export2graphlan package", "URL: https://quay.io/repository/biocontainers/export2graphlan",
          },
        },
        ["export2graphlan/ctr-0.20--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The export2graphlan package",
          Name = "export2graphlan",
          URL = "https://quay.io/repository/biocontainers/export2graphlan",
          Version = "ctr-0.20--py27_0",
          canonical = "ctr-0.20--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/export2graphlan/ctr-0.20--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/export2graphlan:0.20--py27_0, which exposes the
following programs:

 - .dbus-post-link.sh
 - assistant
 - dbus-launch
 - designer
 - export2graphlan.py
 - fixqt4headers.pl
 - gst-device-monitor-1.0
 - gst-discoverer-1.0
 - gst-inspect-1.0
 - gst-launch-1.0
 - gst-play-1.0
 - gst-stats-1.0
 - gst-typefind-1.0
 - hclust2.py
 - lconvert
 - linguist
 - lrelease
 - lupdate
 - moc
 - pixeltool
 - pylupdate5
 - pyrcc5
 - pyuic5
 - qcollectiongenerator
 - qdbus
 - qdbuscpp2xml
 - qdbusviewer
 - qdbusxml2cpp
 - qdoc
 - qhelpconverter
 - qhelpgenerator
 - qlalr
 - qmake
 - qml
 - qmleasing
 - qmlimportscanner
 - qmllint
 - qmlmin
 - qmlplugindump
 - qmlprofiler
 - qmlscene
 - qmltestrunner
 - qtdiag
 - qtpaths
 - qtplugininfo
 - rcc
 - sip
 - syncqt.pl
 - uic
 - xmlpatterns
 - xmlpatternsvalidator

This container was pulled from:

	https://quay.io/repository/biocontainers/export2graphlan

If you encounter errors in export2graphlan or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/export2graphlan

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 17,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000020.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000020.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: export2graphlan", "Version: ctr-0.20--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The export2graphlan package", "URL: https://quay.io/repository/biocontainers/export2graphlan",
          },
        },
      },
    },
    express = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["express/ctr-1.5.1--h2d50403_1"] = {
          Category = "['Bioinformatics']",
          Description = "The express package",
          Name = "express",
          URL = "https://quay.io/repository/biocontainers/express",
          Version = "ctr-1.5.1--h2d50403_1",
          canonical = "ctr-1.5.1--h2d50403_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/express/ctr-1.5.1--h2d50403_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/express:1.5.1--h2d50403_1, which exposes the
following programs:

 - express

This container was pulled from:

	https://quay.io/repository/biocontainers/express

If you encounter errors in express or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/express

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 22,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.000000005.000000001.*h.000000002.*d.000050403.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.000000005.000000001.*h.000000002.*d.000050403.*_.000000001.*zfinal",
          whatis = {
            "Name: express", "Version: ctr-1.5.1--h2d50403_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The express package", "URL: https://quay.io/repository/biocontainers/express",
          },
        },
      },
    },
    extern = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["extern/ctr-0.2.1--py27_0"] = {
          Category = "['Bioinformatics']",
          Description = "The extern package",
          Name = "extern",
          URL = "https://quay.io/repository/biocontainers/extern",
          Version = "ctr-0.2.1--py27_0",
          canonical = "ctr-0.2.1--py27_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extern/ctr-0.2.1--py27_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extern:0.2.1--py27_0, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/extern

If you encounter errors in extern or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extern

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000027.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000027.*_.*zfinal",
          whatis = {
            "Name: extern", "Version: ctr-0.2.1--py27_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extern package", "URL: https://quay.io/repository/biocontainers/extern",
          },
        },
        ["extern/ctr-0.2.1--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The extern package",
          Name = "extern",
          URL = "https://quay.io/repository/biocontainers/extern",
          Version = "ctr-0.2.1--py27_1",
          canonical = "ctr-0.2.1--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extern/ctr-0.2.1--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extern:0.2.1--py27_1, which exposes the
following programs:

None - please invoke manually

This container was pulled from:

	https://quay.io/repository/biocontainers/extern

If you encounter errors in extern or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extern

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000002.000000001.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: extern", "Version: ctr-0.2.1--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extern package", "URL: https://quay.io/repository/biocontainers/extern",
          },
        },
      },
    },
    extract_fullseq = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["extract_fullseq/ctr-3.101--1"] = {
          Category = "['Bioinformatics']",
          Description = "The extract_fullseq package",
          Name = "extract_fullseq",
          URL = "https://quay.io/repository/biocontainers/extract_fullseq",
          Version = "ctr-3.101--1",
          canonical = "ctr-3.101--1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extract_fullseq/ctr-3.101--1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extract_fullseq:3.101--1, which exposes the
following programs:

 - extract_fullseq

This container was pulled from:

	https://quay.io/repository/biocontainers/extract_fullseq

If you encounter errors in extract_fullseq or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extract_fullseq

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000101.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000101.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: extract_fullseq", "Version: ctr-3.101--1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extract_fullseq package", "URL: https://quay.io/repository/biocontainers/extract_fullseq",
          },
        },
        ["extract_fullseq/ctr-3.101--2"] = {
          Category = "['Bioinformatics']",
          Description = "The extract_fullseq package",
          Name = "extract_fullseq",
          URL = "https://quay.io/repository/biocontainers/extract_fullseq",
          Version = "ctr-3.101--2",
          canonical = "ctr-3.101--2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extract_fullseq/ctr-3.101--2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extract_fullseq:3.101--2, which exposes the
following programs:

 - extract_fullseq

This container was pulled from:

	https://quay.io/repository/biocontainers/extract_fullseq

If you encounter errors in extract_fullseq or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extract_fullseq

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000101.*zfinal-.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000101.*zfinal-.000000002.*zfinal",
          whatis = {
            "Name: extract_fullseq", "Version: ctr-3.101--2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extract_fullseq package", "URL: https://quay.io/repository/biocontainers/extract_fullseq",
          },
        },
        ["extract_fullseq/ctr-3.101--3"] = {
          Category = "['Bioinformatics']",
          Description = "The extract_fullseq package",
          Name = "extract_fullseq",
          URL = "https://quay.io/repository/biocontainers/extract_fullseq",
          Version = "ctr-3.101--3",
          canonical = "ctr-3.101--3",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extract_fullseq/ctr-3.101--3.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extract_fullseq:3.101--3, which exposes the
following programs:

 - extract_fullseq

This container was pulled from:

	https://quay.io/repository/biocontainers/extract_fullseq

If you encounter errors in extract_fullseq or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extract_fullseq

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 13,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000003.000000101.*zfinal-.000000003.*zfinal",
          wV = "*ctr.*zfinal-.000000003.000000101.*zfinal-.000000003.*zfinal",
          whatis = {
            "Name: extract_fullseq", "Version: ctr-3.101--3", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extract_fullseq package", "URL: https://quay.io/repository/biocontainers/extract_fullseq",
          },
        },
      },
    },
    extract_genome_region = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["extract_genome_region/ctr-0.0.3--py27_1"] = {
          Category = "['Bioinformatics']",
          Description = "The extract_genome_region package",
          Name = "extract_genome_region",
          URL = "https://quay.io/repository/biocontainers/extract_genome_region",
          Version = "ctr-0.0.3--py27_1",
          canonical = "ctr-0.0.3--py27_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extract_genome_region/ctr-0.0.3--py27_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extract_genome_region:0.0.3--py27_1, which exposes the
following programs:

 - extract_genome_region
 - faidx
 - insserv
 - install_packages
 - locale-gen
 - perl5.20.2
 - update-locale
 - validlocale

This container was pulled from:

	https://quay.io/repository/biocontainers/extract_genome_region

If you encounter errors in extract_genome_region or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extract_genome_region

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000027.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000027.*_.000000001.*zfinal",
          whatis = {
            "Name: extract_genome_region", "Version: ctr-0.0.3--py27_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extract_genome_region package", "URL: https://quay.io/repository/biocontainers/extract_genome_region",
          },
        },
        ["extract_genome_region/ctr-0.0.3--py35_1"] = {
          Category = "['Bioinformatics']",
          Description = "The extract_genome_region package",
          Name = "extract_genome_region",
          URL = "https://quay.io/repository/biocontainers/extract_genome_region",
          Version = "ctr-0.0.3--py35_1",
          canonical = "ctr-0.0.3--py35_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extract_genome_region/ctr-0.0.3--py35_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extract_genome_region:0.0.3--py35_1, which exposes the
following programs:

 - extract_genome_region
 - faidx
 - idle3.5
 - insserv
 - install_packages
 - locale-gen
 - perl5.20.2
 - python3.5-config
 - python3.5m-config
 - pyvenv-3.5
 - update-locale
 - validlocale

This container was pulled from:

	https://quay.io/repository/biocontainers/extract_genome_region

If you encounter errors in extract_genome_region or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extract_genome_region

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000035.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000035.*_.000000001.*zfinal",
          whatis = {
            "Name: extract_genome_region", "Version: ctr-0.0.3--py35_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extract_genome_region package", "URL: https://quay.io/repository/biocontainers/extract_genome_region",
          },
        },
        ["extract_genome_region/ctr-0.0.3--py36_0"] = {
          Category = "['Bioinformatics']",
          Description = "The extract_genome_region package",
          Name = "extract_genome_region",
          URL = "https://quay.io/repository/biocontainers/extract_genome_region",
          Version = "ctr-0.0.3--py36_0",
          canonical = "ctr-0.0.3--py36_0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extract_genome_region/ctr-0.0.3--py36_0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extract_genome_region:0.0.3--py36_0, which exposes the
following programs:

 - easy_install-3.6
 - extract_genome_region
 - faidx
 - insserv
 - perl5.20.2
 - udevd

This container was pulled from:

	https://quay.io/repository/biocontainers/extract_genome_region

If you encounter errors in extract_genome_region or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extract_genome_region

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000036.*_.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000036.*_.*zfinal",
          whatis = {
            "Name: extract_genome_region", "Version: ctr-0.0.3--py36_0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extract_genome_region package", "URL: https://quay.io/repository/biocontainers/extract_genome_region",
          },
        },
        ["extract_genome_region/ctr-0.0.3--py36_1"] = {
          Category = "['Bioinformatics']",
          Description = "The extract_genome_region package",
          Name = "extract_genome_region",
          URL = "https://quay.io/repository/biocontainers/extract_genome_region",
          Version = "ctr-0.0.3--py36_1",
          canonical = "ctr-0.0.3--py36_1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extract_genome_region/ctr-0.0.3--py36_1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extract_genome_region:0.0.3--py36_1, which exposes the
following programs:

 - extract_genome_region
 - faidx
 - insserv
 - install_packages
 - locale-gen
 - perl5.20.2
 - update-locale
 - validlocale

This container was pulled from:

	https://quay.io/repository/biocontainers/extract_genome_region

If you encounter errors in extract_genome_region or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extract_genome_region

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 18,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000036.*_.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000000.000000000.000000003.*y.000000036.*_.000000001.*zfinal",
          whatis = {
            "Name: extract_genome_region", "Version: ctr-0.0.3--py36_1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extract_genome_region package", "URL: https://quay.io/repository/biocontainers/extract_genome_region",
          },
        },
      },
    },
    extractpirs = {
      defaultA = {},
      defaultT = {},
      dirT = {},
      fileT = {
        ["extractpirs/ctr-1.0--0"] = {
          Category = "['Bioinformatics']",
          Description = "The extractpirs package",
          Name = "extractpirs",
          URL = "https://quay.io/repository/biocontainers/extractpirs",
          Version = "ctr-1.0--0",
          canonical = "ctr-1.0--0",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extractpirs/ctr-1.0--0.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extractpirs:1.0--0, which exposes the
following programs:

 - extractPIRs

This container was pulled from:

	https://quay.io/repository/biocontainers/extractpirs

If you encounter errors in extractpirs or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extractpirs

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: extractpirs", "Version: ctr-1.0--0", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extractpirs package", "URL: https://quay.io/repository/biocontainers/extractpirs",
          },
        },
        ["extractpirs/ctr-1.0--1"] = {
          Category = "['Bioinformatics']",
          Description = "The extractpirs package",
          Name = "extractpirs",
          URL = "https://quay.io/repository/biocontainers/extractpirs",
          Version = "ctr-1.0--1",
          canonical = "ctr-1.0--1",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extractpirs/ctr-1.0--1.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extractpirs:1.0--1, which exposes the
following programs:

 - extractPIRs

This container was pulled from:

	https://quay.io/repository/biocontainers/extractpirs

If you encounter errors in extractpirs or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extractpirs

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 11,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*zfinal-.000000001.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*zfinal-.000000001.*zfinal",
          whatis = {
            "Name: extractpirs", "Version: ctr-1.0--1", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extractpirs package", "URL: https://quay.io/repository/biocontainers/extractpirs",
          },
        },
        ["extractpirs/ctr-1.0--he941832_2"] = {
          Category = "['Bioinformatics']",
          Description = "The extractpirs package",
          Name = "extractpirs",
          URL = "https://quay.io/repository/biocontainers/extractpirs",
          Version = "ctr-1.0--he941832_2",
          canonical = "ctr-1.0--he941832_2",
          fn = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf/extractpirs/ctr-1.0--he941832_2.lua",
          help = [[This is a module file for the container quay.io/biocontainers/extractpirs:1.0--he941832_2, which exposes the
following programs:

 - extractPIRs

This container was pulled from:

	https://quay.io/repository/biocontainers/extractpirs

If you encounter errors in extractpirs or need help running the
tools it contains, please contact the developer at

	https://quay.io/repository/biocontainers/extractpirs

For errors in the container or module file, please
submit a ticket at

	gzynda@tacc.utexas.edu
	https://portal.tacc.utexas.edu/tacc-consulting

]],
          luaExt = 20,
          mpath = "/Users/mcawood/Lmod/Lmod/rt/bioContainers/bcd/mf",
          pV = "*ctr.*zfinal-.000000001.*he.000941832.*_.000000002.*zfinal",
          wV = "*ctr.*zfinal-.000000001.*he.000941832.*_.000000002.*zfinal",
          whatis = {
            "Name: extractpirs", "Version: ctr-1.0--he941832_2", "Category: ['Bioinformatics']", "Keywords: ['Biocontainer']", "Description: The extractpirs package", "URL: https://quay.io/repository/biocontainers/extractpirs",
          },
        },
      },
    },
  },
  version = 5,
}
mpathMapT = {}

